<!DOCTYPE html>
<html class=" js flexbox flexboxlegacy canvas canvastext webgl no-touch geolocation postmessage no-websqldatabase indexeddb hashchange history draganddrop websockets rgba hsla multiplebgs backgroundsize borderimage borderradius boxshadow textshadow opacity cssanimations csscolumns cssgradients no-cssreflections csstransforms csstransforms3d csstransitions fontface generatedcontent video audio localstorage sessionstorage webworkers applicationcache svg inlinesvg smil svgclippaths" lang="en"><head>
<meta http-equiv="content-type" content="text/html; charset=UTF-8"><script type="text/javascript" src="Frontiers%20|%20Correcting%20Principal%20Component%20Maps%20for%20Effects%20of%20Spatial%20Autocorrelation%20in%20Population%20Genetic%20Data%20|%20Genetics_fichiers/598a124f17"></script><script src="Frontiers%20|%20Correcting%20Principal%20Component%20Maps%20for%20Effects%20of%20Spatial%20Autocorrelation%20in%20Population%20Genetic%20Data%20|%20Genetics_fichiers/2098247140231737" async=""></script><script async="" src="Frontiers%20|%20Correcting%20Principal%20Component%20Maps%20for%20Effects%20of%20Spatial%20Autocorrelation%20in%20Population%20Genetic%20Data%20|%20Genetics_fichiers/fbevents.js"></script><script type="text/javascript" async="" src="Frontiers%20|%20Correcting%20Principal%20Component%20Maps%20for%20Effects%20of%20Spatial%20Autocorrelation%20in%20Population%20Genetic%20Data%20|%20Genetics_fichiers/analytics.js"></script><script src="Frontiers%20|%20Correcting%20Principal%20Component%20Maps%20for%20Effects%20of%20Spatial%20Autocorrelation%20in%20Population%20Genetic%20Data%20|%20Genetics_fichiers/nr-1118.js"></script><script type="text/javascript" src="Frontiers%20|%20Correcting%20Principal%20Component%20Maps%20for%20Effects%20of%20Spatial%20Autocorrelation%20in%20Population%20Genetic%20Data%20|%20Genetics_fichiers/300lo.js"></script><script type="text/javascript" src="Frontiers%20|%20Correcting%20Principal%20Component%20Maps%20for%20Effects%20of%20Spatial%20Autocorrelation%20in%20Population%20Genetic%20Data%20|%20Genetics_fichiers/_ate.js"></script>
    <!-- Google Tag Manager -->
    <script async="" src="Frontiers%20|%20Correcting%20Principal%20Component%20Maps%20for%20Effects%20of%20Spatial%20Autocorrelation%20in%20Population%20Genetic%20Data%20|%20Genetics_fichiers/analytics.js"></script><script async="" src="Frontiers%20|%20Correcting%20Principal%20Component%20Maps%20for%20Effects%20of%20Spatial%20Autocorrelation%20in%20Population%20Genetic%20Data%20|%20Genetics_fichiers/gtm.js"></script><script>
        (function (w, d, s, l, i) {
            w[l] = w[l] || []; w[l].push({ 'gtm.start': new Date().getTime(), event: 'gtm.js' });
            var f = d.getElementsByTagName(s)[0], j = d.createElement(s), dl = l != 'dataLayer' ? '&l=' + l : ''; j.async = true;
            j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl + '&gtm_auth=fCzlsmYQFAUQ_UXneGJNag&gtm_preview=env-2&gtm_cookies_win=x';
            f.parentNode.insertBefore(j, f);
        })(window, document, 'script', 'dataLayer', 'GTM-TZGWTGG');
    </script>
    <!-- End Google Tag Manager -->

    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge"><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info = {"beacon":"bam.nr-data.net","errorBeacon":"bam.nr-data.net","licenseKey":"598a124f17","applicationID":"3007887","transactionName":"MQcDMkECCkNSW0YMWghNLDBwTCVCR1FRCVAlDQ8SQQwIXFZKHSNACg41A0sXJkl3d3s=","queueTime":0,"applicationTime":259,"agent":"","atts":""}</script><script type="text/javascript">(window.NREUM||(NREUM={})).loader_config={xpid:"VgUHUl5WGwAAVFZaDwY="};window.NREUM||(NREUM={}),__nr_require=function(t,n,e){function r(e){if(!n[e]){var o=n[e]={exports:{}};t[e][0].call(o.exports,function(n){var o=t[e][1][n];return r(o||n)},o,o.exports)}return n[e].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<e.length;o++)r(e[o]);return r}({1:[function(t,n,e){function r(t){try{s.console&&console.log(t)}catch(n){}}var o,i=t("ee"),a=t(16),s={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(s.console=!0,o.indexOf("dev")!==-1&&(s.dev=!0),o.indexOf("nr_dev")!==-1&&(s.nrDev=!0))}catch(c){}s.nrDev&&i.on("internal-error",function(t){r(t.stack)}),s.dev&&i.on("fn-err",function(t,n,e){r(e.stack)}),s.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(s,function(t,n){return t}).join(", ")))},{}],2:[function(t,n,e){function r(t,n,e,r,s){try{p?p-=1:o(s||new UncaughtException(t,n,e),!0)}catch(f){try{i("ierr",[f,c.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,n,e){this.message=t||"Uncaught error with no additional information",this.sourceURL=n,this.line=e}function o(t,n){var e=n?null:c.now();i("err",[t,e])}var i=t("handle"),a=t(17),s=t("ee"),c=t("loader"),f=t("gos"),u=window.onerror,d=!1,l="nr@seenError",p=0;c.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(8),t(7),"addEventListener"in window&&t(5),c.xhrWrappable&&t(9),d=!0)}s.on("fn-start",function(t,n,e){d&&(p+=1)}),s.on("fn-err",function(t,n,e){d&&!e[l]&&(f(e,l,function(){return!0}),this.thrown=!0,o(e))}),s.on("fn-end",function(){d&&!this.thrown&&p>0&&(p-=1)}),s.on("internal-error",function(t){i("ierr",[t,c.now(),!0])})},{}],3:[function(t,n,e){t("loader").features.ins=!0},{}],4:[function(t,n,e){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(8),s=t(7),c="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",l="resource",p="-start",h="-end",m="fn"+p,v="fn"+h,w="bstTimer",y="pushState",g=t("loader");g.features.stn=!0,t(6);var b=NREUM.o.EV;o.on(m,function(t,n){var e=t[0];e instanceof b&&(this.bstStart=g.now())}),o.on(v,function(t,n){var e=t[0];e instanceof b&&i("bst",[e,n,this.bstStart,g.now()])}),a.on(m,function(t,n,e){this.bstStart=g.now(),this.bstType=e}),a.on(v,function(t,n){i(w,[n,this.bstStart,g.now(),this.bstType])}),s.on(m,function(){this.bstStart=g.now()}),s.on(v,function(t,n){i(w,[n,this.bstStart,g.now(),"requestAnimationFrame"])}),o.on(y+p,function(t){this.time=g.now(),this.startPath=location.pathname+location.hash}),o.on(y+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+c]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(l)]),window.performance["c"+c]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(l)]),window.performance["webkitC"+c]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],5:[function(t,n,e){function r(t){for(var n=t;n&&!n.hasOwnProperty(u);)n=Object.getPrototypeOf(n);n&&o(n)}function o(t){s.inPlace(t,[u,d],"-",i)}function i(t,n){return t[1]}var a=t("ee").get("events"),s=t(19)(a,!0),c=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";n.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,n){var e=t[1],r=c(e,"nr@wrapped",function(){function t(){if("function"==typeof e.handleEvent)return e.handleEvent.apply(e,arguments)}var n={object:t,"function":e}[typeof e];return n?s(n,"fn-",null,n.name||"anonymous"):e});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],6:[function(t,n,e){var r=t("ee").get("history"),o=t(19)(r);n.exports=r,o.inPlace(window.history,["pushState","replaceState"],"-")},{}],7:[function(t,n,e){var r=t("ee").get("raf"),o=t(19)(r),i="equestAnimationFrame";n.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],8:[function(t,n,e){function r(t,n,e){t[0]=a(t[0],"fn-",null,e)}function o(t,n,e){this.method=e,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,e)}var i=t("ee").get("timer"),a=t(19)(i),s="setTimeout",c="setInterval",f="clearTimeout",u="-start",d="-";n.exports=i,a.inPlace(window,[s,"setImmediate"],s+d),a.inPlace(window,[c],c+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(c+u,r),i.on(s+u,o)},{}],9:[function(t,n,e){function r(t,n){d.inPlace(n,["onreadystatechange"],"fn-",s)}function o(){var t=this,n=u.context(t);t.readyState>3&&!n.resolved&&(n.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,y,"fn-",s)}function i(t){g.push(t),h&&(x?x.then(a):v?v(a):(E=-E,O.data=E))}function a(){for(var t=0;t<g.length;t++)r([],g[t]);g.length&&(g=[])}function s(t,n){return n}function c(t,n){for(var e in t)n[e]=t[e];return n}t(5);var f=t("ee"),u=f.get("xhr"),d=t(19)(u),l=NREUM.o,p=l.XHR,h=l.MO,m=l.PR,v=l.SI,w="readystatechange",y=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],g=[];n.exports=u;var b=window.XMLHttpRequest=function(t){var n=new p(t);try{u.emit("new-xhr",[n],n),n.addEventListener(w,o,!1)}catch(e){try{u.emit("internal-error",[e])}catch(r){}}return n};if(c(p,b),b.prototype=p.prototype,d.inPlace(b.prototype,["open","send"],"-xhr-",s),u.on("send-xhr-start",function(t,n){r(t,n),i(n)}),u.on("open-xhr-start",r),h){var x=m&&m.resolve();if(!v&&!m){var E=1,O=document.createTextNode(E);new h(a).observe(O,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],10:[function(t,n,e){function r(t){var n=this.params,e=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<d;r++)t.removeEventListener(u[r],this.listener,!1);if(!n.aborted){if(e.duration=a.now()-this.startTime,4===t.readyState){n.status=t.status;var i=o(t,this.lastSize);if(i&&(e.rxSize=i),this.sameOrigin){var c=t.getResponseHeader("X-NewRelic-App-Data");c&&(n.cat=c.split(", ").pop())}}else n.status=0;e.cbTime=this.cbTime,f.emit("xhr-done",[t],t),s("xhr",[n,e,this.startTime])}}}function o(t,n){var e=t.responseType;if("json"===e&&null!==n)return n;var r="arraybuffer"===e||"blob"===e||"json"===e?t.response:t.responseText;return h(r)}function i(t,n){var e=c(n),r=t.params;r.host=e.hostname+":"+e.port,r.pathname=e.pathname,t.sameOrigin=e.sameOrigin}var a=t("loader");if(a.xhrWrappable){var s=t("handle"),c=t(11),f=t("ee"),u=["load","error","abort","timeout"],d=u.length,l=t("id"),p=t(14),h=t(13),m=window.XMLHttpRequest;a.features.xhr=!0,t(9),f.on("new-xhr",function(t){var n=this;n.totalCbs=0,n.called=0,n.cbTime=0,n.end=r,n.ended=!1,n.xhrGuids={},n.lastSize=null,p&&(p>34||p<10)||window.opera||t.addEventListener("progress",function(t){n.lastSize=t.loaded},!1)}),f.on("open-xhr-start",function(t){this.params={method:t[0]},i(this,t[1]),this.metrics={}}),f.on("open-xhr-end",function(t,n){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&n.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid)}),f.on("send-xhr-start",function(t,n){var e=this.metrics,r=t[0],o=this;if(e&&r){var i=h(r);i&&(e.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"===t.type&&(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof n.onload))&&o.end(n)}catch(e){try{f.emit("internal-error",[e])}catch(r){}}};for(var s=0;s<d;s++)n.addEventListener(u[s],this.listener,!1)}),f.on("xhr-cb-time",function(t,n,e){this.cbTime+=t,n?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof e.onload||this.end(e)}),f.on("xhr-load-added",function(t,n){var e=""+l(t)+!!n;this.xhrGuids&&!this.xhrGuids[e]&&(this.xhrGuids[e]=!0,this.totalCbs+=1)}),f.on("xhr-load-removed",function(t,n){var e=""+l(t)+!!n;this.xhrGuids&&this.xhrGuids[e]&&(delete this.xhrGuids[e],this.totalCbs-=1)}),f.on("addEventListener-end",function(t,n){n instanceof m&&"load"===t[0]&&f.emit("xhr-load-added",[t[1],t[2]],n)}),f.on("removeEventListener-end",function(t,n){n instanceof m&&"load"===t[0]&&f.emit("xhr-load-removed",[t[1],t[2]],n)}),f.on("fn-start",function(t,n,e){n instanceof m&&("onload"===e&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),f.on("fn-end",function(t,n){this.xhrCbStart&&f.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,n],n)})}},{}],11:[function(t,n,e){n.exports=function(t){var n=document.createElement("a"),e=window.location,r={};n.href=t,r.port=n.port;var o=n.href.split("://");!r.port&&o[1]&&(r.port=o[1].split("/")[0].split("@").pop().split(":")[1]),r.port&&"0"!==r.port||(r.port="https"===o[0]?"443":"80"),r.hostname=n.hostname||e.hostname,r.pathname=n.pathname,r.protocol=o[0],"/"!==r.pathname.charAt(0)&&(r.pathname="/"+r.pathname);var i=!n.protocol||":"===n.protocol||n.protocol===e.protocol,a=n.hostname===document.domain&&n.port===e.port;return r.sameOrigin=i&&(!n.hostname||a),r}},{}],12:[function(t,n,e){function r(){}function o(t,n,e){return function(){return i(t,[f.now()].concat(s(arguments)),n?null:this,e),n?void 0:this}}var i=t("handle"),a=t(16),s=t(17),c=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],l="api-",p=l+"ixn-";a(d,function(t,n){u[n]=o(l+n,!0,"api")}),u.addPageAction=o(l+"addPageAction",!0),u.setCurrentRouteName=o(l+"routeName",!0),n.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,n){var e={},r=this,o="function"==typeof n;return i(p+"tracer",[f.now(),t,e],r),function(){if(c.emit((o?"":"no-")+"fn-start",[f.now(),r,o],e),o)try{return n.apply(this,arguments)}catch(t){throw c.emit("fn-err",[arguments,this,t],e),t}finally{c.emit("fn-end",[f.now()],e)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,n){h[n]=o(p+n)}),newrelic.noticeError=function(t,n){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now(),!1,n])}},{}],13:[function(t,n,e){n.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(n){return}}}},{}],14:[function(t,n,e){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),n.exports=r},{}],15:[function(t,n,e){function r(t,n){if(!o)return!1;if(t!==o)return!1;if(!n)return!0;if(!i)return!1;for(var e=i.split("."),r=n.split("."),a=0;a<r.length;a++)if(r[a]!==e[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var s=navigator.userAgent,c=s.match(a);c&&s.indexOf("Chrome")===-1&&s.indexOf("Chromium")===-1&&(o="Safari",i=c[1])}n.exports={agent:o,version:i,match:r}},{}],16:[function(t,n,e){function r(t,n){var e=[],r="",i=0;for(r in t)o.call(t,r)&&(e[i]=n(r,t[r]),i+=1);return e}var o=Object.prototype.hasOwnProperty;n.exports=r},{}],17:[function(t,n,e){function r(t,n,e){n||(n=0),"undefined"==typeof e&&(e=t?t.length:0);for(var r=-1,o=e-n||0,i=Array(o<0?0:o);++r<o;)i[r]=t[n+r];return i}n.exports=r},{}],18:[function(t,n,e){n.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],19:[function(t,n,e){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(17),a="nr@original",s=Object.prototype.hasOwnProperty,c=!1;n.exports=function(t,n){function e(t,n,e,o){function nrWrapper(){var r,a,s,c;try{a=this,r=i(arguments),s="function"==typeof e?e(r,a):e||{}}catch(f){l([f,"",[r,a,o],s])}u(n+"start",[r,a,o],s);try{return c=t.apply(a,r)}catch(d){throw u(n+"err",[r,a,d],s),d}finally{u(n+"end",[r,a,c],s)}}return r(t)?t:(n||(n=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,n,o,i){o||(o="");var a,s,c,f="-"===o.charAt(0);for(c=0;c<n.length;c++)s=n[c],a=t[s],r(a)||(t[s]=e(a,f?s+o:o,i,s))}function u(e,r,o){if(!c||n){var i=c;c=!0;try{t.emit(e,r,o,n)}catch(a){l([a,e,r,o])}c=i}}function d(t,n){if(Object.defineProperty&&Object.keys)try{var e=Object.keys(t);return e.forEach(function(e){Object.defineProperty(n,e,{get:function(){return t[e]},set:function(n){return t[e]=n,n}})}),n}catch(r){l([r])}for(var o in t)s.call(t,o)&&(n[o]=t[o]);return n}function l(n){try{t.emit("internal-error",n)}catch(e){}}return t||(t=o),e.inPlace=f,e.flag=a,e}},{}],ee:[function(t,n,e){function r(){}function o(t){function n(t){return t&&t instanceof r?t:t?c(t,s,i):i()}function e(e,r,o,i){if(!l.aborted||i){t&&t(e,r,o);for(var a=n(o),s=m(e),c=s.length,f=0;f<c;f++)s[f].apply(a,r);var d=u[g[e]];return d&&d.push([b,e,r,a]),a}}function p(t,n){y[t]=m(t).concat(n)}function h(t,n){var e=y[t];if(e)for(var r=0;r<e.length;r++)e[r]===n&&e.splice(r,1)}function m(t){return y[t]||[]}function v(t){return d[t]=d[t]||o(e)}function w(t,n){f(t,function(t,e){n=n||"feature",g[e]=n,n in u||(u[n]=[])})}var y={},g={},b={on:p,addEventListener:p,removeEventListener:h,emit:e,get:v,listeners:m,context:n,buffer:w,abort:a,aborted:!1};return b}function i(){return new r}function a(){(u.api||u.feature)&&(l.aborted=!0,u=l.backlog={})}var s="nr@context",c=t("gos"),f=t(16),u={},d={},l=n.exports=o();l.backlog=u},{}],gos:[function(t,n,e){function r(t,n,e){if(o.call(t,n))return t[n];var r=e();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,n,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[n]=r,r}var o=Object.prototype.hasOwnProperty;n.exports=r},{}],handle:[function(t,n,e){function r(t,n,e,r){o.buffer([t],r),o.emit(t,n,e)}var o=t("ee").get("handle");n.exports=r,r.ee=o},{}],id:[function(t,n,e){function r(t){var n=typeof t;return!t||"object"!==n&&"function"!==n?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");n.exports=r},{}],loader:[function(t,n,e){function r(){if(!E++){var t=x.info=NREUM.info,n=p.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&n))return u.abort();f(g,function(n,e){t[n]||(t[n]=e)}),c("mark",["onload",a()+x.offset],null,"api");var e=p.createElement("script");e.src="https://"+t.agent,n.parentNode.insertBefore(e,n)}}function o(){"complete"===p.readyState&&i()}function i(){c("mark",["domContent",a()+x.offset],null,"api")}function a(){return O.exists&&performance.now?Math.round(performance.now()):(s=Math.max((new Date).getTime(),s))-x.offset}var s=(new Date).getTime(),c=t("handle"),f=t(16),u=t("ee"),d=t(15),l=window,p=l.document,h="addEventListener",m="attachEvent",v=l.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:l.setImmediate,CT:clearTimeout,XHR:v,REQ:l.Request,EV:l.Event,PR:l.Promise,MO:l.MutationObserver};var y=""+location,g={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-1118.min.js"},b=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),x=n.exports={offset:s,now:a,origin:y,features:{},xhrWrappable:b,userAgent:d};t(12),p[h]?(p[h]("DOMContentLoaded",i,!1),l[h]("load",r,!1)):(p[m]("onreadystatechange",o),l[m]("onload",r)),c("mark",["firstbyte",s],null,"api");var E=0,O=t(18)},{}]},{},["loader",2,10,4,3]);</script>
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">

    <title>Frontiers | Correcting Principal Component Maps for Effects of Spatial Autocorrelation in Population Genetic Data | Genetics</title>

    <link rel="shortcut icon" href="https://3718aeafc638f96f5bd6-d4a9ca15fc46ba40e71f94dec0aad28c.ssl.cf1.rackcdn.com/favicon_16x16.ico" type="image/x-icon">

    <meta property="og:type" content="article">
    <meta property="frontiers:type" content="Article">
    <meta property="og:site_name" name="site_name" content="Frontiers">
    <meta property="og:title" name="Title" content="Correcting Principal Component Maps for Effects of Spatial Autocorrelation in Population Genetic Data">
    <meta property="og:description" name="Description" content="In many species, spatial genetic variation displays patterns of “isolation-by-distance”. Characterized by locally correlated allele frequencies, these patterns are known to create periodic shapes in geographic maps of principal components which confound signatures of specific migration events and influence interpretations of principal component analyses (PCA). In this study, we introduced models combining probabilistic PCA and kriging models to infer population genetic structure from genetic data while correcting for effects generated by spatial autocorrelation. The corresponding algorithms are based on singular value decomposition and low rank approximation of the genotypic data. As their complexity is close to that of PCA, these algorithms scale with the dimension of the data. To illustrate the utility of these new models, we simulated isolation-by-distance patterns and broad-scale geographic variation using spatial coalescent models. Our methods remove the horseshoe patterns usually observed in PC maps and simplify intrepretations of spatial genetic variation. We demonstrate our approach by analyzing single nucleotide polymorphism data from the Human Genome Diversity Panel, and provide comparisons with other recently introduced methods.">
    <meta property="og:url" name="url" content="https://www.frontiersin.org/article/10.3389/fgene.2012.00254/full">
        <meta property="og:image" content="https://www.frontiersin.org/files/MyHome%20Article%20Library/34644/34644_Thumb_400.jpg">
            <meta name="citation_volume" content="3">
        <meta name="citation_journal_title" content="Frontiers in Genetics">
        <meta name="citation_publisher" content="Frontiers">
        <meta name="citation_journal_abbrev" content="Front. Genet.">
        <meta name="citation_issn" content="1664-8021">
        <meta name="citation_doi" content="10.3389/fgene.2012.00254">
        <meta name="citation_pages" content="254">
        <meta name="citation_language" content="English">
        <meta name="citation_title" content="Correcting Principal Component Maps for Effects of Spatial Autocorrelation in Population Genetic Data">
        <meta name="citation_keywords" content="Principal Component Analysis; isolation-by-distance; spatial autocorrelation; spatial factor analysis; singular value decomposition">
        <meta name="citation_abstract" content="In many species, spatial genetic variation displays patterns of “isolation-by-distance”. Characterized by locally correlated allele frequencies, these patterns are known to create periodic shapes in geographic maps of principal components which confound signatures of specific migration events and influence interpretations of principal component analyses (PCA). In this study, we introduced models combining probabilistic PCA and kriging models to infer population genetic structure from genetic data while correcting for effects generated by spatial autocorrelation. The corresponding algorithms are based on singular value decomposition and low rank approximation of the genotypic data. As their complexity is close to that of PCA, these algorithms scale with the dimension of the data. To illustrate the utility of these new models, we simulated isolation-by-distance patterns and broad-scale geographic variation using spatial coalescent models. Our methods remove the horseshoe patterns usually observed in PC maps and simplify intrepretations of spatial genetic variation. We demonstrate our approach by analyzing single nucleotide polymorphism data from the Human Genome Diversity Panel, and provide comparisons with other recently introduced methods.">
        <meta name="description" content="In many species, spatial genetic variation displays patterns of “isolation-by-distance”. Characterized by locally correlated allele frequencies, these patterns are known to create periodic shapes in geographic maps of principal components which confound signatures of specific migration events and influence interpretations of principal component analyses (PCA). In this study, we introduced models combining probabilistic PCA and kriging models to infer population genetic structure from genetic data while correcting for effects generated by spatial autocorrelation. The corresponding algorithms are based on singular value decomposition and low rank approximation of the genotypic data. As their complexity is close to that of PCA, these algorithms scale with the dimension of the data. To illustrate the utility of these new models, we simulated isolation-by-distance patterns and broad-scale geographic variation using spatial coalescent models. Our methods remove the horseshoe patterns usually observed in PC maps and simplify intrepretations of spatial genetic variation. We demonstrate our approach by analyzing single nucleotide polymorphism data from the Human Genome Diversity Panel, and provide comparisons with other recently introduced methods.">
        <meta name="citation_online_date" content="2012/10/29">
        <meta name="citation_date" content="2012">
        <meta name="citation_publication_date" content="2012/11/20">
        <meta name="citation_pdf_url" content="https://www.frontiersin.org/articles/10.3389/fgene.2012.00254/pdf">
        <meta name="citation_author" content="Frichot, Eric">
        <meta name="citation_author_institution" content="University Joseph Fourier Grenoble, Faculty of Medicine, La Tronche, 38706, France">
        <meta name="citation_author_email" content="Eric.Frichot@imag.fr">
        <meta name="citation_author" content="Schoville, Sean D">
        <meta name="citation_author_institution" content="University Joseph Fourier Grenoble, Faculty of Medicine, La Tronche, 38706, France">
        <meta name="citation_author_email" content="Sean.Schoville@imag.fr">
        <meta name="citation_author" content="Bouchard, Guillaume">
        <meta name="citation_author_institution" content="Xerox Research Center Europe, Meylan, 38042, France">
        <meta name="citation_author_email" content="guillaume.bouchard@xerox.com">
        <meta name="citation_author" content="Francois, Olivier">
        <meta name="citation_author_institution" content="University Joseph Fourier Grenoble, Faculty of Medicine, La Tronche, 38706, France">
        <meta name="citation_author_email" content="olivier.francois@imag.fr">
    <meta name="Keywords" content="Principal Component Analysis, isolation-by-distance, spatial autocorrelation, spatial factor analysis, singular value decomposition">


    <script type="text/javascript">

        var CurrentIBarMenu = 'bysubjects';
        var CurrentPageCode = 'ARTICLE_PAGE_NEW';

        var FRConfiguration = (function() {
            return {
                Environment: 'Live',
                SANVirtualPath: 'http://www.frontiersin.org/files/',
                SharepointWebsiteUrl: 'https://www.frontiersin.org',
                CommunityWebsiteUrl: 'http://community.frontiersin.org',
                FrontiersJournalUIUrl: 'https://www.frontiersin.org',
                FrontiersJournalAPIUrl: 'https://api-journal.frontiersin.org',
                FrontiersReviewUIUrl: '',
                FrontiersReviewAPIUrl: '',
                FrontiersNetworkingAPIUrl: 'https://api-network.frontiersin.org',
                FrontiersCookie: 'frontiersN',
                FrontiersCookieRememberMe: 'frontiersNt',
                FrontiersLoginUrl: 'https://www.frontiersin.org/Login.aspx',
                FrontiersRegistrationUrl: 'http://www.frontiersin.org/Registration/Register.aspx',
                IsJournalCommentApiEnabled : 'True'
            };
        })();

        var FRLanguage = (function() {
            var languageSet = {"ART_FRONTIERS":"Frontiers ","Article_AcceptedDate":"Accepted: ","Article_AnalyticsToolTip":"The total view count is updated once a day, so not to worry if you don\u0027t see immediate results.","Article_AnalyticsTotalViews":"total views","Article_AnalyticsViewImpact":"View Article Impact","Article_ArchiveLinkText":"Articles","Article_BibTex":"BibTex","Article_Citation":"Citation: ","Article_Commentary":"COMMENTARY","Article_Copyright":"Copyright: ","Article_CopyrightText":"This is an open-access article distributed under the terms of the \u003ca href=\"http://creativecommons.org/licenses/by/4.0/\"\u003eCreative Commons Attribution License (CC BY)\u003c/a\u003e. The use, distribution or reproduction in other forums is permitted, provided the original author(s) and the copyright owner(s) are credited and that the original publication in this journal is cited, in accordance with accepted academic practice. No use, distribution or reproduction is permitted which does not comply with these terms.","Article_Correspondence":"* Correspondence: ","Article_DownloadArticle":"Download Article","Article_DownloadPDF":"Download PDF","Article_DownloadProvisionalArticle":"Download Provisional Article","Article_DownloadProvisionalPDF":"Download Provisional PDF","Article_EditedBy":"Edited by: ","Article_EndNote":"EndNote","Article_EPUB":"EPUB","Article_ExportCitation":"Export Citation","Article_JATS":"JATS","Article_Keywords":"Keywords: ","Article_NLM":"XML (NLM)","Article_OriginalArticle":"ORIGINAL ARTICLE","Article_PaperPendingPublishedDate":"Paper pending published: ","Article_PDF":"PDF","Article_ProvisionalPDF":"Provisional PDF","Article_PublishedDate":"Published online: ","Article_ReadFullText":"Read Full Text","Article_ReceivedDate":"Received: ","Article_ReferenceManager":"Reference Manager","Article_ReviewedBy":"Reviewed by: ","Article_RTInfoText":"This article is part of the Research Topic","Article_ShareOn":"SHARE ON","Article_SimpleTEXTfile":"Simple TEXT file","Article_SupplementalData":"SUPPLEMENTAL DATA","Article_TableOfContent":"TABLE OF CONTENTS","Article_ViewEnhancedPDF":"ReadCube","Article_XML":"XML","BrowserWarningText":"\u003ch2\u003eWarning!\u003c/h2\u003e\u003cp\u003eYou are using an \u003cstrong\u003eoutdated\u003c/strong\u003e browser. This page doesn\u0027t support Internet Explorer 6, 7 and 8.\u003cbr /\u003ePlease \u003ca class=\"blue\" href=\"http://browsehappy.com/\"\u003eupgrade your browser\u003c/a\u003e or \u003ca class=\"blue\" href=\"http://www.google.com/chromeframe/?redirect=true\"\u003eactivate Google Chrome Frame\u003c/a\u003e to improve your experience.\u003c/p\u003e","COMMENT_HEADERTEXT":"Comment text too long","COMMENT_WARNINGTEXT":"Comments must be less than 4,000 characters. You have entered ","Impact_BackToArticle":"Back to article","People_Also_LookedAt":"People also looked at"};

            return {
                value: function(key) {
                    if (languageSet[key]) {
                        return languageSet[key];
                    } else {
                        throw new Error('Unable to get the value status from the language set'); // Use Error, not FRError
                    }
                }
            };

        })();

        var FRJournalDetails = (function() {
            return {
                JournalType: 'section',
                JournalId: '240',
                SectionId: '549'
            };
        })();

        var FRArticleRecaptchaSettings = (function() {
            return {
                RecaptchaSiteKey: '6LdG3i0UAAAAAOC4qUh35ubHgJotEHp_STXHgr_v'
            };
        })();

    </script>

    <script src="Frontiers%20|%20Correcting%20Principal%20Component%20Maps%20for%20Effects%20of%20Spatial%20Autocorrelation%20in%20Population%20Genetic%20Data%20|%20Genetics_fichiers/addthis_widget.js"></script><div style="visibility: hidden; height: 1px; width: 1px; position: absolute; top: -9999px; z-index: 100000;" id="_atssh"><iframe id="_atssh298" title="AddThis utility frame" style="height: 1px; width: 1px; position: absolute; top: 0px; z-index: 100000; border: 0px none; left: 0px;" src="Frontiers%20|%20Correcting%20Principal%20Component%20Maps%20for%20Effects%20of%20Spatial%20Autocorrelation%20in%20Population%20Genetic%20Data%20|%20Genetics_fichiers/sh.html"></iframe></div><style type="text/css">.at-icon{fill:#fff;border:0}.at-icon-wrapper{display:inline-block;overflow:hidden}a .at-icon-wrapper{cursor:pointer}.at-rounded,.at-rounded-element .at-icon-wrapper{border-radius:12%}.at-circular,.at-circular-element .at-icon-wrapper{border-radius:50%}.addthis_32x32_style .at-icon{width:2pc;height:2pc}.addthis_24x24_style .at-icon{width:24px;height:24px}.addthis_20x20_style .at-icon{width:20px;height:20px}.addthis_16x16_style .at-icon{width:1pc;height:1pc}#at16lb{display:none;position:absolute;top:0;left:0;width:100%;height:100%;z-index:1001;background-color:#000;opacity:.001}#at_complete,#at_error,#at_share,#at_success{position:static!important}.at15dn{display:none}#at15s,#at16p,#at16p form input,#at16p label,#at16p textarea,#at_share .at_item{font-family:arial,helvetica,tahoma,verdana,sans-serif!important;font-size:9pt!important;outline-style:none;outline-width:0;line-height:1em}* html #at15s.mmborder{position:absolute!important}#at15s.mmborder{position:fixed!important;width:250px!important}#at15s{background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);float:none;line-height:1em;margin:0;overflow:visible;padding:5px;text-align:left;position:absolute}#at15s a,#at15s span{outline:0;direction:ltr;text-transform:none}#at15s .at-label{margin-left:5px}#at15s .at-icon-wrapper{width:1pc;height:1pc;vertical-align:middle}#at15s .at-icon{width:1pc;height:1pc}.at4-icon{display:inline-block;background-repeat:no-repeat;background-position:top left;margin:0;overflow:hidden;cursor:pointer}.addthis_16x16_style .at4-icon,.addthis_default_style .at4-icon,.at4-icon,.at-16x16{width:1pc;height:1pc;line-height:1pc;background-size:1pc!important}.addthis_32x32_style .at4-icon,.at-32x32{width:2pc;height:2pc;line-height:2pc;background-size:2pc!important}.addthis_24x24_style .at4-icon,.at-24x24{width:24px;height:24px;line-height:24px;background-size:24px!important}.addthis_20x20_style .at4-icon,.at-20x20{width:20px;height:20px;line-height:20px;background-size:20px!important}.at4-icon.circular,.circular .at4-icon,.circular.aticon{border-radius:50%}.at4-icon.rounded,.rounded .at4-icon{border-radius:4px}.at4-icon-left{float:left}#at15s .at4-icon{text-indent:20px;padding:0;overflow:visible;white-space:nowrap;background-size:1pc;width:1pc;height:1pc;background-position:top left;display:inline-block;line-height:1pc}.addthis_vertical_style .at4-icon,.at4-follow-container .at4-icon{margin-right:5px}html>body #at15s{width:250px!important}#at15s.atm{background:none!important;padding:0!important;width:10pc!important}#at15s_inner{background:#fff;border:1px solid #fff;margin:0}#at15s_head{position:relative;background:#f2f2f2;padding:4px;cursor:default;border-bottom:1px solid #e5e5e5}.at15s_head_success{background:#cafd99!important;border-bottom:1px solid #a9d582!important}.at15s_head_success a,.at15s_head_success span{color:#000!important;text-decoration:none}#at15s_brand,#at15sptx,#at16_brand{position:absolute}#at15s_brand{top:4px;right:4px}.at15s_brandx{right:20px!important}a#at15sptx{top:4px;right:4px;text-decoration:none;color:#4c4c4c;font-weight:700}#at15sptx:hover{text-decoration:underline}#at16_brand{top:5px;right:30px;cursor:default}#at_hover{padding:4px}#at_hover .at_item,#at_share .at_item{background:#fff!important;float:left!important;color:#4c4c4c!important}#at_share .at_item .at-icon-wrapper{margin-right:5px}#at_hover .at_bold{font-weight:700;color:#000!important}#at_hover .at_item{width:7pc!important;padding:2px 3px!important;margin:1px;text-decoration:none!important}#at_hover .at_item.athov,#at_hover .at_item:focus,#at_hover .at_item:hover{margin:0!important}#at_hover .at_item.athov,#at_hover .at_item:focus,#at_hover .at_item:hover,#at_share .at_item.athov,#at_share .at_item:hover{background:#f2f2f2!important;border:1px solid #e5e5e5;color:#000!important;text-decoration:none}.ipad #at_hover .at_item:focus{background:#fff!important;border:1px solid #fff}.at15t{display:block!important;height:1pc!important;line-height:1pc!important;padding-left:20px!important;background-position:0 0;text-align:left}.addthis_button,.at15t{cursor:pointer}.addthis_toolbox a.at300b,.addthis_toolbox a.at300m{width:auto}.addthis_toolbox a{margin-bottom:5px;line-height:initial}.addthis_toolbox.addthis_vertical_style{width:200px}.addthis_button_facebook_like .fb_iframe_widget{line-height:100%}.addthis_button_facebook_like iframe.fb_iframe_widget_lift{max-width:none}.addthis_toolbox a.addthis_button_counter,.addthis_toolbox a.addthis_button_facebook_like,.addthis_toolbox a.addthis_button_facebook_send,.addthis_toolbox a.addthis_button_facebook_share,.addthis_toolbox a.addthis_button_foursquare,.addthis_toolbox a.addthis_button_linkedin_counter,.addthis_toolbox a.addthis_button_pinterest_pinit,.addthis_toolbox a.addthis_button_tweet{display:inline-block}.addthis_toolbox span.addthis_follow_label{display:none}.addthis_toolbox.addthis_vertical_style span.addthis_follow_label{display:block;white-space:nowrap}.addthis_toolbox.addthis_vertical_style a{display:block}.addthis_toolbox.addthis_vertical_style.addthis_32x32_style a{line-height:2pc;height:2pc}.addthis_toolbox.addthis_vertical_style .at300bs{margin-right:4px;float:left}.addthis_toolbox.addthis_20x20_style span{line-height:20px}.addthis_toolbox.addthis_32x32_style span{line-height:2pc}.addthis_toolbox.addthis_pill_combo_style .addthis_button_compact .at15t_compact,.addthis_toolbox.addthis_pill_combo_style a{float:left}.addthis_toolbox.addthis_pill_combo_style a.addthis_button_tweet{margin-top:-2px}.addthis_toolbox.addthis_pill_combo_style .addthis_button_compact .at15t_compact{margin-right:4px}.addthis_default_style .addthis_separator{margin:0 5px;display:inline}div.atclear{clear:both}.addthis_default_style .addthis_separator,.addthis_default_style .at4-icon,.addthis_default_style .at300b,.addthis_default_style .at300bo,.addthis_default_style .at300bs,.addthis_default_style .at300m{float:left}.at300b img,.at300bo img{border:0}a.at300b .at4-icon,a.at300m .at4-icon{display:block}.addthis_default_style .at300b,.addthis_default_style .at300bo,.addthis_default_style .at300m{padding:0 2px}.at300b,.at300bo,.at300bs,.at300m{cursor:pointer}.addthis_button_facebook_like.at300b:hover,.addthis_button_facebook_like.at300bs:hover,.addthis_button_facebook_send.at300b:hover,.addthis_button_facebook_send.at300bs:hover{opacity:1}.addthis_20x20_style .at15t,.addthis_20x20_style .at300bs{overflow:hidden;display:block;height:20px!important;width:20px!important;line-height:20px!important}.addthis_32x32_style .at15t,.addthis_32x32_style .at300bs{overflow:hidden;display:block;height:2pc!important;width:2pc!important;line-height:2pc!important}.at300bs{overflow:hidden;display:block;background-position:0 0;height:1pc;width:1pc;line-height:1pc!important}.addthis_default_style .at15t_compact,.addthis_default_style .at15t_expanded{margin-right:4px}#at_share .at_item{width:123px!important;padding:4px;margin-right:2px;border:1px solid #fff}#at16p{background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);z-index:10000001;position:absolute;top:50%;left:50%;width:300px;padding:10px;margin:0 auto;margin-top:-185px;margin-left:-155px;font-family:arial,helvetica,tahoma,verdana,sans-serif;font-size:9pt;color:#5e5e5e}#at_share{margin:0;padding:0}#at16pt{position:relative;background:#f2f2f2;height:13px;padding:5px 10px}#at16pt a,#at16pt h4{font-weight:700}#at16pt h4{display:inline;margin:0;padding:0;font-size:9pt;color:#4c4c4c;cursor:default}#at16pt a{position:absolute;top:5px;right:10px;color:#4c4c4c;text-decoration:none;padding:2px}#at15sptx:focus,#at16pt a:focus{outline:thin dotted}#at15s #at16pf a{top:1px}#_atssh{width:1px!important;height:1px!important;border:0!important}.atm{width:10pc!important;padding:0;margin:0;line-height:9pt;letter-spacing:normal;font-family:arial,helvetica,tahoma,verdana,sans-serif;font-size:9pt;color:#444;background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);padding:4px}.atm-f{text-align:right;border-top:1px solid #ddd;padding:5px 8px}.atm-i{background:#fff;border:1px solid #d5d6d6;padding:0;margin:0;box-shadow:1px 1px 5px rgba(0,0,0,.15)}.atm-s{margin:0!important;padding:0!important}.atm-s a:focus{border:transparent;outline:0;transition:none}#at_hover.atm-s a,.atm-s a{display:block;text-decoration:none;padding:4px 10px;color:#235dab!important;font-weight:400;font-style:normal;transition:none}#at_hover.atm-s .at_bold{color:#235dab!important}#at_hover.atm-s a:hover,.atm-s a:hover{background:#2095f0;text-decoration:none;color:#fff!important}#at_hover.atm-s .at_bold{font-weight:700}#at_hover.atm-s a:hover .at_bold{color:#fff!important}.atm-s a .at-label{vertical-align:middle;margin-left:5px;direction:ltr}.at_PinItButton{display:block;width:40px;height:20px;padding:0;margin:0;background-image:url(//s7.addthis.com/static/t00/pinit00.png);background-repeat:no-repeat}.at_PinItButton:hover{background-position:0 -20px}.addthis_toolbox .addthis_button_pinterest_pinit{position:relative}.at-share-tbx-element .fb_iframe_widget span{vertical-align:baseline!important}#at16pf{height:auto;text-align:right;padding:4px 8px}.at-privacy-info{position:absolute;left:7px;bottom:7px;cursor:pointer;text-decoration:none;font-family:helvetica,arial,sans-serif;font-size:10px;line-height:9pt;letter-spacing:.2px;color:#666}.at-privacy-info:hover{color:#000}.body .wsb-social-share .wsb-social-share-button-vert{padding-top:0;padding-bottom:0}.body .wsb-social-share.addthis_counter_style .addthis_button_tweet.wsb-social-share-button{padding-top:40px}.body .wsb-social-share.addthis_counter_style .addthis_button_facebook_like.wsb-social-share-button{padding-top:21px}@media print{#at4-follow,#at4-share,#at4-thankyou,#at4-whatsnext,#at4m-mobile,#at15s,.at4,.at4-recommended{display:none!important}}@media screen and (max-width:400px){.at4win{width:100%}}@media screen and (max-height:700px) and (max-width:400px){.at4-thankyou-inner .at4-recommended-container{height:122px;overflow:hidden}.at4-thankyou-inner .at4-recommended .at4-recommended-item:first-child{border-bottom:1px solid #c5c5c5}}</style><style type="text/css">.at-branding-logo{font-family:helvetica,arial,sans-serif;text-decoration:none;font-size:10px;display:inline-block;margin:2px 0;letter-spacing:.2px}.at-branding-logo .at-branding-icon{background-image:url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAMAAAC67D+PAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAAAZQTFRF////+GlNUkcc1QAAAB1JREFUeNpiYIQDBjQmAwMmkwEM0JnY1WIxFyDAABGeAFEudiZsAAAAAElFTkSuQmCC")}.at-branding-logo .at-branding-icon,.at-branding-logo .at-privacy-icon{display:inline-block;height:10px;width:10px;margin-left:4px;margin-right:3px;margin-bottom:-1px;background-repeat:no-repeat}.at-branding-logo .at-privacy-icon{background-image:url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAkAAAAKCAMAAABR24SMAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABhQTFRF8fr9ot/xXcfn2/P5AKva////////AKTWodjhjAAAAAd0Uk5T////////ABpLA0YAAAA6SURBVHjaJMzBDQAwCAJAQaj7b9xifV0kUKJ9ciWxlzWEWI5gMF65KUTv0VKkjVeTerqE/x7+9BVgAEXbAWI8QDcfAAAAAElFTkSuQmCC")}.at-branding-logo span{text-decoration:none}.at-branding-logo .at-branding-addthis,.at-branding-logo .at-branding-powered-by{color:#666}.at-branding-logo .at-branding-addthis:hover{color:#333}.at-cv-with-image .at-branding-addthis,.at-cv-with-image .at-branding-addthis:hover{color:#fff}a.at-branding-logo:visited{color:initial}.at-branding-info{display:inline-block;padding:0 5px;color:#666;border:1px solid #666;border-radius:50%;font-size:10px;line-height:9pt;opacity:.7;transition:all .3s ease;text-decoration:none}.at-branding-info span{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}.at-branding-info:before{content:'i';font-family:Times New Roman}.at-branding-info:hover{color:#0780df;border-color:#0780df}</style><style id="service-icons-0"></style>


    <script language="javascript">
        var addthis_config = addthis_config || {};
        addthis_config.data_track_addressbar = false;
        addthis_config.data_track_clickback = false;
    </script>

    
    
    <!--[if lt IE 9]>
      <script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>

      <script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>

    <![endif]-->

    <script src="Frontiers%20|%20Correcting%20Principal%20Component%20Maps%20for%20Effects%20of%20Spatial%20Autocorrelation%20in%20Population%20Genetic%20Data%20|%20Genetics_fichiers/js"></script>

    <script src="Frontiers%20|%20Correcting%20Principal%20Component%20Maps%20for%20Effects%20of%20Spatial%20Autocorrelation%20in%20Population%20Genetic%20Data%20|%20Genetics_fichiers/frontiers_002.js"></script>



    <link href="Frontiers%20|%20Correcting%20Principal%20Component%20Maps%20for%20Effects%20of%20Spatial%20Autocorrelation%20in%20Population%20Genetic%20Data%20|%20Genetics_fichiers/font-awesome.css" rel="stylesheet">

    <link href="Frontiers%20|%20Correcting%20Principal%20Component%20Maps%20for%20Effects%20of%20Spatial%20Autocorrelation%20in%20Population%20Genetic%20Data%20|%20Genetics_fichiers/museo-sans.css" rel="stylesheet">

    <link href="Frontiers%20|%20Correcting%20Principal%20Component%20Maps%20for%20Effects%20of%20Spatial%20Autocorrelation%20in%20Population%20Genetic%20Data%20|%20Genetics_fichiers/museo-slab.css" rel="stylesheet">

    <link href="Frontiers%20|%20Correcting%20Principal%20Component%20Maps%20for%20Effects%20of%20Spatial%20Autocorrelation%20in%20Population%20Genetic%20Data%20|%20Genetics_fichiers/open-sans.css" rel="stylesheet">

    <link href="Frontiers%20|%20Correcting%20Principal%20Component%20Maps%20for%20Effects%20of%20Spatial%20Autocorrelation%20in%20Population%20Genetic%20Data%20|%20Genetics_fichiers/app.css" rel="stylesheet">

    
    <style type="text/css"> .journal-genetics {
 background: #fff url('https://3718aeafc638f96f5bd6-d4a9ca15fc46ba40e71f94dec0aad28c.ssl.cf1.rackcdn.com/journal-genetics.png') no-repeat 100% 10px;
 background-size: 51%;
}

/* Displays/Screens (e.g. 19" WS @ 1440x900) --------------- */ @media only screen and (max-width: 1649px) {
.journal-genetics {
  background: #fff url('https://3718aeafc638f96f5bd6-d4a9ca15fc46ba40e71f94dec0aad28c.ssl.cf1.rackcdn.com/journal-genetics.png') no-repeat 140% 70px;
 background-size: 54%;
 }}

/* Displays/Screens (e.g. MacBook @ 1280x800) -------------- */
@media only screen and (max-width: 1409px) {
.journal-genetics {
  background: #fff url('https://3718aeafc638f96f5bd6-d4a9ca15fc46ba40e71f94dec0aad28c.ssl.cf1.rackcdn.com/journal-genetics.png') no-repeat 140% 70px;
 background-size: 54%;
 }}
 
/* Large Devices, Wide Screens --------- */  @media only screen and (max-width : 1250px) {
.journal-genetics {
  background: #fff url('https://3718aeafc638f96f5bd6-d4a9ca15fc46ba40e71f94dec0aad28c.ssl.cf1.rackcdn.com/journal-genetics.png') no-repeat 140% 70px;
 background-size: 54%;
 }}

/* Medium Devices, Desktops ---------- */  @media only screen and (max-width : 992px) { 
.journal-genetics {
  background: #fff url('https://3718aeafc638f96f5bd6-d4a9ca15fc46ba40e71f94dec0aad28c.ssl.cf1.rackcdn.com/journal-genetics.png') no-repeat 140% 70px;
 background-size: 54%;
 }}

/* Small Devices, Tablets ------------ */  @media only screen and (max-width : 768px) {
.journal-genetics {
  background: #fff url('https://3718aeafc638f96f5bd6-d4a9ca15fc46ba40e71f94dec0aad28c.ssl.cf1.rackcdn.com/journal-genetics.png') no-repeat 140% 70px;
 background-size: 74%;
 }}

/* Small Devices, Tablets  --------- */ @media only screen and (max-width : 640px) {
.journal-genetics {
  background: #fff url('https://3718aeafc638f96f5bd6-d4a9ca15fc46ba40e71f94dec0aad28c.ssl.cf1.rackcdn.com/journal-genetics.png') no-repeat 250px 0px;
 }}

/* Extra Small Devices, Phones-----------  */  @media only screen and (max-width : 480px) {
.journal-genetics {
  background: #fff url('https://3718aeafc638f96f5bd6-d4a9ca15fc46ba40e71f94dec0aad28c.ssl.cf1.rackcdn.com/journal-genetics.png') no-repeat 225px 150px;
  background-size: 51%; 
 }}

/* Custom, iPhone Retina  -------*/  @media only screen and (max-width : 320px) {
.journal-genetics {
  background: #fff url('https://3718aeafc638f96f5bd6-d4a9ca15fc46ba40e71f94dec0aad28c.ssl.cf1.rackcdn.com/journal-genetics.png') no-repeat 165px 170px;
  background-size: 51%; 
 }} </style>
    
<style type="text/css">.container-cookies { position: fixed; overflow: hidden; box-sizing: border-box; z-index: 9999; background: #fafafa; left: 0; right: 0; bottom: 0; border-top: 1px solid #e0e0e0; -moz-box-shadow: 0px -3px 5px 0px rgba(0, 0, 0, 0.13); -webkit-box-shadow: 0px -3px 5px 0px rgba(0, 0, 0, 0.13); box-shadow: 0px -3px 5px 0px rgba(0, 0, 0, 0.13); font-family: "MuseoSans", Helvetica, Arial, sans-serif; padding: 10px 20px; display: none; }
.container-cookies.loop .message-cookies a { color: #f29400; }
.container-cookies.loop .close-cookies { background: #f29400; }
.container-cookies.frontiers .message-cookies a { color: #00a0d2; }
.container-cookies.frontiers .close-cookies { background: #00a0d2; }
.container-cookies .message-cookies { float: left; font-size: 15px; line-height: 20px; color: #555555; font-weight: 300; width: calc(100% - 75px); margin-top: 4px; }
.container-cookies .message-cookies a { font-weight: 700; text-decoration: underline; }
.container-cookies .close-cookies { height: 30px; width: 75px; color: white; text-align: center; text-transform: uppercase; line-height: 32px; font-weight: 700; cursor: pointer; font-size: 11px; float: right; }

@media screen and (max-width: 767px) { .container-cookies .message-cookies { width: 100%; display: block; clear: both; }
  .container-cookies .close-cookies { margin-top: 20px; width: 100%; display: block; clear: both; } }
</style><script id="altmetric-embed-js" src="Frontiers%20|%20Correcting%20Principal%20Component%20Maps%20for%20Effects%20of%20Spatial%20Autocorrelation%20in%20Population%20Genetic%20Data%20|%20Genetics_fichiers/altmetric_badges-06268e9dbd9b8916d66d0d0f4a2ce5de.js"></script><link href="Frontiers%20|%20Correcting%20Principal%20Component%20Maps%20for%20Effects%20of%20Spatial%20Autocorrelation%20in%20Population%20Genetic%20Data%20|%20Genetics_fichiers/font.css" rel="stylesheet"><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><style type="text/css">.MathJax_Preview .MJXf-math {color: inherit!important}
</style><style type="text/css">.MJX_Assistive_MathML {position: absolute!important; top: 0; left: 0; clip: rect(1px, 1px, 1px, 1px); padding: 1px 0 0 0!important; border: 0!important; height: 1px!important; width: 1px!important; overflow: hidden!important; display: block!important; -webkit-touch-callout: none; -webkit-user-select: none; -khtml-user-select: none; -moz-user-select: none; -ms-user-select: none; user-select: none}
.MJX_Assistive_MathML.MJX_Assistive_MathML_Block {width: 100%!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MJXp-script {font-size: .8em}
.MJXp-right {-webkit-transform-origin: right; -moz-transform-origin: right; -ms-transform-origin: right; -o-transform-origin: right; transform-origin: right}
.MJXp-bold {font-weight: bold}
.MJXp-italic {font-style: italic}
.MJXp-scr {font-family: MathJax_Script,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-frak {font-family: MathJax_Fraktur,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-sf {font-family: MathJax_SansSerif,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-cal {font-family: MathJax_Caligraphic,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-mono {font-family: MathJax_Typewriter,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-largeop {font-size: 150%}
.MJXp-largeop.MJXp-int {vertical-align: -.2em}
.MJXp-math {display: inline-block; line-height: 1.2; text-indent: 0; font-family: 'Times New Roman',Times,STIXGeneral,serif; white-space: nowrap; border-collapse: collapse}
.MJXp-display {display: block; text-align: center; margin: 1em 0}
.MJXp-math span {display: inline-block}
.MJXp-box {display: block!important; text-align: center}
.MJXp-box:after {content: " "}
.MJXp-rule {display: block!important; margin-top: .1em}
.MJXp-char {display: block!important}
.MJXp-mo {margin: 0 .15em}
.MJXp-mfrac {margin: 0 .125em; vertical-align: .25em}
.MJXp-denom {display: inline-table!important; width: 100%}
.MJXp-denom > * {display: table-row!important}
.MJXp-surd {vertical-align: top}
.MJXp-surd > * {display: block!important}
.MJXp-script-box > *  {display: table!important; height: 50%}
.MJXp-script-box > * > * {display: table-cell!important; vertical-align: top}
.MJXp-script-box > *:last-child > * {vertical-align: bottom}
.MJXp-script-box > * > * > * {display: block!important}
.MJXp-mphantom {visibility: hidden}
.MJXp-munderover {display: inline-table!important}
.MJXp-over {display: inline-block!important; text-align: center}
.MJXp-over > * {display: block!important}
.MJXp-munderover > * {display: table-row!important}
.MJXp-mtable {vertical-align: .25em; margin: 0 .125em}
.MJXp-mtable > * {display: inline-table!important; vertical-align: middle}
.MJXp-mtr {display: table-row!important}
.MJXp-mtd {display: table-cell!important; text-align: center; padding: .5em 0 0 .5em}
.MJXp-mtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-mlabeledtr {display: table-row!important}
.MJXp-mlabeledtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mlabeledtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-merror {background-color: #FFFF88; color: #CC0000; border: 1px solid #CC0000; padding: 1px 3px; font-style: normal; font-size: 90%}
.MJXp-scale0 {-webkit-transform: scaleX(.0); -moz-transform: scaleX(.0); -ms-transform: scaleX(.0); -o-transform: scaleX(.0); transform: scaleX(.0)}
.MJXp-scale1 {-webkit-transform: scaleX(.1); -moz-transform: scaleX(.1); -ms-transform: scaleX(.1); -o-transform: scaleX(.1); transform: scaleX(.1)}
.MJXp-scale2 {-webkit-transform: scaleX(.2); -moz-transform: scaleX(.2); -ms-transform: scaleX(.2); -o-transform: scaleX(.2); transform: scaleX(.2)}
.MJXp-scale3 {-webkit-transform: scaleX(.3); -moz-transform: scaleX(.3); -ms-transform: scaleX(.3); -o-transform: scaleX(.3); transform: scaleX(.3)}
.MJXp-scale4 {-webkit-transform: scaleX(.4); -moz-transform: scaleX(.4); -ms-transform: scaleX(.4); -o-transform: scaleX(.4); transform: scaleX(.4)}
.MJXp-scale5 {-webkit-transform: scaleX(.5); -moz-transform: scaleX(.5); -ms-transform: scaleX(.5); -o-transform: scaleX(.5); transform: scaleX(.5)}
.MJXp-scale6 {-webkit-transform: scaleX(.6); -moz-transform: scaleX(.6); -ms-transform: scaleX(.6); -o-transform: scaleX(.6); transform: scaleX(.6)}
.MJXp-scale7 {-webkit-transform: scaleX(.7); -moz-transform: scaleX(.7); -ms-transform: scaleX(.7); -o-transform: scaleX(.7); transform: scaleX(.7)}
.MJXp-scale8 {-webkit-transform: scaleX(.8); -moz-transform: scaleX(.8); -ms-transform: scaleX(.8); -o-transform: scaleX(.8); transform: scaleX(.8)}
.MJXp-scale9 {-webkit-transform: scaleX(.9); -moz-transform: scaleX(.9); -ms-transform: scaleX(.9); -o-transform: scaleX(.9); transform: scaleX(.9)}
.MathJax_PHTML .noError {vertical-align: ; font-size: 90%; text-align: left; color: black; padding: 1px 3px; border: 1px solid}
</style><link id="altmetric-embed-css" rel="stylesheet" type="text/css" href="Frontiers%20|%20Correcting%20Principal%20Component%20Maps%20for%20Effects%20of%20Spatial%20Autocorrelation%20in%20Population%20Genetic%20Data%20|%20Genetics_fichiers/embed-531871f0e7b8e764e338c9a38afd5621.css"><style type="text/css">.MathJax_Display {text-align: center; margin: 1em 0em; position: relative; display: block!important; text-indent: 0; max-width: none; max-height: none; min-width: 0; min-height: 0; width: 100%}
.MathJax .merror {background-color: #FFFF88; color: #CC0000; border: 1px solid #CC0000; padding: 1px 3px; font-style: normal; font-size: 90%}
.MathJax .MJX-monospace {font-family: monospace}
.MathJax .MJX-sans-serif {font-family: sans-serif}
#MathJax_Tooltip {background-color: InfoBackground; color: InfoText; border: 1px solid black; box-shadow: 2px 2px 5px #AAAAAA; -webkit-box-shadow: 2px 2px 5px #AAAAAA; -moz-box-shadow: 2px 2px 5px #AAAAAA; -khtml-box-shadow: 2px 2px 5px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true'); padding: 3px 4px; z-index: 401; position: absolute; left: 0; top: 0; width: auto; height: auto; display: none}
.MathJax {display: inline; font-style: normal; font-weight: normal; line-height: normal; font-size: 100%; font-size-adjust: none; text-indent: 0; text-align: left; text-transform: none; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; direction: ltr; max-width: none; max-height: none; min-width: 0; min-height: 0; border: 0; padding: 0; margin: 0}
.MathJax:focus, body :focus .MathJax {display: inline-table}
.MathJax.MathJax_FullWidth {text-align: center; display: table-cell!important; width: 10000em!important}
.MathJax img, .MathJax nobr, .MathJax a {border: 0; padding: 0; margin: 0; max-width: none; max-height: none; min-width: 0; min-height: 0; vertical-align: 0; line-height: normal; text-decoration: none}
img.MathJax_strut {border: 0!important; padding: 0!important; margin: 0!important; vertical-align: 0!important}
.MathJax span {display: inline; position: static; border: 0; padding: 0; margin: 0; vertical-align: 0; line-height: normal; text-decoration: none}
.MathJax nobr {white-space: nowrap!important}
.MathJax img {display: inline!important; float: none!important}
.MathJax * {transition: none; -webkit-transition: none; -moz-transition: none; -ms-transition: none; -o-transition: none}
.MathJax_Processing {visibility: hidden; position: fixed; width: 0; height: 0; overflow: hidden}
.MathJax_Processed {display: none!important}
.MathJax_ExBox {display: block!important; overflow: hidden; width: 1px; height: 60ex; min-height: 0; max-height: none}
.MathJax .MathJax_EmBox {display: block!important; overflow: hidden; width: 1px; height: 60em; min-height: 0; max-height: none}
.MathJax_LineBox {display: table!important}
.MathJax_LineBox span {display: table-cell!important; width: 10000em!important; min-width: 0; max-width: none; padding: 0; border: 0; margin: 0}
.MathJax .MathJax_HitBox {cursor: text; background: white; opacity: 0; filter: alpha(opacity=0)}
.MathJax .MathJax_HitBox * {filter: none; opacity: 1; background: transparent}
#MathJax_Tooltip * {filter: none; opacity: 1; background: transparent}
@font-face {font-family: MathJax_Blank; src: url('about:blank')}
.MathJax .noError {vertical-align: ; font-size: 90%; text-align: left; color: black; padding: 1px 3px; border: 1px solid}
</style><script type="text/javascript" charset="UTF-8" src="Frontiers%20|%20Correcting%20Principal%20Component%20Maps%20for%20Effects%20of%20Spatial%20Autocorrelation%20in%20Population%20Genetic%20Data%20|%20Genetics_fichiers/common.js"></script><script type="text/javascript" charset="UTF-8" src="Frontiers%20|%20Correcting%20Principal%20Component%20Maps%20for%20Effects%20of%20Spatial%20Autocorrelation%20in%20Population%20Genetic%20Data%20|%20Genetics_fichiers/util.js"></script></head>
<body class="journal-genetics section-applied-genetic-epidemiology v2 ibar-frontiers"><div style="visibility: hidden; overflow: hidden; position: absolute; top: 0px; height: 1px; width: auto; padding: 0px; border: 0px none; margin: 0px; text-align: left; text-indent: 0px; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal;"><div id="MathJax_Hidden"></div></div><div id="MathJax_Message" style="display: none;"></div>
    <!-- Google Tag Manager (noscript) -->
    <noscript>
        <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-TZGWTGG&gtm_auth=fCzlsmYQFAUQ_UXneGJNag&gtm_preview=env-2&gtm_cookies_win=x"
                height="0" width="0" style="display:none;visibility:hidden"></iframe>
    </noscript>
    <!-- End Google Tag Manager (noscript) -->


    <link href="Frontiers%20|%20Correcting%20Principal%20Component%20Maps%20for%20Effects%20of%20Spatial%20Autocorrelation%20in%20Population%20Genetic%20Data%20|%20Genetics_fichiers/frontiers.css" rel="stylesheet"><div class="header-compact v2 ibar-frontiers">
    <div class="navbar navbar-fixed-top">
        <div class="row container">
            <div class="navbar-header">
                <div class="col-xs-6 header-left-container">
                    <ul class="navbar-toggle inline-list navbar-mobile" data-toggle="collapse" data-target="#">
    <li class="no-left-margin">
        <a class="brand home" href="https://www.frontiersin.org/">
            <img class="header-logo" src="Frontiers%20|%20Correcting%20Principal%20Component%20Maps%20for%20Effects%20of%20Spatial%20Autocorrelation%20in%20Population%20Genetic%20Data%20|%20Genetics_fichiers/thin-header-logo.png">
        </a>
    </li>
    <li class="clickable no-left-margin">
        <div class="journal-dropdown-responsive" ssd-id="2">
            <span class="sr-only">Toggle navigation</span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
        </div><div id="ssd-dropdown-2" class="ssd-dropdown" style="margin-left: -9px; width: 200px; top: -11px; height: auto; display: none;"><div id="ssd-container-2" class="ssd-container" style="margin-left: 0px;"><table><tbody><tr id="ssd-dropdown-2-tr-1"><td><div class="ssd-dropdown_container ssd-dropdown_container_small" id="sdd-dropdown-2-container-level-1" style="height: auto;"><ul class="ssd-menu-list ssd-dropdown_ul"><li><a class="sdd-link" href="https://www.frontiersin.org/" data-header-tracking="{&quot;category&quot;:&quot;Header_Action&quot;,&quot;action&quot;:&quot;link&quot;, &quot;label&quot;: &quot;click_header_home&quot; }" data-test-id="home">Home</a></li><li><a class="sdd-link" href="https://www.frontiersin.org/AboutFrontiers.aspx" data-header-tracking="{&quot;category&quot;:&quot;Header_Action&quot;,&quot;action&quot;:&quot;link&quot;, &quot;label&quot;: &quot;click_header_about&quot; }" data-test-id="about">About</a></li><li data-test-id="journals"><img src="Frontiers%20|%20Correcting%20Principal%20Component%20Maps%20for%20Effects%20of%20Spatial%20Autocorrelation%20in%20Population%20Genetic%20Data%20|%20Genetics_fichiers/nav-arrow-right-v2.png">Journals</li><li><a class="sdd-link" href="https://www.frontiersin.org/researchtopic" data-header-tracking="{ &quot;category&quot;: &quot;Header_Action&quot;, &quot;action&quot;: &quot;link&quot;, &quot;label&quot;: &quot;click_header_rt&quot; }" data-test-id="research_topics">Research Topics</a></li><li><a class="sdd-link" href="https://www.frontiersin.org/Submission/SubmissionInfo.aspx" data-header-tracking="{&quot;category&quot;:&quot;Header_Action&quot;,&quot;action&quot;:&quot;link&quot;, &quot;label&quot;: &quot;click_header_submit&quot; }" data-test-id="submit">Submit</a></li></ul></div></td></tr></tbody></table></div></div><div id="ssd-dropdown-indicator-2" class="ssd-dropdown-indicator" style="top: 0px; left: 5px; display: none;"></div>
    </li>
</ul>

<ul class="inline-list navbar-collapse collapse navbar-desktop">
    <li class="header-inline-element no-left-margin">
        <a class="brand home" href="https://www.frontiersin.org/" onclick="FRHeader.trackOutboundLink('Header_Action', 'link', 'click_header_home', 'https://www.frontiersin.org');return false;">
            <img class="header-logo" src="Frontiers%20|%20Correcting%20Principal%20Component%20Maps%20for%20Effects%20of%20Spatial%20Autocorrelation%20in%20Population%20Genetic%20Data%20|%20Genetics_fichiers/thin-header-logo.png">
        </a>
    </li>
    <li class="no-left-margin">
        <a class="home" href="https://www.frontiersin.org/" onclick="FRHeader.trackOutboundLink('Header_Action', 'link', 'click_header_home', 'https://www.frontiersin.org');return false;">
            Home
        </a>
    </li>
    <li class="no-left-margin">
        <a class="about" href="https://www.frontiersin.org/AboutFrontiers.aspx" onclick="FRHeader.trackOutboundLink('Header_Action', 'link', 'click_header_about', 'https://www.frontiersin.org/AboutFrontiers.aspx');return false;">
            About
        </a>
    </li>
    <li class="submit-container">
        <a class="submit" href="https://www.frontiersin.org/Submission/SubmissionInfo.aspx" onclick="FRHeader.trackOutboundLink('Header_Action', 'link', 'click_header_submit', 'https://www.frontiersin.org/Submission/SubmissionInfo.aspx');return false;">
            Submit
        </a>
    </li>
    <li class="no-padding-right journal-drop highlight">
        <a class="byjournal bysubjects" data-test-id="journal-bydropdown" ssd-id="1">
            <span>Journals</span><img class="drop-down-arrow" src="Frontiers%20|%20Correcting%20Principal%20Component%20Maps%20for%20Effects%20of%20Spatial%20Autocorrelation%20in%20Population%20Genetic%20Data%20|%20Genetics_fichiers/header-arrow-down.png" alt="">
        </a><div id="ssd-dropdown-1" class="ssd-dropdown" style="margin-left: -9px; width: 200px; top: 56px; height: auto; display: none;"><div id="ssd-container-1" class="ssd-container" style="margin-left: 0px;"><table><tbody><tr id="ssd-dropdown-1-tr-1"><td><div class="ssd-dropdown_container ssd-dropdown_container_small" id="sdd-dropdown-1-container-level-1" style="height: auto;"><ul class="ssd-menu-list ssd-dropdown_ul"><li><a class="sdd-link" href="https://www.frontiersin.org/AboutFrontiers.aspx?stage=journalseries" data-test-id="journals_a-z">Journals A-Z</a></li><li class="ssd-label-li"><span class="ssd-label">By Subject</span></li><li data-test-id="science"><img src="Frontiers%20|%20Correcting%20Principal%20Component%20Maps%20for%20Effects%20of%20Spatial%20Autocorrelation%20in%20Population%20Genetic%20Data%20|%20Genetics_fichiers/nav-arrow-right-v2.png">Science</li><li data-test-id="health"><img src="Frontiers%20|%20Correcting%20Principal%20Component%20Maps%20for%20Effects%20of%20Spatial%20Autocorrelation%20in%20Population%20Genetic%20Data%20|%20Genetics_fichiers/nav-arrow-right-v2.png">Health</li><li data-test-id="engineering"><img src="Frontiers%20|%20Correcting%20Principal%20Component%20Maps%20for%20Effects%20of%20Spatial%20Autocorrelation%20in%20Population%20Genetic%20Data%20|%20Genetics_fichiers/nav-arrow-right-v2.png">Engineering</li><li data-test-id="humanities_and_social_sciences"><img src="Frontiers%20|%20Correcting%20Principal%20Component%20Maps%20for%20Effects%20of%20Spatial%20Autocorrelation%20in%20Population%20Genetic%20Data%20|%20Genetics_fichiers/nav-arrow-right-v2.png">Humanities and Social Sciences</li><li data-test-id="young_minds_"><img src="Frontiers%20|%20Correcting%20Principal%20Component%20Maps%20for%20Effects%20of%20Spatial%20Autocorrelation%20in%20Population%20Genetic%20Data%20|%20Genetics_fichiers/nav-arrow-right-v2.png">Young Minds </li><li data-test-id="sustainability"><img src="Frontiers%20|%20Correcting%20Principal%20Component%20Maps%20for%20Effects%20of%20Spatial%20Autocorrelation%20in%20Population%20Genetic%20Data%20|%20Genetics_fichiers/nav-arrow-right-v2.png">Sustainability</li></ul></div></td></tr></tbody></table></div></div><div id="ssd-dropdown-indicator-1" class="ssd-dropdown-indicator" style="top: 51px; left: 65px; display: none;"></div>
    </li>
    <li class="journalAZ hidden highlight">
        <a class="bysubjects" href="https://www.frontiersin.org/AboutFrontiers.aspx?stage=journalseries" data-test-id="journal-atoz">
            Journals A-Z
        </a>
    </li>
    
    <li class="research-topic-container">
        <a class="research-topic" href="https://www.frontiersin.org/research-topics" onclick="FRHeader.trackOutboundLink('Header_Action', 'link', 'click_header_rt', 'https://www.frontiersin.org/research-topics');return false;">
            Research Topics
        </a>
    </li>
    
</ul>

                </div>
                <div class="header-search">
                    <div class="search-swappable hidden-xs">
    <a class="search-icon-container" data-test-id="search-icon">
        <img class="search-icon" src="Frontiers%20|%20Correcting%20Principal%20Component%20Maps%20for%20Effects%20of%20Spatial%20Autocorrelation%20in%20Population%20Genetic%20Data%20|%20Genetics_fichiers/header-search.png">
    </a>
    <form class="search-bar-container hide" style="display: none;">
        <input data-test-id="search-field" placeholder="Search for articles, people, events and more." class="header-search-field">
        <input data-test-id="search-btn" class="header-search-btn" value="" style="left: 321px;" type="button">
    </form>
</div>

                </div>
                <div class="col-xs-6 pull-right unlogged-user">


    <ul class="inline-list pull-right">
        <li>
            <a class="login-container" data-test-id="login-link" href="https://www.frontiersin.org/people/login" data-header-tracking="{ &quot;category&quot;: &quot;Header_Action&quot;, &quot;action&quot;: &quot;link&quot;, &quot;label&quot;: &quot;click_header_login&quot; }">
                <span>Login</span>
            </a>
        </li>
        <li>
            <a class="register-container" data-test-id="register-link" href="https://www.frontiersin.org/register">
                <span>Register</span>
            </a>
        </li>
    </ul>

<div class="popover-wrapper popover-login" style="display: none">
    <div class="popover-menu login-popover">
        <form action="https://www.frontiersin.org/Login.aspx" name="login-form" method="POST" class="login-form">
            <div class="third-party-signin-row row-inside-column">
                <h1>Login using</h1>
                <div class="third-party-buttons">
                    <div class="third-party-button-wrapper" id="third-party-btn-for-linkedin">
                        <a data-type="3">
                            <div class="radius">
                                <div class="third-party-icon linkedin-icon-wrapper">
                                    <div class="linkedin-icon"></div>
                                </div>
                            </div>
                            <div class="third-party-name">LinkedIn</div>
                        </a>
                    </div><div class="third-party-button-wrapper" id="third-party-btn-for-twitter">
                        <a data-type="2">
                            <div class="radius">
                                <div class="third-party-icon twitter-icon-wrapper">
                                    <div class="twitter-icon"></div>
                                </div>
                            </div>
                            <div class="third-party-name">Twitter</div>
                        </a>
                    </div>
                    <div class="third-party-button-wrapper" id="third-party-btn-for-facebook">
                        <a data-type="1">
                            <div class="radius">
                                <div class="third-party-icon facebook-icon-wrapper">
                                    <div class="facebook-icon"></div>
                                </div>
                            </div>
                            <div class="third-party-name">Facebook</div>
                        </a>
                    </div>
                    <div class="info-button-wrapper" id="third-party-information-button">
                        <a class="popover-login-info-trigger">
                            <div class="info-button">i</div>
                        </a>
                        <div class="popover-wrapper popover-login-info" style="display: none">
                            <div class="popover-menu">
                                <h1 class="popover-title">You can login by using one of your existing accounts.</h1>
                                <span style="float: none; display: inline">
                                    We will be provided with an 
authorization token (please note: passwords are not shared with us) and 
will sync your accounts for you. This means that you will not need to 
remember your user name and password in the future and you will be able 
to login with the account you choose to sync, with the click of a 
button.
                                </span>
                            </div>
                        </div>
                    </div>
                </div>
            </div>
            <div class="clear"></div>
            <div class="or-row" id="or-circle-divider">
                <div class="or-divider or-divider-inside-column">
                    <div class="or-circle">OR</div>
                </div>
            </div>
            <div class="loginForm">
                <div class="form-field-label">Email *</div>
                <div class="form-field">
                    <input runat="server" tabindex="5" name="txtLoginEmail" type="text">
                    <div class="form-field-error-msg">
                        <div class="error-icon-small"></div>
                        Please fill in this field
                        <div class="clear"></div>
                    </div>
                </div>
                <div class="form-field-label">Password *</div>
                <div class="form-field">
                    <input runat="server" tabindex="6" name="txtLoginPassword" type="password">
                    <div class="form-field-error-msg">
                        <div class="error-icon-small"></div>
                        Please fill in this field
                        <div class="clear"></div>
                    </div>
                </div>
                <div id="rememberMeWrapper">
                    <div class="checkbox-remember-container">
                        <input runat="server" tabindex="7" name="chkLoginRememberMe" type="checkbox">
                    </div>
                    <div class="checkbox-remember-label">Remember me</div>
                    <div class="forgot-password">
                        <a href="https://registration.frontiersin.org/people/forgot-password">Forgot Password?</a>
                    </div>
                </div>
                <div class="login-page-submit" id="formSubmitWrapper">
                    <div>
                        <input name="hdnIsPopupLogin" runat="server" value="1" type="hidden">
                        <button type="submit" tabindex="8" class="btn btn-loginlogout-blue btn-flat input-block-level">Login</button>
                    </div>
                </div>
            </div>
        </form>
    </div>
</div>
                </div>
                <div class="col-xs-6 pull-right logged-user hidden">
                    <ul class="logged navbar-toggle inline-list pull-right navbar-mobile" data-toggle="collapse" data-target="#LefttHeader">

    <li>
        <div id="myfrontierssponsive" class="clickable profile-dropdown-responsive hidden">
            <a class="myfrontiersmobile">
                <p class="frontiers-letter">
                    Frontiers
                </p>
            </a>
        </div>
    </li>
    <li class="myofficecontainer">
        <div id="myoffice" class="clickable profile-dropdown-responsive">
            <p class="frontiers-letter">
                Frontiers
            </p>
            <em class="down-arrow-rhs"></em>
        </div>

        <div id="myofficeresponsive" class="clickable profile-dropdown-responsive hidden">
            <p class="frontiers-letter">
                Office
            </p>
            <em class="down-arrow-rhs"></em>
        </div>
    </li>
    <li class="no-left-margin profile-container-responsive">
        <a href="javascript:void(0)">
            <img class="profile-pic responsive lo_sensitive">
        </a>
    </li>

</ul>

<ul class="logged inline-list pull-right navbar-collapse collapse navbar-desktop">
    <li class="myfrontiers-container">
        <a>
            <span>My frontiers</span>
            <em></em>
        </a>
    </li>

    <li class="myfrontiersbeta-container hidden">
        <a class="myfrontiersbeta">
            <span>My Frontiers</span>
        </a>
    </li>

    <li class="myoffice-container hidden">
        <a>
            <span>Office</span>
            <em></em>
        </a>
    </li>

    <li class="myhome-container">
        <a class="myhome" title="" data-placement="left" data-header-tracking="{ &quot;category&quot;: &quot;Header_Action&quot;, &quot;action&quot;: &quot;link&quot;, &quot;label&quot;: &quot;click_header_loop&quot; }" data-original-title="My Home">
            <i class="fromSprite loop-logo"></i>
        </a>
    </li>
    <li class="no-left-margin impact-container">
        <a class="popover-impact-disabled" title="" data-placement="left" data-header-tracking="{ &quot;category&quot;: &quot;Header_Action&quot;, &quot;action&quot;: &quot;link&quot;, &quot;label&quot;: &quot;click_header_impact&quot; }" data-original-title="My Impact">
            <i class="fromSprite impactLogo"></i>
        </a>
    </li>
    
    <li class="no-left-margin username-container">
        <a class="username lo_sensitive" data-header-tracking="{ &quot;category&quot;: &quot;Header_Action&quot;, &quot;action&quot;: &quot;link&quot;, &quot;label&quot;: &quot;click_header_user&quot; }"></a>
    </li>
    <li class="no-left-margin profile-container">
        <a class="popover-profile-trigger">
            <img class="profile-pic desktop lo_sensitive">
        </a>
    </li>
</ul>

<div class="popover-wrapper popover-impact" style="display: none">
    <em></em>
    <div class="popover-menu impact-popover"></div>
</div>

<div class="popover-wrapper popover-notification" style="display: none">
    <em></em>
    <input id="hdnLastNotificationId" type="hidden">
    <div class="popover-menu notifications-popover"></div>
</div>

                </div>
            </div>
        </div>
    </div>
</div>





    

    <!--[if lte IE 8]>
        <div class="row">
            <div class="col-md-16">
                <div style="margin-bottom: 15px;"><h2>Warning!</h2><p>You are using an <strong>outdated</strong> browser. This page doesn't support Internet Explorer 6, 7 and 8.<br />Please <a class="blue" href="http://browsehappy.com/">upgrade your browser</a> or <a class="blue" href="http://www.google.com/chromeframe/?redirect=true">activate Google Chrome Frame</a> to improve your experience.</p></div>
            </div>
        </div>
    <![endif]-->

    <div class="page-container">
        

<header>

            <a href="https://blog.frontiersin.org/2018/07/06/journal-impact-factor-frontiers-in-genetics/?utm_source=FWEB&amp;utm_medium=FJOUR&amp;utm_campaign=IF18_JOURN_FGENE-TOP" target="_blank">
            <div class="impact-metric-marketing animated slideInRight">
                
                <div class="pull-right impact-metric-impact-factor text-center ">
                    <span class="impact-metric-impact-label">Impact Factor</span>
                    <span class="impact-metric-impact-number impact-numerical-padding">4.151</span>
                    <span class="clearfix"></span>
                    <div class="impact-from">2017 JCR, Clarivate Analytics 2018</div>
                </div>


                <span class="impact-metric-message">
                    <p><span style="font-family: Calibri, 'Segoe UI', Calibri, Thonburi, Arial, Verdana, sans-serif, 'Mongolian Baiti', 'Microsoft Yi Baiti', 'Javanese Text'; font-size: 13.3333px; white-space: pre-wrap;">Frontiers journals are at the </span><strong style="font-family: Calibri, 'Segoe UI', Calibri, Thonburi, Arial, Verdana, sans-serif, 'Mongolian Baiti', 'Microsoft Yi Baiti', 'Javanese Text'; font-size: 13.3333px; white-space: pre-wrap;">top of citation and impact metrics</strong></p>
                </span>

            </div>
        </a>
<div id="journal-header" class="journal-home" style="position: relative;">
    <div class="container-fluid main-container-xxl">
        <table id="logo-table" style="width:100%">
            <tbody><tr>
                    <td class="left-cell" width="50%" valign="bottom" align="right">
                        <h1>
                            <a href="https://www.frontiersin.org/journals/240" data-test-id="journal-link">
                                <span>Frontiers </span>in Genetics
                            </a>
                        </h1>
                    </td>
                        <td class="right-cell" style="vertical-align: bottom" width="50%">
                            <h2>
                                <a href="https://www.frontiersin.org/journals/240/sections/549" data-test-id="section-link" class="btn btn-link">
                                    Applied Genetic Epidemiology


                                </a>
                            </h2>
                        </td>
            </tr>
        </tbody></table>
    </div>
</div>


    <div id="journal-nav">
    <div class="container-fluid main-container-xxl">
        <div class="row">
            <div class="col-md-12">
                <div class="nav-wrapper">
                    <nav class="navbar navbar-default">
                        <!-- Brand and toggle get grouped for better mobile display -->
                        <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#journal-navbar-collapse">
                            <span class="sr-only">Toggle navigation</span>
                            <span class="pull-left">
                                <span class="icon-bar"></span>
                                <span class="icon-bar"></span>
                                <span class="icon-bar"></span>
                            </span>
                            <span class="pull-right hidden-xs">Section</span>
                        </button>
                        <!-- Collect the nav links, forms, and other content for toggling -->
                        <div class="collapse navbar-collapse no-transition journal-nav-bar" id="journal-navbar-collapse">
                            <ul class="nav navbar-nav">
                                <li class="tab-overview icon icon-left">
                                    <a href="https://www.frontiersin.org/journals/genetics/sections/applied-genetic-epidemiology#"><span class="sr-only">(current)</span><i class="fa fa-home"></i>Section</a>
                                </li>
                                <li class="tab-about">
                                    <a href="https://www.frontiersin.org/journals/genetics/sections/applied-genetic-epidemiology#about">About</a>
                                </li>
                                <li class="tab-archive active">
                                    <a href="https://www.frontiersin.org/journals/genetics/sections/applied-genetic-epidemiology#articles">Articles</a>
                                </li>
                                <li class="tab-research-topics">
                                    <a href="https://www.frontiersin.org/journals/genetics/sections/applied-genetic-epidemiology#research-topics">Research topics</a>
                                </li>
                                <li class="dropdown">
                                    <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">For authors <span class="caret"></span></a>
                                    <ul class="dropdown-menu" role="menu">
                                        <li><a href="https://www.frontiersin.org/journals/genetics/sections/applied-genetic-epidemiology#why-submit">Why submit?</a></li>
                                        <li><a href="https://www.frontiersin.org/about/publishing-fees" target="_blank">Fees</a></li>
                                        <li><a href="https://www.frontiersin.org/journals/genetics/sections/applied-genetic-epidemiology#article-types">Article types</a></li>
                                        <li><a href="https://www.frontiersin.org/journals/genetics/sections/applied-genetic-epidemiology#author-guidelines">Author guidelines</a></li>
                                        <li><a href="https://www.frontiersin.org/journals/genetics/sections/applied-genetic-epidemiology#review-guidelines">Review guidelines</a></li>
                                        <li><a href="https://www.frontiersin.org/journals/genetics/sections/applied-genetic-epidemiology#submission-checklist">Submission checklist</a></li>
                                        <li><a href="https://www.frontiersin.org/journals/genetics/sections/applied-genetic-epidemiology#contact-editorial-office">Contact editorial office</a></li>
                                            <li><a href="https://www.frontiersin.org/submission/submissioninfo.aspx" class="action-link">Submit your manuscript</a></li>
                                    </ul>
                                </li>
                                <li class="tab-editorial-board">
                                    <a href="https://www.frontiersin.org/journals/genetics/sections/applied-genetic-epidemiology#editorial-board">Editorial board</a>
                                </li>
                            </ul>
                        </div><!-- /.navbar-collapse -->
                        <ul class="list-inline follow-us nav navbar-nav">
                            <li class="twitter icon button">
                                <a data-test-id="twitter-link" href="https://twitter.com/@FrontGenetics">
                                    <i class="fa fa-twitter"></i>
                                </a>
                            </li>
                            <li class="rss icon button">
                                <a data-test-id="rss-link" href="https://www.frontiersin.org/journals/genetics/sections/applied-genetic-epidemiology/rss">
                                    <i class="fa fa-rss"></i>
                                </a>
                            </li>
                            <li class="alerts icon button">
                                <a data-test-id="alert-link" href="http://connect.frontiersin.org/subscriptions/subscribe?item=1&amp;field=30">
                                    <i class="fa fa-bell" style="width: auto; padding-right: 7px; padding-left: 7px;"><span class="article-alert-text" style="font-family: MuseoSans, 'Helvetica Neue', Helvetica, Arial, sans-serif; margin-left: 6px; font-weight: 700; letter-spacing: 0px; font-size: 13px;">Article alerts</span></i>
                                </a>
                            </li>
                                    
                            
                        </ul>

                    </nav>
                </div><!-- / .nav-wrapper -->
            </div>
        </div>
    </div>
</div>

</header>

<div class="journal-actions article-actions">
    <div class="container-fluid main-container-xxl">
        <div class="row">
            <div class="col-lg-6 col-md-6 col-lg-push-6 col-md-push-6 col-research-topic">
            </div>
            <section class="col-lg-4 col-lg-offset-2 col-md-6 col-md-offset-0 col-lg-pull-6 col-md-pull-6">
                <a href="https://www.frontiersin.org/journals/240/sections/549#articles" class="archive-link hidden-sm hidden-xs" data-test-id="archive-link">
                    <i class="fa fa-angle-left"></i> Articles
                </a>
            </section>
        </div>
    </div>
</div>


<aside id="anchors" class="pull-left table-of-contents side-article is_stuck" style="position: fixed; top: 0px; width: 267px;">
    <div class="side-people hidden-sm hidden-xs">
    </div>
    <nav>
            <header><h5 class="like-h4" style="padding-top: 14px; padding-left: 6px; margin-bottom: 6px;">TABLE OF CONTENTS</h5></header>
            <ul class="nav nav-list list-unstyled contents">
                <li>
<ul class="flyoutJournal">
<li><a href="#h1">Abstract</a></li>
<li><a href="#h2">Introduction</a></li>
<li><a href="#h3">Materials and Methods</a></li>
<li><a href="#h4">Results</a></li>
<li><a href="#h5">Discussion</a></li>
<li><a href="#h6">Conclusion</a></li>
<li><a href="#h7">Conflict of Interest Statement</a></li>
<li><a href="#h8">Acknowledgments</a></li>
<li><a href="#h9">Footnotes</a></li>
<li><a href="#h10">References</a></li>
</ul>                </li>
            </ul>
    </nav>
</aside><div style="position: fixed; width: 362.4px; height: 596px; display: block; vertical-align: baseline; float: none;"></div>


<div id="article" class="boxed white no-padding">
    <div class="container-fluid main-container-xxl">
            <div class="side-article-mrk hidden-md hidden-lg top clearfix">
                <style type="text/css"><!--
.spotlight-title:hover,
.spotlight-title:focus {
 color: #555 !important; 
}
.spotlight-link {
 color: #f2b12f !important;
}
.spotlight-link:hover,
.spotlight-link:focus {
 color: #e5a812 !important;
}
--></style>
<div class="spotlight"><a href="https://spotlight.frontiersin.org/how-to-enter/?utm_source=FWEB&amp;utm_medium=FMAIN&amp;utm_campaign=SPOTLIGHT&amp;utm_term=BANNER&amp;utm_content=WEB"><img src="Frontiers%20|%20Correcting%20Principal%20Component%20Maps%20for%20Effects%20of%20Spatial%20Autocorrelation%20in%20Population%20Genetic%20Data%20|%20Genetics_fichiers/article-marketing-message-1f3be380-af93-4b04-a218-32e22eb3bd.png" alt=""></a><br>
<h3 style="font-weight: 500;"><a class="spotlight-title" tabindex="0" href="https://spotlight.frontiersin.org/how-to-enter/?utm_source=FWEB&amp;utm_medium=FMAIN&amp;utm_campaign=SPOTLIGHT&amp;utm_term=BANNER&amp;utm_content=WEB" target="_blank">Want to win $100,000 to host your own conference?</a></h3>
<a class="spotlight-link" href="https://spotlight.frontiersin.org/how-to-enter/?utm_source=FWEB&amp;utm_medium=FMAIN&amp;utm_campaign=SPOTLIGHT&amp;utm_term=BANNER&amp;utm_content=WEB" target="_blank">Suggest a Research Topic</a></div>
            </div>
        <div class="row">

            
<aside class="a col-xs-12 col-sm-12 col-lg-2 col-md-3 pull-right side-article right-container">
        <button type="button" class="navbar-toggle navbar-download collapsed" data-toggle="collapse" data-target=".show" aria-expanded="false">
            <span class="icon"><i class="fa fa-download"></i></span>
        </button>
    <button type="button" class="navbar-toggle navbar-share collapsed" data-toggle="collapse" data-target=".show" aria-expanded="false">
        <span class="icon"><i class="fa fa-share-alt"></i></span>
    </button>
        <div class="side-article-download show collapse clearfix">
            <ul class="list-unstyled list-inline clearfix">
                    <li class="dropdown text-center paper">
                        <a data-target="#" class="dropdown-toggle" data-test-id="download-button" data-toggle="dropdown" role="button" aria-expanded="false">
                            <span class="icon icon-article"><i class="fa fa-file-pdf-o"></i></span>
                            <span class="icon-label">Download Article</span>
                        </a>
                        <ul class="dropdown-menu" role="menu">
                                <li>
                                    <a class="download-files-pdf action-link" href="https://www.frontiersin.org/articles/10.3389/fgene.2012.00254/pdf" data-test-id="article-downloadpdf">
                                        Download PDF
                                    </a>
                                </li>
                                <li>
                                    <a class="download-files-readcube" href="http://www.readcube.com/articles/10.3389/fgene.2012.00254" data-test-id="article-viewenhancedpdf">
                                        ReadCube
                                    </a>
                                </li>
                                <li>
                                    <a class="download-files-nlm" href="https://www.frontiersin.org/articles/10.3389/fgene.2012.00254/xml/nlm" data-test-id="article-nlm">
                                        XML (NLM)
                                    </a>
                                </li>
                                                            <li class="disable">

                                        <span class="supplemental-data-disabled">
                                            Supplementary
                                            <br>
                                            Material
                                        </span>

                                </li>
                        </ul>
                    </li>
                <li class="dropdown text-center citation">
                    <a data-target="#" class="dropdown-toggle" data-test-id="citation-button" data-toggle="dropdown" role="button" aria-expanded="false">
                        <span class="icon icon-citation"><i class="fa fa-quote-left"></i></span>
                        <span class="icon-label">Export citation</span>
                    </a>
                    <ul class="dropdown-menu" role="menu">
                            <li>
                                <a data-test-id="article-endnote" href="https://www.frontiersin.org/articles/10.3389/fgene.2012.00254/endNote">
                                    EndNote
                                </a>
                            </li>
                            <li>
                                <a data-test-id="article-referencemanager" href="https://www.frontiersin.org/articles/10.3389/fgene.2012.00254/reference">
                                    Reference Manager
                                </a>
                            </li>
                            <li>
                                <a data-test-id="article-simpletextfile" href="https://www.frontiersin.org/articles/10.3389/fgene.2012.00254/text">
                                    Simple TEXT file
                                </a>
                            </li>
                            <li>
                                <a data-test-id="article-bibtex" href="https://www.frontiersin.org/articles/10.3389/fgene.2012.00254/bibTex">
                                    BibTex
                                </a>
                            </li>
                    </ul>
                </li>
            </ul>
        </div>
    <div class="side-article-impact">
        <ul class="nav">
            <li class="impact-data" title="" data-original-title="The total view count is updated once a day, so not to worry if you don't see immediate results.">
                <span class="title-number">6,490</span>
                <span class="title-text">total views</span>
            </li>
            <li class="hidden-sm hidden-xs">
                <div class="altmetric-icon">
                    <div class="altmetric-embed" data-badge-type="1" data-doi="10.3389/fgene.2012.00254" data-link-target="new" data-uuid="ac135270-d9a3-b2ae-5615-6e92a5890a40"><a target="new" href="https://www.altmetric.com/details.php?domain=www.frontiersin.org&amp;citation_id=1078857" style="display:inline-block;">
        <img alt="Article has an altmetric score of 1" src="Frontiers%20|%20Correcting%20Principal%20Component%20Maps%20for%20Effects%20of%20Spatial%20Autocorrelation%20in%20Population%20Genetic%20Data%20|%20Genetics_fichiers/1.png" style="border:0; margin:0; max-width: none;" width="110px" height="20px">
</a></div>
                </div>
            </li>
        </ul>
            <a type="button" class="btn btn-default hidden-sm hidden-xs btn-impact " data-test-id="view-article-impact" href="http://loop-impact.frontiersin.org/impact/article/34644#views" target="_blank">
                <span class="icon-impact"><i class="fa fa-line-chart"></i></span> View Article Impact
            </a>            
    </div>
        <div class="side-article-mrk hidden-sm hidden-xs clearfix">
            <style type="text/css"><!--
.spotlight-title:hover,
.spotlight-title:focus {
 color: #555 !important; 
}
.spotlight-link {
 color: #f2b12f !important;
}
.spotlight-link:hover,
.spotlight-link:focus {
 color: #e5a812 !important;
}
--></style>
<div class="spotlight"><a href="https://spotlight.frontiersin.org/how-to-enter/?utm_source=FWEB&amp;utm_medium=FMAIN&amp;utm_campaign=SPOTLIGHT&amp;utm_term=BANNER&amp;utm_content=WEB"><img src="Frontiers%20|%20Correcting%20Principal%20Component%20Maps%20for%20Effects%20of%20Spatial%20Autocorrelation%20in%20Population%20Genetic%20Data%20|%20Genetics_fichiers/article-marketing-message-1f3be380-af93-4b04-a218-32e22eb3bd.png" alt=""></a><br>
<h3 style="font-weight: 500;"><a class="spotlight-title" tabindex="0" href="https://spotlight.frontiersin.org/how-to-enter/?utm_source=FWEB&amp;utm_medium=FMAIN&amp;utm_campaign=SPOTLIGHT&amp;utm_term=BANNER&amp;utm_content=WEB" target="_blank">Want to win $100,000 to host your own conference?</a></h3>
<a class="spotlight-link" href="https://spotlight.frontiersin.org/how-to-enter/?utm_source=FWEB&amp;utm_medium=FMAIN&amp;utm_campaign=SPOTLIGHT&amp;utm_term=BANNER&amp;utm_content=WEB" target="_blank">Suggest a Research Topic</a></div>
        </div>
    <div class="side-article-share show collapse">        
        <h5 class="like-h4">SHARE ON</h5>
        <ul class="share-media clearfix" style="padding: 0;">
            <li class="social_block">
                <div class="atButton">
                    <a class="addthis_button_facebook at300b" addthis:title="Correcting Principal Component Maps for Effects of Spatial Autocorrelation in Population Genetic Data" addthis:description="In many species, spatial genetic variation displays patterns of “isolation-by-distance”. Characterized by locally correlated allele frequencies, these patterns are known to create periodic shapes in geographic maps of principal components which confound signatures of specific migration events and influence interpretations of principal component analyses (PCA). In this study, we introduced models combining probabilistic PCA and kriging models to infer population genetic structure from genetic data while correcting for effects generated by spatial autocorrelation. The corresponding algorithms are based on singular value decomposition and low rank approximation of the genotypic data. As their complexity is close to that of PCA, these algorithms scale with the dimension of the data. To illustrate the utility of these new models, we simulated isolation-by-distance patterns and broad-scale geographic variation using spatial coalescent models. Our methods remove the horseshoe patterns usually observed in PC maps and simplify intrepretations of spatial genetic variation. We demonstrate our approach by analyzing single nucleotide polymorphism data from the Human Genome Diversity Panel, and provide comparisons with other recently introduced methods." addthis:url="https://www.frontiersin.org/article/10.3389/fgene.2012.00254" title="Facebook" href="#"><span class="at-icon-wrapper" style="background-color: rgb(59, 89, 152); line-height: 16px; height: 16px; width: 16px;"><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 32 32" version="1.1" role="img" aria-labelledby="at-svg-facebook-1" class="at-icon at-icon-facebook" style="width: 16px; height: 16px;" title="Facebook" alt="Facebook"><title id="at-svg-facebook-1">Facebook</title><g><path d="M22 5.16c-.406-.054-1.806-.16-3.43-.16-3.4 0-5.733 1.825-5.733 5.17v2.882H9v3.913h3.837V27h4.604V16.965h3.823l.587-3.913h-4.41v-2.5c0-1.123.347-1.903 2.198-1.903H22V5.16z" fill-rule="evenodd"></path></g></svg></span></a>
                </div>
                <a><span class="facebook_count">0</span></a>
            </li>
            <li class="social_block">
                <div class="atButton">
                    <a class="addthis_button_twitter at300b" addthis:title="Correcting Principal Component Maps for Effects of Spatial Autocorrelation in Population Genetic Data" addthis:description="In many species, spatial genetic variation displays patterns of “isolation-by-distance”. Characterized by locally correlated allele frequencies, these patterns are known to create periodic shapes in geographic maps of principal components which confound signatures of specific migration events and influence interpretations of principal component analyses (PCA). In this study, we introduced models combining probabilistic PCA and kriging models to infer population genetic structure from genetic data while correcting for effects generated by spatial autocorrelation. The corresponding algorithms are based on singular value decomposition and low rank approximation of the genotypic data. As their complexity is close to that of PCA, these algorithms scale with the dimension of the data. To illustrate the utility of these new models, we simulated isolation-by-distance patterns and broad-scale geographic variation using spatial coalescent models. Our methods remove the horseshoe patterns usually observed in PC maps and simplify intrepretations of spatial genetic variation. We demonstrate our approach by analyzing single nucleotide polymorphism data from the Human Genome Diversity Panel, and provide comparisons with other recently introduced methods." addthis:url="https://www.frontiersin.org/article/10.3389/fgene.2012.00254" title="Tweet" href="#"><span class="at-icon-wrapper" style="background-color: rgb(29, 161, 242); line-height: 16px; height: 16px; width: 16px;"><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 32 32" version="1.1" role="img" aria-labelledby="at-svg-twitter-2" class="at-icon at-icon-twitter" style="width: 16px; height: 16px;" title="Twitter" alt="Twitter"><title id="at-svg-twitter-2">Twitter</title><g><path d="M27.996 10.116c-.81.36-1.68.602-2.592.71a4.526 4.526 0 0 0 1.984-2.496 9.037 9.037 0 0 1-2.866 1.095 4.513 4.513 0 0 0-7.69 4.116 12.81 12.81 0 0 1-9.3-4.715 4.49 4.49 0 0 0-.612 2.27 4.51 4.51 0 0 0 2.008 3.755 4.495 4.495 0 0 1-2.044-.564v.057a4.515 4.515 0 0 0 3.62 4.425 4.52 4.52 0 0 1-2.04.077 4.517 4.517 0 0 0 4.217 3.134 9.055 9.055 0 0 1-5.604 1.93A9.18 9.18 0 0 1 6 23.85a12.773 12.773 0 0 0 6.918 2.027c8.3 0 12.84-6.876 12.84-12.84 0-.195-.005-.39-.014-.583a9.172 9.172 0 0 0 2.252-2.336" fill-rule="evenodd"></path></g></svg></span></a>
                </div>
                <a><span class="twitter_count">4</span></a>
            </li>
            <li class="social_block">
                <div class="atButton">
                    <a class="addthis_button_google at300b" addthis:title="Correcting Principal Component Maps for Effects of Spatial Autocorrelation in Population Genetic Data" addthis:description="In many species, spatial genetic variation displays patterns of “isolation-by-distance”. Characterized by locally correlated allele frequencies, these patterns are known to create periodic shapes in geographic maps of principal components which confound signatures of specific migration events and influence interpretations of principal component analyses (PCA). In this study, we introduced models combining probabilistic PCA and kriging models to infer population genetic structure from genetic data while correcting for effects generated by spatial autocorrelation. The corresponding algorithms are based on singular value decomposition and low rank approximation of the genotypic data. As their complexity is close to that of PCA, these algorithms scale with the dimension of the data. To illustrate the utility of these new models, we simulated isolation-by-distance patterns and broad-scale geographic variation using spatial coalescent models. Our methods remove the horseshoe patterns usually observed in PC maps and simplify intrepretations of spatial genetic variation. We demonstrate our approach by analyzing single nucleotide polymorphism data from the Human Genome Diversity Panel, and provide comparisons with other recently introduced methods." addthis:url="https://www.frontiersin.org/article/10.3389/fgene.2012.00254" title="Google" target="_blank" href="#"><span class="at-icon-wrapper" style="background-color: rgb(66, 133, 244); line-height: 16px; height: 16px; width: 16px;"><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 32 32" version="1.1" role="img" aria-labelledby="at-svg-google-3" class="at-icon at-icon-google" style="width: 16px; height: 16px;" title="Google Bookmark" alt="Google Bookmark"><title id="at-svg-google-3">Google Bookmark</title><g><path d="M16.213 13.998H26.72c.157.693.28 1.342.28 2.255C27 22.533 22.7 27 16.224 27 10.03 27 5 22.072 5 16S10.03 5 16.224 5c3.03 0 5.568 1.09 7.51 2.87l-3.188 3.037c-.808-.748-2.223-1.628-4.322-1.628-3.715 0-6.745 3.024-6.745 6.73 0 3.708 3.03 6.733 6.744 6.733 4.3 0 5.882-2.915 6.174-4.642h-6.185V14z" fill-rule="evenodd"></path></g></svg></span></a>
                </div>
                <a><span class="googleplus_count">0</span></a>
            </li>
            <li class="social_block">
                <div class="atButton">
                    <a class="addthis_button_linkedin at300b" addthis:title="Correcting Principal Component Maps for Effects of Spatial Autocorrelation in Population Genetic Data" addthis:description="In many species, spatial genetic variation displays patterns of “isolation-by-distance”. Characterized by locally correlated allele frequencies, these patterns are known to create periodic shapes in geographic maps of principal components which confound signatures of specific migration events and influence interpretations of principal component analyses (PCA). In this study, we introduced models combining probabilistic PCA and kriging models to infer population genetic structure from genetic data while correcting for effects generated by spatial autocorrelation. The corresponding algorithms are based on singular value decomposition and low rank approximation of the genotypic data. As their complexity is close to that of PCA, these algorithms scale with the dimension of the data. To illustrate the utility of these new models, we simulated isolation-by-distance patterns and broad-scale geographic variation using spatial coalescent models. Our methods remove the horseshoe patterns usually observed in PC maps and simplify intrepretations of spatial genetic variation. We demonstrate our approach by analyzing single nucleotide polymorphism data from the Human Genome Diversity Panel, and provide comparisons with other recently introduced methods." addthis:url="https://www.frontiersin.org/article/10.3389/fgene.2012.00254" title="LinkedIn" target="_blank" href="#"><span class="at-icon-wrapper" style="background-color: rgb(0, 119, 181); line-height: 16px; height: 16px; width: 16px;"><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 32 32" version="1.1" role="img" aria-labelledby="at-svg-linkedin-4" class="at-icon at-icon-linkedin" style="width: 16px; height: 16px;" title="LinkedIn" alt="LinkedIn"><title id="at-svg-linkedin-4">LinkedIn</title><g><path d="M26 25.963h-4.185v-6.55c0-1.56-.027-3.57-2.175-3.57-2.18 0-2.51 1.7-2.51 3.46v6.66h-4.182V12.495h4.012v1.84h.058c.558-1.058 1.924-2.174 3.96-2.174 4.24 0 5.022 2.79 5.022 6.417v7.386zM8.23 10.655a2.426 2.426 0 0 1 0-4.855 2.427 2.427 0 0 1 0 4.855zm-2.098 1.84h4.19v13.468h-4.19V12.495z" fill-rule="evenodd"></path></g></svg></span></a>
                </div>
                <a><span class="linkedin_count">0</span></a>
            </li>
            <li class="social_block">
                <div class="atButton">
                    <a class="addthis_button_more at300b" addthis:title="Correcting Principal Component Maps for Effects of Spatial Autocorrelation in Population Genetic Data" addthis:description="In many species, spatial genetic variation displays patterns of “isolation-by-distance”. Characterized by locally correlated allele frequencies, these patterns are known to create periodic shapes in geographic maps of principal components which confound signatures of specific migration events and influence interpretations of principal component analyses (PCA). In this study, we introduced models combining probabilistic PCA and kriging models to infer population genetic structure from genetic data while correcting for effects generated by spatial autocorrelation. The corresponding algorithms are based on singular value decomposition and low rank approximation of the genotypic data. As their complexity is close to that of PCA, these algorithms scale with the dimension of the data. To illustrate the utility of these new models, we simulated isolation-by-distance patterns and broad-scale geographic variation using spatial coalescent models. Our methods remove the horseshoe patterns usually observed in PC maps and simplify intrepretations of spatial genetic variation. We demonstrate our approach by analyzing single nucleotide polymorphism data from the Human Genome Diversity Panel, and provide comparisons with other recently introduced methods." addthis:url="https://www.frontiersin.org/article/10.3389/fgene.2012.00254" title="View more services" target="_blank" href="#"><span class="at-icon-wrapper" style="background-color: rgb(255, 101, 80); line-height: 16px; height: 16px; width: 16px;"><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 32 32" version="1.1" role="img" aria-labelledby="at-svg-addthis-5" class="at-icon at-icon-addthis" style="width: 16px; height: 16px;" title="More" alt="More"><title id="at-svg-addthis-5">AddThis</title><g><path d="M18 14V8h-4v6H8v4h6v6h4v-6h6v-4h-6z" fill-rule="evenodd"></path></g></svg></span></a>
                </div>
                <a><span class="total_count">148</span></a>
            </li>
        </ul>
    </div>
</aside>


            
<main class="b col-xs-12 col-sm-12 col-lg-8 col-lg-push-2 col-md-7 col-md-push-2 pull-left">
    <div class="article-section">
        <div class="article-container" data-html="True">
            <div class="abstract-container">
                
                <div class="article-header-container">
                    <!-- Start CrossMark Snippet v2.0 -->
                    <!-- End CrossMark Snippet -->
                    <div class="header-bar-one">
                      <h2>Original Research ARTICLE

                        </h2>
                    </div>
                    <div class="header-bar-three">
                        Front. Genet., 20 November 2012
                                 | <a href="https://doi.org/10.3389/fgene.2012.00254">https://doi.org/10.3389/fgene.2012.00254</a>
                    </div>
                </div>
                
<div class="JournalAbstract">
   <a id="h1" class="reset-hash-position"></a><h1>Correcting principal component maps for effects of spatial autocorrelation in population genetic data</h1>
   <div class="authors">
      <a href="http://www.frontiersin.org/Community/WhosWhoActivity.aspx?sname=EricFrichot&amp;UID=65514">Eric Frichot</a><sup>1</sup>, 
      <a href="http://community.frontiersin.org/people/SeanSchoville/72382">Sean Schoville</a><sup>1</sup>, 
      <a href="http://community.frontiersin.org/people/GuillaumeBouchard/72399">Guillaume Bouchard</a><sup>2</sup> and <a href="http://www.frontiersin.org/Community/WhosWhoActivity.aspx?sname=OlivierFrancois&amp;UID=65620">Olivier François</a><sup>1</sup>*</div>
   <ul class="notes">
      <li><span><sup>1</sup></span>Université Joseph Fourier Grenoble, Centre National de la Recherche, Grenoble, France</li>
      <li><span><sup>2</sup></span>Xerox Research Center Europe, Meylan, France</li>
   </ul>
   <p>In many species, spatial genetic variation displays patterns of 
“isolation-by-distance.” Characterized by locally correlated allele 
frequencies, these patterns are known to create periodic shapes in 
geographic maps of principal components which confound signatures of 
specific migration events and influence interpretations of principal 
component analyses (PCA). In this study, we introduced models combining 
probabilistic PCA and kriging models to infer population genetic 
structure from genetic data while correcting for effects generated by 
spatial autocorrelation. The corresponding algorithms are based on 
singular value decomposition and low rank approximation of the genotypic
 data. As their complexity is close to that of PCA, these algorithms 
scale with the dimensions of the data. To illustrate the utility of 
these new models, we simulated isolation-by-distance patterns and 
broad-scale geographic variation using spatial coalescent models. Our 
methods remove the horseshoe patterns usually observed in PC maps and 
simplify interpretations of spatial genetic variation. We demonstrate 
our approach by analyzing single nucleotide polymorphism data from the 
Human Genome Diversity Panel, and provide comparisons with other 
recently introduced methods.</p>
   <div class="clear"></div>
</div>
<div class="JournalFullText">
   <a id="h2" class="reset-hash-position"></a><h2>Introduction</h2>
   <p class="mb15">The concept of “isolation-by-distance” (IBD) was 
introduced by S. Wright to describe the accumulation of local genetic 
differences under spatially restricted dispersal (<a href="#B28">Wright, 1943</a>).
 In species that are continuously distributed in geographic space and 
disperse over short distances, the theory predicts that genetic 
differentiation will increase with geographic distance (<a href="#B16">Malécot, 1948</a>; <a href="#B14">Kimura and Weiss, 1964</a>).
 IBD can be described by spatial autocorrelation, a measure of the 
degree of dependency among observations in a geographic space. Although 
studying IBD patterns could lead to useful estimates of gene dispersal (<a href="#B26">Rousset, 1997</a>),
 spatial autocorrelation derived from IBD often presents a problem for 
population genetic analyses. More specifically, the presence of spatial 
autocorrelation patterns can increase the rate of false positive tests 
for hierarchical population structure or for the detection of loci under
 selection (<a href="#B19">Meirmans, 2012</a>).</p>
   <p class="mb15">Recently, it has been acknowledged that distortions 
caused by spatial autocorrelation could also bias interpretations of 
population genetic structure as inferred from principal component 
analysis (PCA) or from Bayesian clustering methods (<a href="#B20">Novembre and Stephens, 2008</a>; <a href="#B9">François et al., 2010</a>).
 PCA is a method that searches for axes, called principal components, 
along which projected individuals show the highest variance. As a 
result, the first PCs are often used to explore the structure of 
variation in the sample. Characterized by locally correlated allele 
frequencies, IBD patterns create periodic shapes in PC maps that can 
confound signatures of migration events and influence interpretations of
 principal component analyses (<a href="#B20">Novembre and Stephens, 2008</a>).
 In scenarios where covariance decays exponentially with geographic 
distance, PC plots are indeed expected to exhibit horseshoe effects, an 
artifact in which the second axis is curved relative to the first axis. 
These effects lead to counterintuitive representations of the data (<a href="#B15">Legendre and Gallagher, 2001</a>; <a href="#B5">Diaconis et al., 2008</a>).</p>
   <p class="mb0">Several methods have been proposed to correct for the 
effects of spatial autocorrelation in exploratory data analyses. In 
particular, those methods include spatial Principal Component Analysis 
(sPCA, <a href="#B2">Borcard and Legendre, 2002</a>; <a href="#B3">Borcard et al., 2004</a>; <a href="#B6">Dray et al., 2006</a>; <a href="#B13">Jombart et al., 2008</a>), and sparse factor analysis (SFA, <a href="#B8">Engelhardt and Stephens, 2010</a>).
 Generally the methods share the objective of separating local and 
regional geographic scales in the data. In this study, we introduce a 
novel approach, based on latent factors models, that addresses the 
separation of geographic scales more directly than the two previous 
methods. The new method, spatial factor analysis (spFA), combines 
probabilistic PCA (<a href="#B27">Tipping and Bishop, 1999</a>) and kriging models (<a href="#B4">Cressie, 1993</a>)
 to infer population genetic structure from genetic data while 
correcting for errors introduced by spatial autocorrelation. While many 
approaches have been argued to improve interpretations of the data, 
their outputs have not yet been compared to each other on the basis of 
spatial simulations. To compare methods, we generated patterns of IBD 
and broad-scale geographic variation using computer simulations of 
spatial coalescent models. We compared the outcomes of methods under 
population genetic models of isolation-by-distance, and we argued that 
the methods provided insights on distinct aspects of the data. We report
 that the new spFA method was able to remove the horseshoe effect 
observed in spatially structured data, whereas this was not the case in 
PCA, sPCA, and SFA analyses. We discuss the significance of this result 
in an assessment of single nucleotide polymorphism data from worldwide 
samples of the Human Genome Diversity Panel.</p>
   <a id="h3" class="reset-hash-position"></a><h2>Materials and Methods</h2>
   <p class="mb15">We considered single nucleotide polymorphism (SNP) data for <i>n</i> individuals genotyped at <i>L</i> loci. For these data, the genotypic matrix entries, (<i>G<sub>il</sub></i>), record the number of derived alleles at locus <i>l</i> for individual <i>i</i>. For autosomal data, <i>G<sub>il</sub></i> is thus equal to 0, 1, or 2, and corresponds to the genotype at locus <i>l</i>. The data were centered by subtracting the mean value of each column of <i>G</i> and scaled by dividing by the standard deviation value of each column of <i>G</i>. In addition to the genotypic data, we assumed that geographical coordinates, (<i>X<sub>i</sub></i>), were recorded for each individual.</p>
   <p class="mb0">We evaluated the effects of IBD patterns on inference 
of population genetic structure using 4 statistical methods: Principal 
Component Analysis (PCA, <a href="#B12">Jolliffe, 1986</a>; <a href="#B21">Patterson et al., 2006</a>), spatial PCA (sPCA, <a href="#B13">Jombart et al., 2008</a>), Sparse Factor Analysis (SFA, <a href="#B8">Engelhardt and Stephens, 2010</a>), and a new method called <i>spatial Factor Analysis</i> (spFA).</p>
   <h3>Principal Component Analysis</h3>
   <p class="mb0">PCA is a popular method that searches for a set of <i>K</i>
 orthogonal axes (the principal components), each of which is a linear 
combination of the original axes, such that projections of the original 
data display maximal variance onto the new axes (<a href="#B18">McVean, 2009</a>). We computed the score matrix, <i>U</i> of dimension <i>n</i> × <i>K</i>, and the loading matrix, <i>V</i> of dimensions <i>K</i> × <i>L</i>, using the rank <i>K</i> singular value decomposition method implemented in the R function <code>prcomp</code> and in the computer program <i>SmartPCA</i> (<a href="#B21">Patterson et al., 2006</a>).</p>
   <h3>Moran Eigenvectors and Spatial PCA</h3>
   <p class="mb15">Moran eigenvectors maps were proposed as an 
alternative to trend surface analysis for incorporating spatial 
variation in population genetics models (<a href="#B6">Dray et al., 2006</a>; <a href="#B13">Jombart et al., 2008</a>).
 In Moran eigenvectors maps, there are positive and negative 
eigenvalues. Eigenvectors associated with positive eigenvalues have 
positive autocorrelation, and they describe global structures. 
Eigenvectors associated with negative eigenvalues describe local 
structures. Implemented in an algorithm called spatial PCA (sPCA), 
Moran’s eigenvector maps (MEM) maximize Moran’s spatial autocorrelation 
index, defined as follows</p>
<div style="font-size:100%" class="equationImageholder">
         <span class="MathJax_Preview" style="color: inherit; display: none;"></span><span class="MathJax" id="MathJax-Element-1-Frame" tabindex="0" style="position: relative;" data-mathml="&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;M26&quot;&gt;&lt;mi&gt;I&lt;/mi&gt;&lt;mfenced separators=&quot;&quot; open=&quot;(&quot; close=&quot;)&quot;&gt;&lt;mrow&gt;&lt;mi&gt;G&lt;/mi&gt;&lt;/mrow&gt;&lt;/mfenced&gt;&lt;mo class=&quot;MathClass-rel&quot;&gt;=&lt;/mo&gt;&lt;mfrac&gt;&lt;mrow&gt;&lt;msub&gt;&lt;mrow&gt;&lt;mo mathsize=&quot;big&quot;&gt;&amp;#x2211;&lt;/mo&gt;&lt;/mrow&gt;&lt;mrow&gt;&lt;mi&gt;i&lt;/mi&gt;&lt;mo class=&quot;MathClass-punc&quot;&gt;,&lt;/mo&gt;&lt;mi&gt;j&lt;/mi&gt;&lt;/mrow&gt;&lt;/msub&gt;&lt;msub&gt;&lt;mrow&gt;&lt;mi&gt;w&lt;/mi&gt;&lt;/mrow&gt;&lt;mrow&gt;&lt;mi&gt;i&lt;/mi&gt;&lt;mi&gt;j&lt;/mi&gt;&lt;/mrow&gt;&lt;/msub&gt;&lt;mfenced separators=&quot;&quot; open=&quot;(&quot; close=&quot;)&quot;&gt;&lt;mrow&gt;&lt;msub&gt;&lt;mrow&gt;&lt;mi&gt;g&lt;/mi&gt;&lt;/mrow&gt;&lt;mrow&gt;&lt;mi&gt;i&lt;/mi&gt;&lt;/mrow&gt;&lt;/msub&gt;&lt;mo class=&quot;MathClass-bin&quot;&gt;-&lt;/mo&gt;&lt;mi&gt;&amp;#x1E21;&lt;/mi&gt;&lt;/mrow&gt;&lt;/mfenced&gt;&lt;mfenced separators=&quot;&quot; open=&quot;(&quot; close=&quot;)&quot;&gt;&lt;mrow&gt;&lt;msub&gt;&lt;mrow&gt;&lt;mi&gt;g&lt;/mi&gt;&lt;/mrow&gt;&lt;mrow&gt;&lt;mi&gt;j&lt;/mi&gt;&lt;/mrow&gt;&lt;/msub&gt;&lt;mo class=&quot;MathClass-bin&quot;&gt;-&lt;/mo&gt;&lt;mi&gt;&amp;#x1E21;&lt;/mi&gt;&lt;/mrow&gt;&lt;/mfenced&gt;&lt;mspace class=&quot;thinspace&quot; width=&quot;0.3em&quot; /&gt;&lt;/mrow&gt;&lt;mrow&gt;&lt;msub&gt;&lt;mrow&gt;&lt;mo mathsize=&quot;big&quot;&gt;&amp;#x2211;&lt;/mo&gt;&lt;/mrow&gt;&lt;mrow&gt;&lt;mi&gt;i&lt;/mi&gt;&lt;mo class=&quot;MathClass-punc&quot;&gt;,&lt;/mo&gt;&lt;mi&gt;j&lt;/mi&gt;&lt;/mrow&gt;&lt;/msub&gt;&lt;msub&gt;&lt;mrow&gt;&lt;mi&gt;w&lt;/mi&gt;&lt;/mrow&gt;&lt;mrow&gt;&lt;mi&gt;i&lt;/mi&gt;&lt;mi&gt;j&lt;/mi&gt;&lt;/mrow&gt;&lt;/msub&gt;&lt;msub&gt;&lt;mrow&gt;&lt;mo mathsize=&quot;big&quot;&gt;&amp;#x2211;&lt;/mo&gt;&lt;/mrow&gt;&lt;mrow&gt;&lt;mi&gt;i&lt;/mi&gt;&lt;/mrow&gt;&lt;/msub&gt;&lt;msup&gt;&lt;mrow&gt;&lt;mfenced separators=&quot;&quot; open=&quot;(&quot; close=&quot;)&quot;&gt;&lt;mrow&gt;&lt;msub&gt;&lt;mrow&gt;&lt;mi&gt;g&lt;/mi&gt;&lt;/mrow&gt;&lt;mrow&gt;&lt;mi&gt;i&lt;/mi&gt;&lt;/mrow&gt;&lt;/msub&gt;&lt;mo class=&quot;MathClass-bin&quot;&gt;-&lt;/mo&gt;&lt;mi&gt;&amp;#x1E21;&lt;/mi&gt;&lt;/mrow&gt;&lt;/mfenced&gt;&lt;/mrow&gt;&lt;mrow&gt;&lt;mn&gt;2&lt;/mn&gt;&lt;/mrow&gt;&lt;/msup&gt;&lt;/mrow&gt;&lt;/mfrac&gt;&lt;/math&gt;" role="presentation"><nobr aria-hidden="true"><span class="math" id="M26" style="width: 12.472em; display: inline-block;"><span style="display: inline-block; position: relative; width: 11.543em; height: 0px; font-size: 108%;"><span style="position: absolute; clip: rect(0.502em, 1011.54em, 4.306em, -1000em); top: -2.654em; left: 0em;"><span class="mrow" id="MathJax-Span-2"><span class="mi" id="MathJax-Span-3" style="font-family: MathJax_Math; font-style: italic;">I<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.064em;"></span></span><span class="mfenced" id="MathJax-Span-4" style="padding-left: 0.167em;"><span class="mo" id="MathJax-Span-35" style=""><span style="font-family: MathJax_Main;">(</span></span><span class="mrow" id="MathJax-Span-6"><span class="mi" id="MathJax-Span-7" style="font-family: MathJax_Math; font-style: italic;">G</span></span><span class="mo" id="MathJax-Span-36" style=""><span style="font-family: MathJax_Main;">)</span></span></span><span class="mo MathClass-rel" id="MathJax-Span-9" style="font-family: MathJax_Main; padding-left: 0.278em;">=</span><span class="mfrac" id="MathJax-Span-10" style="padding-left: 0.278em;"><span style="display: inline-block; position: relative; width: 7.753em; height: 0px; margin-right: 0.12em; margin-left: 0.12em;"><span style="position: absolute; clip: rect(2.767em, 1007.63em, 4.678em, -1000em); top: -4.919em; left: 50%; margin-left: -3.817em;"><span class="mrow" id="MathJax-Span-11"><span class="msub" id="MathJax-Span-12"><span style="display: inline-block; position: relative; width: 2.064em; height: 0px;"><span style="position: absolute; clip: rect(2.767em, 1001.41em, 4.551em, -1000em); top: -4.012em; left: 0em;"><span class="mrow" id="MathJax-Span-13"><span class="mo" id="MathJax-Span-14" style="font-size: 141.4%; font-family: MathJax_Size1; vertical-align: 0em;">∑</span></span><span style="display: inline-block; width: 0px; height: 4.012em;"></span></span><span style="position: absolute; top: -3.634em; left: 1.493em;"><span class="mrow" id="MathJax-Span-15"><span class="mi" id="MathJax-Span-16" style="font-size: 50%; font-family: MathJax_Math; font-style: italic;">i</span><span class="mo MathClass-punc" id="MathJax-Span-17" style="font-size: 50%; font-family: MathJax_Main;">,</span><span class="mi" id="MathJax-Span-18" style="font-size: 50%; font-family: MathJax_Math; font-style: italic;">j</span></span><span style="display: inline-block; width: 0px; height: 4.012em;"></span></span></span></span><span class="msub" id="MathJax-Span-19" style="padding-left: 0.167em;"><span style="display: inline-block; position: relative; width: 0.938em; height: 0px;"><span style="position: absolute; clip: rect(3.514em, 1000.49em, 4.205em, -1000em); top: -4.012em; left: 0em;"><span class="mrow" id="MathJax-Span-20"><span class="mi" id="MathJax-Span-21" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">w</span></span><span style="display: inline-block; width: 0px; height: 4.012em;"></span></span><span style="position: absolute; top: -3.906em; left: 0.506em;"><span class="mrow" id="MathJax-Span-22"><span class="mi" id="MathJax-Span-23" style="font-size: 50%; font-family: MathJax_Math; font-style: italic;">i</span><span class="mi" id="MathJax-Span-24" style="font-size: 50%; font-family: MathJax_Math; font-style: italic;">j</span></span><span style="display: inline-block; width: 0px; height: 4.012em;"></span></span></span></span><span class="mfenced" id="MathJax-Span-25"><span class="mo" id="MathJax-Span-37" style=""><span><span style="font-size: 70.7%; font-family: MathJax_Main;">(</span></span></span><span class="mrow" id="MathJax-Span-27"><span class="msub" id="MathJax-Span-28"><span style="display: inline-block; position: relative; width: 0.563em; height: 0px;"><span style="position: absolute; clip: rect(3.515em, 1000.34em, 4.342em, -1000em); top: -4.012em; left: 0em;"><span class="mrow" id="MathJax-Span-29"><span class="mi" id="MathJax-Span-30" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">g<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.002em;"></span></span></span><span style="display: inline-block; width: 0px; height: 4.012em;"></span></span><span style="position: absolute; top: -3.842em; left: 0.337em;"><span class="mrow" id="MathJax-Span-31"><span class="mi" id="MathJax-Span-32" style="font-size: 50%; font-family: MathJax_Math; font-style: italic;">i</span></span><span style="display: inline-block; width: 0px; height: 4.012em;"></span></span></span></span><span class="mo MathClass-bin" id="MathJax-Span-33" style="font-size: 70.7%; font-family: MathJax_Main;">−</span><span class="mi" id="MathJax-Span-34" style="font-size: 70.7%;"><span style="font-family: STIXGeneral, &quot;Arial Unicode MS&quot;, serif; font-size: 93%; font-style: italic; font-weight: normal;">ḡ</span></span></span><span class="mo" id="MathJax-Span-38" style=""><span><span style="font-size: 70.7%; font-family: MathJax_Main;">)</span></span></span></span><span class="mfenced" id="MathJax-Span-39"><span class="mo" id="MathJax-Span-40" style="vertical-align: 0em;"><span><span style="font-size: 70.7%; font-family: MathJax_Size1;">(</span></span></span><span class="mrow" id="MathJax-Span-41"><span class="msub" id="MathJax-Span-42"><span style="display: inline-block; position: relative; width: 0.596em; height: 0px;"><span style="position: absolute; clip: rect(3.515em, 1000.34em, 4.342em, -1000em); top: -4.012em; left: 0em;"><span class="mrow" id="MathJax-Span-43"><span class="mi" id="MathJax-Span-44" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">g<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.002em;"></span></span></span><span style="display: inline-block; width: 0px; height: 4.012em;"></span></span><span style="position: absolute; top: -3.842em; left: 0.337em;"><span class="mrow" id="MathJax-Span-45"><span class="mi" id="MathJax-Span-46" style="font-size: 50%; font-family: MathJax_Math; font-style: italic;">j</span></span><span style="display: inline-block; width: 0px; height: 4.012em;"></span></span></span></span><span class="mo MathClass-bin" id="MathJax-Span-47" style="font-size: 70.7%; font-family: MathJax_Main;">−</span><span class="mi" id="MathJax-Span-48" style="font-size: 70.7%;"><span style="font-family: STIXGeneral, &quot;Arial Unicode MS&quot;, serif; font-size: 93%; font-style: italic; font-weight: normal;">ḡ</span></span></span><span class="mo" id="MathJax-Span-49" style="vertical-align: 0em;"><span><span style="font-size: 70.7%; font-family: MathJax_Size1;">)</span></span></span></span><span class="mspace thinspace" id="MathJax-Span-50" style="height: 0em; vertical-align: 0em; width: 0.3em; display: inline-block; overflow: hidden;"></span></span><span style="display: inline-block; width: 0px; height: 4.012em;"></span></span><span style="position: absolute; clip: rect(2.767em, 1007.54em, 4.678em, -1000em); top: -3.026em; left: 50%; margin-left: -3.771em;"><span class="mrow" id="MathJax-Span-51"><span class="msub" id="MathJax-Span-52"><span style="display: inline-block; position: relative; width: 2.064em; height: 0px;"><span style="position: absolute; clip: rect(2.767em, 1001.41em, 4.551em, -1000em); top: -4.012em; left: 0em;"><span class="mrow" id="MathJax-Span-53"><span class="mo" id="MathJax-Span-54" style="font-size: 141.4%; font-family: MathJax_Size1; vertical-align: 0em;">∑</span></span><span style="display: inline-block; width: 0px; height: 4.012em;"></span></span><span style="position: absolute; top: -3.634em; left: 1.493em;"><span class="mrow" id="MathJax-Span-55"><span class="mi" id="MathJax-Span-56" style="font-size: 50%; font-family: MathJax_Math; font-style: italic;">i</span><span class="mo MathClass-punc" id="MathJax-Span-57" style="font-size: 50%; font-family: MathJax_Main;">,</span><span class="mi" id="MathJax-Span-58" style="font-size: 50%; font-family: MathJax_Math; font-style: italic;">j</span></span><span style="display: inline-block; width: 0px; height: 4.012em;"></span></span></span></span><span class="msub" id="MathJax-Span-59" style="padding-left: 0.167em;"><span style="display: inline-block; position: relative; width: 0.938em; height: 0px;"><span style="position: absolute; clip: rect(3.514em, 1000.49em, 4.205em, -1000em); top: -4.012em; left: 0em;"><span class="mrow" id="MathJax-Span-60"><span class="mi" id="MathJax-Span-61" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">w</span></span><span style="display: inline-block; width: 0px; height: 4.012em;"></span></span><span style="position: absolute; top: -3.906em; left: 0.506em;"><span class="mrow" id="MathJax-Span-62"><span class="mi" id="MathJax-Span-63" style="font-size: 50%; font-family: MathJax_Math; font-style: italic;">i</span><span class="mi" id="MathJax-Span-64" style="font-size: 50%; font-family: MathJax_Math; font-style: italic;">j</span></span><span style="display: inline-block; width: 0px; height: 4.012em;"></span></span></span></span><span class="msub" id="MathJax-Span-65" style="padding-left: 0.167em;"><span style="display: inline-block; position: relative; width: 1.719em; height: 0px;"><span style="position: absolute; clip: rect(2.767em, 1001.41em, 4.551em, -1000em); top: -4.012em; left: 0em;"><span class="mrow" id="MathJax-Span-66"><span class="mo" id="MathJax-Span-67" style="font-size: 141.4%; font-family: MathJax_Size1; vertical-align: 0em;">∑</span></span><span style="display: inline-block; width: 0px; height: 4.012em;"></span></span><span style="position: absolute; top: -3.634em; left: 1.493em;"><span class="mrow" id="MathJax-Span-68"><span class="mi" id="MathJax-Span-69" style="font-size: 50%; font-family: MathJax_Math; font-style: italic;">i</span></span><span style="display: inline-block; width: 0px; height: 4.012em;"></span></span></span></span><span class="msup" id="MathJax-Span-70" style="padding-left: 0.167em;"><span style="display: inline-block; position: relative; width: 2.32em; height: 0px;"><span style="position: absolute; clip: rect(3.261em, 1001.95em, 4.374em, -1000em); top: -4.012em; left: 0em;"><span class="mrow" id="MathJax-Span-71"><span class="mfenced" id="MathJax-Span-72"><span class="mo" id="MathJax-Span-73" style=""><span><span style="font-size: 70.7%; font-family: MathJax_Main;">(</span></span></span><span class="mrow" id="MathJax-Span-74"><span class="msub" id="MathJax-Span-75"><span style="display: inline-block; position: relative; width: 0.563em; height: 0px;"><span style="position: absolute; clip: rect(3.515em, 1000.34em, 4.342em, -1000em); top: -4.012em; left: 0em;"><span class="mrow" id="MathJax-Span-76"><span class="mi" id="MathJax-Span-77" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">g<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.002em;"></span></span></span><span style="display: inline-block; width: 0px; height: 4.012em;"></span></span><span style="position: absolute; top: -3.842em; left: 0.337em;"><span class="mrow" id="MathJax-Span-78"><span class="mi" id="MathJax-Span-79" style="font-size: 50%; font-family: MathJax_Math; font-style: italic;">i</span></span><span style="display: inline-block; width: 0px; height: 4.012em;"></span></span></span></span><span class="mo MathClass-bin" id="MathJax-Span-80" style="font-size: 70.7%; font-family: MathJax_Main;">−</span><span class="mi" id="MathJax-Span-81" style="font-size: 70.7%;"><span style="font-family: STIXGeneral, &quot;Arial Unicode MS&quot;, serif; font-size: 93%; font-style: italic; font-weight: normal;">ḡ</span></span></span><span class="mo" id="MathJax-Span-82" style=""><span><span style="font-size: 70.7%; font-family: MathJax_Main;">)</span></span></span></span></span><span style="display: inline-block; width: 0px; height: 4.012em;"></span></span><span style="position: absolute; top: -4.385em; left: 2.017em;"><span class="mrow" id="MathJax-Span-83"><span class="mn" id="MathJax-Span-84" style="font-size: 50%; font-family: MathJax_Main;">2</span></span><span style="display: inline-block; width: 0px; height: 4.012em;"></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 4.012em;"></span></span><span style="position: absolute; clip: rect(0.787em, 1007.75em, 1.235em, -1000em); top: -1.269em; left: 0em;"><span style="display: inline-block; overflow: hidden; vertical-align: 0em; border-top: 1.3px solid; width: 7.753em; height: 0px;"></span><span style="display: inline-block; width: 0px; height: 1.049em;"></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 2.654em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -1.651em; border-left: 0px solid; width: 0px; height: 3.842em;"></span></span></nobr><span class="MJX_Assistive_MathML" role="presentation"><math xmlns="http://www.w3.org/1998/Math/MathML" id="M26"><mi>I</mi><mfenced separators="" open="(" close=")"><mrow><mi>G</mi></mrow></mfenced><mo class="MathClass-rel">=</mo><mfrac><mrow><msub><mrow><mo mathsize="big">∑</mo></mrow><mrow><mi>i</mi><mo class="MathClass-punc">,</mo><mi>j</mi></mrow></msub><msub><mrow><mi>w</mi></mrow><mrow><mi>i</mi><mi>j</mi></mrow></msub><mfenced separators="" open="(" close=")"><mrow><msub><mrow><mi>g</mi></mrow><mrow><mi>i</mi></mrow></msub><mo class="MathClass-bin">-</mo><mi>ḡ</mi></mrow></mfenced><mfenced separators="" open="(" close=")"><mrow><msub><mrow><mi>g</mi></mrow><mrow><mi>j</mi></mrow></msub><mo class="MathClass-bin">-</mo><mi>ḡ</mi></mrow></mfenced><mspace class="thinspace" width="0.3em"></mspace></mrow><mrow><msub><mrow><mo mathsize="big">∑</mo></mrow><mrow><mi>i</mi><mo class="MathClass-punc">,</mo><mi>j</mi></mrow></msub><msub><mrow><mi>w</mi></mrow><mrow><mi>i</mi><mi>j</mi></mrow></msub><msub><mrow><mo mathsize="big">∑</mo></mrow><mrow><mi>i</mi></mrow></msub><msup><mrow><mfenced separators="" open="(" close=")"><mrow><msub><mrow><mi>g</mi></mrow><mrow><mi>i</mi></mrow></msub><mo class="MathClass-bin">-</mo><mi>ḡ</mi></mrow></mfenced></mrow><mrow><mn>2</mn></mrow></msup></mrow></mfrac></math></span></span><script type="math/mml" id="MathJax-Element-1"><math id="M26"><mi>I</mi><mfenced separators="" open="(" close=")"><mrow><mi>G</mi></mrow></mfenced><mo class="MathClass-rel">=</mo><mfrac><mrow><msub><mrow><mo mathsize="big">∑</mo></mrow><mrow><mi>i</mi><mo class="MathClass-punc">,</mo><mi>j</mi></mrow></msub><msub><mrow><mi>w</mi></mrow><mrow><mi>i</mi><mi>j</mi></mrow></msub><mfenced separators="" open="(" close=")"><mrow><msub><mrow><mi>g</mi></mrow><mrow><mi>i</mi></mrow></msub><mo class="MathClass-bin">-</mo><mi>ḡ</mi></mrow></mfenced><mfenced separators="" open="(" close=")"><mrow><msub><mrow><mi>g</mi></mrow><mrow><mi>j</mi></mrow></msub><mo class="MathClass-bin">-</mo><mi>ḡ</mi></mrow></mfenced><mspace width="0.3em" class="thinspace"></mspace></mrow><mrow><msub><mrow><mo mathsize="big">∑</mo></mrow><mrow><mi>i</mi><mo class="MathClass-punc">,</mo><mi>j</mi></mrow></msub><msub><mrow><mi>w</mi></mrow><mrow><mi>i</mi><mi>j</mi></mrow></msub><msub><mrow><mo mathsize="big">∑</mo></mrow><mrow><mi>i</mi></mrow></msub><msup><mrow><mfenced separators="" open="(" close=")"><mrow><msub><mrow><mi>g</mi></mrow><mrow><mi>i</mi></mrow></msub><mo class="MathClass-bin">-</mo><mi>ḡ</mi></mrow></mfenced></mrow><mrow><mn>2</mn></mrow></msup></mrow></mfrac></math></script>
         <div class="clear"></div>
<div class="DottedLine"></div>
      </div>
<p>with respect to a spatial weighting matrix, <i>W</i>, deduced from geographical distances and where <span class="MathJax_Preview" style="color: inherit; display: none;"></span><span class="MathJax" id="MathJax-Element-2-Frame" tabindex="0" style="position: relative;" data-mathml="&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;M1&quot;&gt;&lt;mrow&gt;&lt;msub&gt;&lt;mrow&gt;&lt;mi&gt;g&lt;/mi&gt;&lt;/mrow&gt;&lt;mrow&gt;&lt;mi&gt;i&lt;/mi&gt;&lt;/mrow&gt;&lt;/msub&gt;&lt;/mrow&gt;&lt;/math&gt;" role="presentation"><nobr aria-hidden="true"><span class="math" id="M1" style="width: 0.96em; display: inline-block;"><span style="display: inline-block; position: relative; width: 0.814em; height: 0px; font-size: 116%;"><span style="position: absolute; clip: rect(1.569em, 1000.81em, 2.547em, -1000em); top: -2.155em; left: 0em;"><span class="mrow" id="MathJax-Span-86"><span class="mrow" id="MathJax-Span-87"><span class="msub" id="MathJax-Span-88"><span style="display: inline-block; position: relative; width: 0.796em; height: 0px;"><span style="position: absolute; clip: rect(3.437em, 1000.48em, 4.372em, -1000em); top: -4.023em; left: 0em;"><span class="mrow" id="MathJax-Span-89"><span class="mi" id="MathJax-Span-90" style="font-family: MathJax_Math; font-style: italic;">g<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span></span><span style="display: inline-block; width: 0px; height: 4.023em;"></span></span><span style="position: absolute; top: -3.783em; left: 0.477em;"><span class="mrow" id="MathJax-Span-91"><span class="mi" id="MathJax-Span-92" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">i</span></span><span style="display: inline-block; width: 0px; height: 4.023em;"></span></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 2.155em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.343em; border-left: 0px solid; width: 0px; height: 0.912em;"></span></span></nobr><span class="MJX_Assistive_MathML" role="presentation"><math xmlns="http://www.w3.org/1998/Math/MathML" id="M1"><mrow><msub><mrow><mi>g</mi></mrow><mrow><mi>i</mi></mrow></msub></mrow></math></span></span><script type="math/mml" id="MathJax-Element-2"><math id="M1"><mrow><msub><mrow><mi>g</mi></mrow><mrow><mi>i</mi></mrow></msub></mrow></math></script> is the <i>i</i>th column of <i>G</i> (<a href="#B6">Dray et al., 2006</a>). We implemented MEMs and sPCA using the R package <code>adegenet</code> using a Delaunay weighting matrix (<a href="#B13">Jombart et al., 2008</a>).</p>
   <h3>Spatial Factor Analysis</h3>
   <p class="mb15">We introduce a new spatial factor analysis model 
(spFA) which incorporates spatial information in factor analysis in an 
explicit way. In spFA, inference is performed in a matrix factorization 
model similar to probabilistic PCA (<a href="#B27">Tipping and Bishop, 1999</a>).</p>
   <div style="font-size:100%" class="equationImageholder">
         <span class="MathJax_Preview" style="color: inherit; display: none;"></span><span class="MathJax" id="MathJax-Element-3-Frame" tabindex="0" style="position: relative;" data-mathml="&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;M27&quot;&gt;&lt;mtable&gt;&lt;mtr&gt;&lt;mtd&gt;&lt;msub&gt;&lt;mrow&gt;&lt;mi&gt;G&lt;/mi&gt;&lt;/mrow&gt;&lt;mrow&gt;&lt;mi&gt;i&lt;/mi&gt;&lt;mi&gt;&amp;#x2113;&lt;/mi&gt;&lt;/mrow&gt;&lt;/msub&gt;&lt;mo class=&quot;MathClass-rel&quot;&gt;=&lt;/mo&gt;&lt;msubsup&gt;&lt;mrow&gt;&lt;mi&gt;U&lt;/mi&gt;&lt;/mrow&gt;&lt;mrow&gt;&lt;mi&gt;i&lt;/mi&gt;&lt;/mrow&gt;&lt;mrow&gt;&lt;mi&gt;T&lt;/mi&gt;&lt;/mrow&gt;&lt;/msubsup&gt;&lt;msub&gt;&lt;mrow&gt;&lt;mi&gt;V&lt;/mi&gt;&lt;/mrow&gt;&lt;mrow&gt;&lt;mi&gt;&amp;#x2113;&lt;/mi&gt;&lt;/mrow&gt;&lt;/msub&gt;&lt;mo class=&quot;MathClass-bin&quot;&gt;+&lt;/mo&gt;&lt;msub&gt;&lt;mrow&gt;&lt;mi&gt;&amp;#x3F5;&lt;/mi&gt;&lt;/mrow&gt;&lt;mrow&gt;&lt;mi&gt;i&lt;/mi&gt;&lt;mi&gt;&amp;#x2113;&lt;/mi&gt;&lt;/mrow&gt;&lt;/msub&gt;&lt;mo class=&quot;MathClass-punc&quot;&gt;,&lt;/mo&gt;&lt;/mtd&gt;&lt;mtd&gt;&lt;mtext class=&quot;eqnarray&quot;&gt;(1)&lt;/mtext&gt;&lt;/mtd&gt;&lt;/mtr&gt;&lt;/mtable&gt;&lt;/math&gt;" role="presentation"><nobr aria-hidden="true"><span class="math" id="M27" style="width: 10.867em; display: inline-block;"><span style="display: inline-block; position: relative; width: 10.062em; height: 0px; font-size: 108%;"><span style="position: absolute; clip: rect(1.153em, 1009.8em, 2.668em, -1000em); top: -2.16em; left: 0em;"><span class="mrow" id="MathJax-Span-94"><span class="mtable" id="MathJax-Span-95" style="padding-right: 0.167em; padding-left: 0.167em;"><span style="display: inline-block; position: relative; width: 9.699em; height: 0px;"><span style="position: absolute; clip: rect(3.005em, 1007.55em, 4.519em, -1000em); top: -4.012em; left: 0em;"><span style="display: inline-block; position: relative; width: 7.621em; height: 0px;"><span style="position: absolute; clip: rect(2.938em, 1007.55em, 4.453em, -1000em); top: -3.945em; left: 50%; margin-left: -3.811em;"><span class="mtd" id="MathJax-Span-96"><span class="mrow" id="MathJax-Span-97"><span class="msub" id="MathJax-Span-98"><span style="display: inline-block; position: relative; width: 1.4em; height: 0px;"><span style="position: absolute; clip: rect(3.122em, 1000.76em, 4.22em, -1000em); top: -4.012em; left: 0em;"><span class="mrow" id="MathJax-Span-99"><span class="mi" id="MathJax-Span-100" style="font-family: MathJax_Math; font-style: italic;">G</span></span><span style="display: inline-block; width: 0px; height: 4.012em;"></span></span><span style="position: absolute; top: -3.858em; left: 0.786em;"><span class="mrow" id="MathJax-Span-101"><span class="mi" id="MathJax-Span-102" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">i</span><span class="mi" id="MathJax-Span-103" style="font-size: 70.7%; font-family: MathJax_Main;">ℓ</span></span><span style="display: inline-block; width: 0px; height: 4.012em;"></span></span></span></span><span class="mo MathClass-rel" id="MathJax-Span-104" style="font-family: MathJax_Main; padding-left: 0.278em;">=</span><span class="msubsup" id="MathJax-Span-105" style="padding-left: 0.278em;"><span style="display: inline-block; position: relative; width: 1.415em; height: 0px;"><span style="position: absolute; clip: rect(3.144em, 1000.77em, 4.22em, -1000em); top: -4.012em; left: 0em;"><span class="mrow" id="MathJax-Span-106"><span class="mi" id="MathJax-Span-107" style="font-family: MathJax_Math; font-style: italic;">U<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.084em;"></span></span></span><span style="display: inline-block; width: 0px; height: 4.012em;"></span></span><span style="position: absolute; clip: rect(3.348em, 1000.57em, 4.198em, -1000em); top: -4.422em; left: 0.842em;"><span class="mrow" id="MathJax-Span-108"><span class="mi" id="MathJax-Span-109" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">T<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.085em;"></span></span></span><span style="display: inline-block; width: 0px; height: 4.012em;"></span></span><span style="position: absolute; clip: rect(3.36em, 1000.32em, 4.205em, -1000em); top: -3.765em; left: 0.683em;"><span class="mrow" id="MathJax-Span-110"><span class="mi" id="MathJax-Span-111" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">i</span></span><span style="display: inline-block; width: 0px; height: 4.012em;"></span></span></span></span><span class="msub" id="MathJax-Span-112"><span style="display: inline-block; position: relative; width: 0.953em; height: 0px;"><span style="position: absolute; clip: rect(3.144em, 1000.77em, 4.22em, -1000em); top: -4.012em; left: 0em;"><span class="mrow" id="MathJax-Span-113"><span class="mi" id="MathJax-Span-114" style="font-family: MathJax_Math; font-style: italic; text-rendering: optimizelegibility;">V<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.186em;"></span></span></span><span style="display: inline-block; width: 0px; height: 4.012em;"></span></span><span style="position: absolute; top: -3.858em; left: 0.583em;"><span class="mrow" id="MathJax-Span-115"><span class="mi" id="MathJax-Span-116" style="font-size: 70.7%; font-family: MathJax_Main;">ℓ</span></span><span style="display: inline-block; width: 0px; height: 4.012em;"></span></span></span></span><span class="mo MathClass-bin" id="MathJax-Span-117" style="font-family: MathJax_Main; padding-left: 0.222em;">+</span><span class="msub" id="MathJax-Span-118" style="padding-left: 0.222em;"><span style="display: inline-block; position: relative; width: 1.02em; height: 0px;"><span style="position: absolute; clip: rect(3.396em, 1000.38em, 4.209em, -1000em); top: -4.012em; left: 0em;"><span class="mrow" id="MathJax-Span-119"><span class="mi" id="MathJax-Span-120" style="font-family: MathJax_Math; font-style: italic;">ϵ</span></span><span style="display: inline-block; width: 0px; height: 4.012em;"></span></span><span style="position: absolute; top: -3.858em; left: 0.406em;"><span class="mrow" id="MathJax-Span-121"><span class="mi" id="MathJax-Span-122" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">i</span><span class="mi" id="MathJax-Span-123" style="font-size: 70.7%; font-family: MathJax_Main;">ℓ</span></span><span style="display: inline-block; width: 0px; height: 4.012em;"></span></span></span></span><span class="mo MathClass-punc" id="MathJax-Span-124" style="font-family: MathJax_Main;">,</span></span></span><span style="display: inline-block; width: 0px; height: 4.012em;"></span></span></span><span style="display: inline-block; width: 0px; height: 4.012em;"></span></span><span style="position: absolute; clip: rect(3.144em, 1001.18em, 4.514em, -1000em); top: -4.012em; left: 8.421em;"><span style="display: inline-block; position: relative; width: 1.278em; height: 0px;"><span style="position: absolute; clip: rect(3.077em, 1001.18em, 4.448em, -1000em); top: -3.945em; left: 50%; margin-left: -0.639em;"><span class="mtd" id="MathJax-Span-125"><span class="mrow" id="MathJax-Span-126"><span class="mtext eqnarray" id="MathJax-Span-127" style="font-family: MathJax_Main;">(1)</span></span></span><span style="display: inline-block; width: 0px; height: 4.012em;"></span></span></span><span style="display: inline-block; width: 0px; height: 4.012em;"></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 2.16em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.414em; border-left: 0px solid; width: 0px; height: 1.369em;"></span></span></nobr><span class="MJX_Assistive_MathML" role="presentation"><math xmlns="http://www.w3.org/1998/Math/MathML" id="M27"><mtable><mtr><mtd><msub><mrow><mi>G</mi></mrow><mrow><mi>i</mi><mi>ℓ</mi></mrow></msub><mo class="MathClass-rel">=</mo><msubsup><mrow><mi>U</mi></mrow><mrow><mi>i</mi></mrow><mrow><mi>T</mi></mrow></msubsup><msub><mrow><mi>V</mi></mrow><mrow><mi>ℓ</mi></mrow></msub><mo class="MathClass-bin">+</mo><msub><mrow><mi>ϵ</mi></mrow><mrow><mi>i</mi><mi>ℓ</mi></mrow></msub><mo class="MathClass-punc">,</mo></mtd><mtd><mtext class="eqnarray">(1)</mtext></mtd></mtr></mtable></math></span></span><script type="math/mml" id="MathJax-Element-3"><math id="M27"><mtable><mtr><mtd><msub><mrow><mi>G</mi></mrow><mrow><mi>i</mi><mi>ℓ</mi></mrow></msub><mo class="MathClass-rel">=</mo><msubsup><mrow><mi>U</mi></mrow><mrow><mi>i</mi></mrow><mrow><mi>T</mi></mrow></msubsup><msub><mrow><mi>V</mi></mrow><mrow><mi>ℓ</mi></mrow></msub><mo class="MathClass-bin">+</mo><msub><mrow><mi>ϵ</mi></mrow><mrow><mi>i</mi><mi>ℓ</mi></mrow></msub><mo class="MathClass-punc">,</mo></mtd><mtd><mtext class="eqnarray">(1)</mtext></mtd></mtr></mtable></math></script>
         <div class="clear"></div>
<div class="DottedLine"></div>
      </div>
<p>where <i>ε<sub>iℓ</sub></i> are statistically dependent Gaussian variables with mean zero and with covariance matrix Σ<i><sub>θ</sub></i>. Similarly to Kriging approaches (<a href="#B4">Cressie, 1993</a>), a radial basis covariance matrix was chosen to model spatial autocorrelation patterns generated by IBD (see also <a href="#B7">Durand et al., 2009</a>). The covariance matrix Σ<i><sub>θ</sub></i> was defined as follows. For all pairs of individuals, <i>i</i> and <i>j</i>, we have</p>
<div style="font-size:100%" class="equationImageholder">
         <span class="MathJax_Preview" style="color: inherit; display: none;"></span><span class="MathJax" id="MathJax-Element-4-Frame" tabindex="0" style="position: relative;" data-mathml="&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;M28&quot;&gt;&lt;mtable&gt;&lt;mtr&gt;&lt;mtd&gt;&lt;msub&gt;&lt;mrow&gt;&lt;mo mathsize=&quot;big&quot;&gt;&amp;#x2211;&lt;/mo&gt;&lt;/mrow&gt;&lt;mrow&gt;&lt;mi&gt;&amp;#x3B8;&lt;/mi&gt;&lt;/mrow&gt;&lt;/msub&gt;&lt;mfenced separators=&quot;&quot; open=&quot;(&quot; close=&quot;)&quot;&gt;&lt;mrow&gt;&lt;mi&gt;i&lt;/mi&gt;&lt;mo class=&quot;MathClass-punc&quot;&gt;,&lt;/mo&gt;&lt;mi&gt;j&lt;/mi&gt;&lt;/mrow&gt;&lt;/mfenced&gt;&lt;mo class=&quot;MathClass-rel&quot;&gt;=&lt;/mo&gt;&lt;mo class=&quot;qopname&quot;&gt;exp&lt;/mo&gt;&lt;mfenced separators=&quot;&quot; open=&quot;(&quot; close=&quot;)&quot;&gt;&lt;mrow&gt;&lt;mo class=&quot;MathClass-bin&quot;&gt;-&lt;/mo&gt;&lt;mi&gt;d&lt;/mi&gt;&lt;mfenced separators=&quot;&quot; open=&quot;(&quot; close=&quot;)&quot;&gt;&lt;mrow&gt;&lt;msub&gt;&lt;mrow&gt;&lt;mi&gt;X&lt;/mi&gt;&lt;/mrow&gt;&lt;mrow&gt;&lt;mi&gt;i&lt;/mi&gt;&lt;/mrow&gt;&lt;/msub&gt;&lt;mo class=&quot;MathClass-punc&quot;&gt;,&lt;/mo&gt;&lt;msub&gt;&lt;mrow&gt;&lt;mi&gt;X&lt;/mi&gt;&lt;/mrow&gt;&lt;mrow&gt;&lt;mi&gt;j&lt;/mi&gt;&lt;/mrow&gt;&lt;/msub&gt;&lt;/mrow&gt;&lt;/mfenced&gt;&lt;mo&gt;/&lt;/mo&gt;&lt;mi&gt;&amp;#x3B8;&lt;/mi&gt;&lt;/mrow&gt;&lt;/mfenced&gt;&lt;mo class=&quot;MathClass-punc&quot;&gt;,&lt;/mo&gt;&lt;mspace class=&quot;quad&quot; width=&quot;1em&quot; /&gt;&lt;mi&gt;&amp;#x3B8;&lt;/mi&gt;&lt;mo class=&quot;MathClass-rel&quot;&gt;&amp;gt;&lt;/mo&gt;&lt;mn&gt;0&lt;/mn&gt;&lt;mo class=&quot;MathClass-punc&quot;&gt;,&lt;/mo&gt;&lt;/mtd&gt;&lt;mtd&gt;&lt;mtext class=&quot;eqnarray&quot;&gt;(2)&lt;/mtext&gt;&lt;/mtd&gt;&lt;/mtr&gt;&lt;/mtable&gt;&lt;/math&gt;" role="presentation"><nobr aria-hidden="true"><span class="math" id="M28" style="width: 22.596em; display: inline-block;"><span style="display: inline-block; position: relative; width: 20.926em; height: 0px; font-size: 108%;"><span style="position: absolute; clip: rect(0.704em, 1020.66em, 3.117em, -1000em); top: -2.16em; left: 0em;"><span class="mrow" id="MathJax-Span-129"><span class="mtable" id="MathJax-Span-130" style="padding-right: 0.167em; padding-left: 0.167em;"><span style="display: inline-block; position: relative; width: 20.578em; height: 0px;"><span style="position: absolute; clip: rect(2.556em, 1018.43em, 4.969em, -1000em); top: -4.012em; left: 0em;"><span style="display: inline-block; position: relative; width: 18.5em; height: 0px;"><span style="position: absolute; clip: rect(2.327em, 1018.43em, 4.74em, -1000em); top: -3.784em; left: 50%; margin-left: -9.25em;"><span class="mtd" id="MathJax-Span-131"><span class="mrow" id="MathJax-Span-132"><span class="msub" id="MathJax-Span-133"><span style="display: inline-block; position: relative; width: 2.519em; height: 0px;"><span style="position: absolute; clip: rect(2.327em, 1002em, 4.698em, -1000em); top: -4.012em; left: 0em;"><span class="mrow" id="MathJax-Span-134"><span class="mo" id="MathJax-Span-135" style="font-size: 200%; font-family: MathJax_Size1; vertical-align: 0em;">∑</span></span><span style="display: inline-block; width: 0px; height: 4.012em;"></span></span><span style="position: absolute; top: -3.477em; left: 2.112em;"><span class="mrow" id="MathJax-Span-136"><span class="mi" id="MathJax-Span-137" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">θ</span></span><span style="display: inline-block; width: 0px; height: 4.012em;"></span></span></span></span><span class="mfenced" id="MathJax-Span-138" style="padding-left: 0.167em;"><span class="mo" id="MathJax-Span-139" style=""><span style="font-family: MathJax_Main;">(</span></span><span class="mrow" id="MathJax-Span-140"><span class="mi" id="MathJax-Span-141" style="font-family: MathJax_Math; font-style: italic;">i</span><span class="mo MathClass-punc" id="MathJax-Span-142" style="font-family: MathJax_Main;">,</span><span class="mi" id="MathJax-Span-143" style="font-family: MathJax_Math; font-style: italic; padding-left: 0.167em;">j</span></span><span class="mo" id="MathJax-Span-144" style=""><span style="font-family: MathJax_Main;">)</span></span></span><span class="mo MathClass-rel" id="MathJax-Span-145" style="font-family: MathJax_Main; padding-left: 0.278em;">=</span><span class="mo qopname" id="MathJax-Span-146" style="font-family: MathJax_Main; padding-left: 0.278em;">exp</span><span class="mfenced" id="MathJax-Span-147" style="padding-left: 0.167em;"><span class="mo" id="MathJax-Span-148" style=""><span style="font-family: MathJax_Main;">(</span></span><span class="mrow" id="MathJax-Span-149"><span class="mo MathClass-bin" id="MathJax-Span-150" style="font-family: MathJax_Main;">−</span><span class="mi" id="MathJax-Span-151" style="font-family: MathJax_Math; font-style: italic;">d<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span><span class="mfenced" id="MathJax-Span-152" style="padding-left: 0.167em;"><span class="mo" id="MathJax-Span-153" style=""><span style="font-family: MathJax_Main;">(</span></span><span class="mrow" id="MathJax-Span-154"><span class="msub" id="MathJax-Span-155"><span style="display: inline-block; position: relative; width: 1.147em; height: 0px;"><span style="position: absolute; clip: rect(3.144em, 1000.85em, 4.198em, -1000em); top: -4.012em; left: 0em;"><span class="mrow" id="MathJax-Span-156"><span class="mi" id="MathJax-Span-157" style="font-family: MathJax_Math; font-style: italic;">X<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.024em;"></span></span></span><span style="display: inline-block; width: 0px; height: 4.012em;"></span></span><span style="position: absolute; top: -3.862em; left: 0.828em;"><span class="mrow" id="MathJax-Span-158"><span class="mi" id="MathJax-Span-159" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">i</span></span><span style="display: inline-block; width: 0px; height: 4.012em;"></span></span></span></span><span class="mo MathClass-punc" id="MathJax-Span-160" style="font-family: MathJax_Main;">,</span><span class="msub" id="MathJax-Span-161" style="padding-left: 0.167em;"><span style="display: inline-block; position: relative; width: 1.194em; height: 0px;"><span style="position: absolute; clip: rect(3.144em, 1000.85em, 4.198em, -1000em); top: -4.012em; left: 0em;"><span class="mrow" id="MathJax-Span-162"><span class="mi" id="MathJax-Span-163" style="font-family: MathJax_Math; font-style: italic;">X<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.024em;"></span></span></span><span style="display: inline-block; width: 0px; height: 4.012em;"></span></span><span style="position: absolute; top: -3.862em; left: 0.828em;"><span class="mrow" id="MathJax-Span-164"><span class="mi" id="MathJax-Span-165" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">j</span></span><span style="display: inline-block; width: 0px; height: 4.012em;"></span></span></span></span></span><span class="mo" id="MathJax-Span-166" style=""><span style="font-family: MathJax_Main;">)</span></span></span><span class="mo" id="MathJax-Span-167" style="font-family: MathJax_Main;">/</span><span class="mi" id="MathJax-Span-168" style="font-family: MathJax_Math; font-style: italic;">θ</span></span><span class="mo" id="MathJax-Span-169" style=""><span style="font-family: MathJax_Main;">)</span></span></span><span class="mo MathClass-punc" id="MathJax-Span-170" style="font-family: MathJax_Main;">,</span><span class="mspace quad" id="MathJax-Span-171" style="height: 0em; vertical-align: 0em; width: 1em; display: inline-block; overflow: hidden;"></span><span class="mi" id="MathJax-Span-172" style="font-family: MathJax_Math; font-style: italic; padding-left: 0.167em;">θ</span><span class="mo MathClass-rel" id="MathJax-Span-173" style="font-family: MathJax_Main; padding-left: 0.278em;">&gt;</span><span class="mn" id="MathJax-Span-174" style="font-family: MathJax_Main; padding-left: 0.278em;">0</span><span class="mo MathClass-punc" id="MathJax-Span-175" style="font-family: MathJax_Main;">,</span></span></span><span style="display: inline-block; width: 0px; height: 4.012em;"></span></span></span><span style="display: inline-block; width: 0px; height: 4.012em;"></span></span><span style="position: absolute; clip: rect(3.306em, 1001.18em, 4.676em, -1000em); top: -4.012em; left: 19.3em;"><span style="display: inline-block; position: relative; width: 1.278em; height: 0px;"><span style="position: absolute; clip: rect(3.077em, 1001.18em, 4.448em, -1000em); top: -3.784em; left: 50%; margin-left: -0.639em;"><span class="mtd" id="MathJax-Span-176"><span class="mrow" id="MathJax-Span-177"><span class="mtext eqnarray" id="MathJax-Span-178" style="font-family: MathJax_Main;">(2)</span></span></span><span style="display: inline-block; width: 0px; height: 4.012em;"></span></span></span><span style="display: inline-block; width: 0px; height: 4.012em;"></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 2.16em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.9em; border-left: 0px solid; width: 0px; height: 2.339em;"></span></span></nobr><span class="MJX_Assistive_MathML" role="presentation"><math xmlns="http://www.w3.org/1998/Math/MathML" id="M28"><mtable><mtr><mtd><msub><mrow><mo mathsize="big">∑</mo></mrow><mrow><mi>θ</mi></mrow></msub><mfenced separators="" open="(" close=")"><mrow><mi>i</mi><mo class="MathClass-punc">,</mo><mi>j</mi></mrow></mfenced><mo class="MathClass-rel">=</mo><mo class="qopname">exp</mo><mfenced separators="" open="(" close=")"><mrow><mo class="MathClass-bin">-</mo><mi>d</mi><mfenced separators="" open="(" close=")"><mrow><msub><mrow><mi>X</mi></mrow><mrow><mi>i</mi></mrow></msub><mo class="MathClass-punc">,</mo><msub><mrow><mi>X</mi></mrow><mrow><mi>j</mi></mrow></msub></mrow></mfenced><mo>/</mo><mi>θ</mi></mrow></mfenced><mo class="MathClass-punc">,</mo><mspace class="quad" width="1em"></mspace><mi>θ</mi><mo class="MathClass-rel">&gt;</mo><mn>0</mn><mo class="MathClass-punc">,</mo></mtd><mtd><mtext class="eqnarray">(2)</mtext></mtd></mtr></mtable></math></span></span><script type="math/mml" id="MathJax-Element-4"><math id="M28"><mtable><mtr><mtd><msub><mrow><mo mathsize="big">∑</mo></mrow><mrow><mi>θ</mi></mrow></msub><mfenced separators="" open="(" close=")"><mrow><mi>i</mi><mo class="MathClass-punc">,</mo><mi>j</mi></mrow></mfenced><mo class="MathClass-rel">=</mo><mo class="qopname">exp</mo><mfenced separators="" open="(" close=")"><mrow><mo class="MathClass-bin">-</mo><mi>d</mi><mfenced separators="" open="(" close=")"><mrow><msub><mrow><mi>X</mi></mrow><mrow><mi>i</mi></mrow></msub><mo class="MathClass-punc">,</mo><msub><mrow><mi>X</mi></mrow><mrow><mi>j</mi></mrow></msub></mrow></mfenced><mo>/</mo><mi>θ</mi></mrow></mfenced><mo class="MathClass-punc">,</mo><mspace width="1em" class="quad"></mspace><mi>θ</mi><mo class="MathClass-rel">&gt;</mo><mn>0</mn><mo class="MathClass-punc">,</mo></mtd><mtd><mtext class="eqnarray">(2)</mtext></mtd></mtr></mtable></math></script>
         <div class="clear"></div>
<div class="DottedLine"></div>
      </div>
<p>where <i>d</i>(<i>X<sub>i</sub></i>, <i>X<sub>j</sub></i>) represents the squared Euclidean or great-circle distance between sites with coordinate <i>X<sub>i</sub></i> and with coordinate <i>X<sub>j</sub></i>.
 To avoid collinearity issues, we assumed that the individual 
geographical coordinates were distinct from each other (ties were broken
 by adding small perturbations to the original spatial coordinates). The
 parameter <i>θ</i> is a scale parameter measured in units of average pairwise distance between geographic sites, <span class="MathJax_Preview" style="color: inherit; display: none;"></span><span class="MathJax" id="MathJax-Element-5-Frame" tabindex="0" style="position: relative;" data-mathml="&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;M31&quot;&gt;&lt;mover accent=&quot;true&quot;&gt;&lt;mi&gt;d&lt;/mi&gt;&lt;mo&gt;&amp;#xAF;&lt;/mo&gt;&lt;/mover&gt;&lt;/math&gt;" role="presentation"><nobr aria-hidden="true"><span class="math" id="M31" style="width: 0.864em; display: inline-block;"><span style="display: inline-block; position: relative; width: 0.718em; height: 0px; font-size: 116%;"><span style="position: absolute; clip: rect(1.152em, 1000.65em, 2.309em, -1000em); top: -2.155em; left: 0em;"><span class="mrow" id="MathJax-Span-180"><span class="mover" id="MathJax-Span-181"><span style="display: inline-block; position: relative; width: 0.705em; height: 0px;"><span style="position: absolute; clip: rect(3.185em, 1000.52em, 4.177em, -1000em); top: -4.023em; left: 0em;"><span class="mi" id="MathJax-Span-182" style="font-family: MathJax_Math; font-style: italic;">d<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span><span style="display: inline-block; width: 0px; height: 4.023em;"></span></span><span style="position: absolute; clip: rect(3.289em, 1000.43em, 3.623em, -1000em); top: -4.293em; left: 0.205em;"><span class="mo" id="MathJax-Span-183" style=""><span style="font-family: MathJax_Main;">¯</span></span><span style="display: inline-block; width: 0px; height: 4.023em;"></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 2.155em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.067em; border-left: 0px solid; width: 0px; height: 1.12em;"></span></span></nobr><span class="MJX_Assistive_MathML" role="presentation"><math xmlns="http://www.w3.org/1998/Math/MathML" id="M31"><mover accent="true"><mi>d</mi><mo>¯</mo></mover></math></span></span><script type="math/mml" id="MathJax-Element-5"><math id="M31"><mover accent="true"><mi>d</mi><mo>¯</mo></mover></math></script> In practice, spFA requires that an array of <i>θ</i> values (scale parameter) are explored, so <i>θ</i> was varied in the range <span class="MathJax_Preview" style="color: inherit; display: none;"></span><span class="MathJax" id="MathJax-Element-6-Frame" tabindex="0" style="position: relative;" data-mathml="&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;M2&quot;&gt;&lt;mrow&gt;&lt;mrow&gt;&lt;mo class=&quot;MathClass-open&quot;&gt;(&lt;/mo&gt;&lt;mrow&gt;&lt;mn&gt;0&lt;/mn&gt;&lt;mo class=&quot;MathClass-punc&quot;&gt;,&lt;/mo&gt;&lt;mspace class=&quot;tmspace&quot; width=&quot;2.77695pt&quot; /&gt;&lt;mn&gt;10&lt;/mn&gt;&lt;mover accent=&quot;true&quot;&gt;&lt;mrow&gt;&lt;mi&gt;d&lt;/mi&gt;&lt;/mrow&gt;&lt;mo class=&quot;MathClass-op&quot;&gt;&amp;#x304;&lt;/mo&gt;&lt;/mover&gt;&lt;/mrow&gt;&lt;mo class=&quot;MathClass-close&quot;&gt;)&lt;/mo&gt;&lt;/mrow&gt;&lt;mo class=&quot;MathClass-punc&quot;&gt;.&lt;/mo&gt;&lt;/mrow&gt;&lt;/math&gt;" role="presentation"><nobr aria-hidden="true"><span class="math" id="M2" style="width: 4.887em; display: inline-block;"><span style="display: inline-block; position: relative; width: 4.215em; height: 0px; font-size: 116%;"><span style="position: absolute; clip: rect(1.774em, 1004.14em, 3.271em, -1000em); top: -2.778em; left: 0em;"><span class="mrow" id="MathJax-Span-185"><span class="mrow" id="MathJax-Span-186"><span class="mrow" id="MathJax-Span-187"><span class="mo MathClass-open" id="MathJax-Span-188" style="vertical-align: 0em;"><span style="font-family: MathJax_Size1;">(</span></span><span class="mrow" id="MathJax-Span-189"><span class="mn" id="MathJax-Span-190" style="font-family: MathJax_Main;">0</span><span class="mo MathClass-punc" id="MathJax-Span-191" style="font-family: MathJax_Main;">,</span><span class="mspace tmspace" id="MathJax-Span-192" style="height: 0em; vertical-align: 0em; width: 0.278em; display: inline-block; overflow: hidden;"></span><span class="mn" id="MathJax-Span-193" style="font-family: MathJax_Main; padding-left: 0.167em;">10</span><span class="mover" id="MathJax-Span-194"><span style="display: inline-block; position: relative; width: 0.636em; height: 0px;"><span style="position: absolute; clip: rect(3.185em, 1000.52em, 4.177em, -1000em); top: -4.023em; left: 0em;"><span class="mrow" id="MathJax-Span-195"><span class="mi" id="MathJax-Span-196" style="font-family: MathJax_Math; font-style: italic;">d<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span></span><span style="display: inline-block; width: 0px; height: 4.023em;"></span></span><span style="position: absolute; top: -4.293em; left: 0.275em;"><span style="height: 0em; vertical-align: 0em; width: 0.431em; display: inline-block; overflow: hidden;"></span><span class="mo MathClass-op" id="MathJax-Span-197" style="font-family: MathJax_Main;">̄&nbsp;<span style="height: 0em; vertical-align: 0em; margin-left: -0.239em;"></span></span><span style="display: inline-block; overflow: hidden; height: 1px; width: 0em;"></span><span style="display: inline-block; width: 0px; height: 4.023em;"></span></span></span></span></span><span class="mo MathClass-close" id="MathJax-Span-198" style="vertical-align: 0em;"><span style="font-family: MathJax_Size1;">)</span></span></span><span class="mo MathClass-punc" id="MathJax-Span-199" style="font-family: MathJax_Main; padding-left: 0.167em;">.</span></span></span><span style="display: inline-block; width: 0px; height: 2.778em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.461em; border-left: 0px solid; width: 0px; height: 1.514em;"></span></span></nobr><span class="MJX_Assistive_MathML" role="presentation"><math xmlns="http://www.w3.org/1998/Math/MathML" id="M2"><mrow><mrow><mo class="MathClass-open">(</mo><mrow><mn>0</mn><mo class="MathClass-punc">,</mo><mspace class="tmspace" width="2.77695pt"></mspace><mn>10</mn><mover accent="true"><mrow><mi>d</mi></mrow><mo class="MathClass-op">̄</mo></mover></mrow><mo class="MathClass-close">)</mo></mrow><mo class="MathClass-punc">.</mo></mrow></math></span></span><script type="math/mml" id="MathJax-Element-6"><math id="M2"><mrow><mrow><mo class="MathClass-open">(</mo><mrow><mn>0</mn><mo class="MathClass-punc">,</mo><mspace width="2.77695pt" class="tmspace"></mspace><mn>10</mn><mover accent="true"><mrow><mi>d</mi></mrow><mo class="MathClass-op">̄</mo></mover></mrow><mo class="MathClass-close">)</mo></mrow><mo class="MathClass-punc">.</mo></mrow></math></script></p>
   <p class="mb15">To solve the spFA model, we used a Cholesky decomposition, <span class="MathJax_Preview" style="color: inherit; display: none;"></span><span class="MathJax" id="MathJax-Element-7-Frame" tabindex="0" style="position: relative;" data-mathml="&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;M3&quot;&gt;&lt;mrow&gt;&lt;msup&gt;&lt;mrow&gt;&lt;mi&gt;C&lt;/mi&gt;&lt;/mrow&gt;&lt;mrow&gt;&lt;mi&gt;T&lt;/mi&gt;&lt;/mrow&gt;&lt;/msup&gt;&lt;mi&gt;C&lt;/mi&gt;&lt;mo class=&quot;MathClass-rel&quot;&gt;=&lt;/mo&gt;&lt;msubsup&gt;&lt;mrow&gt;&lt;mi&gt;&amp;#x3A3;&lt;/mi&gt;&lt;/mrow&gt;&lt;mrow&gt;&lt;mi&gt;&amp;#x3B8;&lt;/mi&gt;&lt;/mrow&gt;&lt;mrow&gt;&lt;mo class=&quot;MathClass-bin&quot;&gt;-&lt;/mo&gt;&lt;mn&gt;1&lt;/mn&gt;&lt;/mrow&gt;&lt;/msubsup&gt;&lt;mo class=&quot;MathClass-punc&quot;&gt;,&lt;/mo&gt;&lt;/mrow&gt;&lt;/math&gt;" role="presentation"><nobr aria-hidden="true"><span class="math" id="M3" style="width: 6.516em; display: inline-block;"><span style="display: inline-block; position: relative; width: 5.603em; height: 0px; font-size: 116%;"><span style="position: absolute; clip: rect(1.628em, 1005.54em, 3.159em, -1000em); top: -2.682em; left: 0em;"><span class="mrow" id="MathJax-Span-201"><span class="mrow" id="MathJax-Span-202"><span class="msup" id="MathJax-Span-203"><span style="display: inline-block; position: relative; width: 1.396em; height: 0px;"><span style="position: absolute; clip: rect(3.174em, 1000.76em, 4.189em, -1000em); top: -4.023em; left: 0em;"><span class="mrow" id="MathJax-Span-204"><span class="mi" id="MathJax-Span-205" style="font-family: MathJax_Math; font-style: italic;">C<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.045em;"></span></span></span><span style="display: inline-block; width: 0px; height: 4.023em;"></span></span><span style="position: absolute; top: -4.455em; left: 0.824em;"><span class="mrow" id="MathJax-Span-206"><span class="mi" id="MathJax-Span-207" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">T<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.085em;"></span></span></span><span style="display: inline-block; width: 0px; height: 4.023em;"></span></span></span></span><span class="mi" id="MathJax-Span-208" style="font-family: MathJax_Math; font-style: italic;">C<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.045em;"></span></span><span class="mo MathClass-rel" id="MathJax-Span-209" style="font-family: MathJax_Main; padding-left: 0.278em;">=</span><span class="msubsup" id="MathJax-Span-210" style="padding-left: 0.278em;"><span style="display: inline-block; position: relative; width: 1.842em; height: 0px;"><span style="position: absolute; clip: rect(3.196em, 1000.81em, 4.167em, -1000em); top: -4.023em; left: 0em;"><span class="mrow" id="MathJax-Span-211"><span class="mi" id="MathJax-Span-212" style="font-family: MathJax_Math; font-style: italic;">Σ<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.026em;"></span></span></span><span style="display: inline-block; width: 0px; height: 4.023em;"></span></span><span style="position: absolute; clip: rect(3.408em, 1000.98em, 4.225em, -1000em); top: -4.433em; left: 0.864em;"><span class="mrow" id="MathJax-Span-213"><span class="mo MathClass-bin" id="MathJax-Span-214" style="font-size: 70.7%; font-family: MathJax_Main;">−</span><span class="mn" id="MathJax-Span-215" style="font-size: 70.7%; font-family: MathJax_Main;">1</span></span><span style="display: inline-block; width: 0px; height: 4.023em;"></span></span><span style="position: absolute; clip: rect(3.381em, 1000.41em, 4.174em, -1000em); top: -3.696em; left: 0.78em;"><span class="mrow" id="MathJax-Span-216"><span class="mi" id="MathJax-Span-217" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">θ</span></span><span style="display: inline-block; width: 0px; height: 4.023em;"></span></span></span></span><span class="mo MathClass-punc" id="MathJax-Span-218" style="font-family: MathJax_Main;">,</span></span></span><span style="display: inline-block; width: 0px; height: 2.682em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.443em; border-left: 0px solid; width: 0px; height: 1.554em;"></span></span></nobr><span class="MJX_Assistive_MathML" role="presentation"><math xmlns="http://www.w3.org/1998/Math/MathML" id="M3"><mrow><msup><mrow><mi>C</mi></mrow><mrow><mi>T</mi></mrow></msup><mi>C</mi><mo class="MathClass-rel">=</mo><msubsup><mrow><mi>Σ</mi></mrow><mrow><mi>θ</mi></mrow><mrow><mo class="MathClass-bin">-</mo><mn>1</mn></mrow></msubsup><mo class="MathClass-punc">,</mo></mrow></math></span></span><script type="math/mml" id="MathJax-Element-7"><math id="M3"><mrow><msup><mrow><mi>C</mi></mrow><mrow><mi>T</mi></mrow></msup><mi>C</mi><mo class="MathClass-rel">=</mo><msubsup><mrow><mi>Σ</mi></mrow><mrow><mi>θ</mi></mrow><mrow><mo class="MathClass-bin">-</mo><mn>1</mn></mrow></msubsup><mo class="MathClass-punc">,</mo></mrow></math></script> and we established an equivalence with the following matrix factorization model</p>
<div style="font-size:100%" class="equationImageholder">
         <span class="MathJax_Preview" style="color: inherit; display: none;"></span><span class="MathJax" id="MathJax-Element-8-Frame" tabindex="0" style="position: relative;" data-mathml="&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;M29&quot;&gt;&lt;mtable&gt;&lt;mtr&gt;&lt;mtd&gt;&lt;msub&gt;&lt;mrow&gt;&lt;mover accent=&quot;true&quot;&gt;&lt;mrow&gt;&lt;mi&gt;G&lt;/mi&gt;&lt;/mrow&gt;&lt;mo class=&quot;MathClass-op&quot;&gt;&amp;#x303;&lt;/mo&gt;&lt;/mover&gt;&lt;/mrow&gt;&lt;mrow&gt;&lt;mi&gt;i&lt;/mi&gt;&lt;mi&gt;&amp;#x2113;&lt;/mi&gt;&lt;/mrow&gt;&lt;/msub&gt;&lt;mo class=&quot;MathClass-rel&quot;&gt;=&lt;/mo&gt;&lt;msubsup&gt;&lt;mrow&gt;&lt;mo class=&quot;MathClass-op&quot;&gt;&amp;#x168;&lt;/mo&gt;&lt;/mrow&gt;&lt;mrow&gt;&lt;mi&gt;i&lt;/mi&gt;&lt;/mrow&gt;&lt;mrow&gt;&lt;mi&gt;T&lt;/mi&gt;&lt;/mrow&gt;&lt;/msubsup&gt;&lt;msub&gt;&lt;mrow&gt;&lt;mo class=&quot;MathClass-op&quot;&gt;&amp;#x1E7C;&lt;/mo&gt;&lt;/mrow&gt;&lt;mrow&gt;&lt;mi&gt;&amp;#x2113;&lt;/mi&gt;&lt;/mrow&gt;&lt;/msub&gt;&lt;mo class=&quot;MathClass-bin&quot;&gt;+&lt;/mo&gt;&lt;msub&gt;&lt;mrow&gt;&lt;mover accent=&quot;true&quot;&gt;&lt;mrow&gt;&lt;mi&gt;&amp;#x3F5;&lt;/mi&gt;&lt;/mrow&gt;&lt;mo class=&quot;MathClass-op&quot;&gt;&amp;#x303;&lt;/mo&gt;&lt;/mover&gt;&lt;/mrow&gt;&lt;mrow&gt;&lt;mi&gt;i&lt;/mi&gt;&lt;mi&gt;&amp;#x2113;&lt;/mi&gt;&lt;/mrow&gt;&lt;/msub&gt;&lt;mo class=&quot;MathClass-punc&quot;&gt;,&lt;/mo&gt;&lt;/mtd&gt;&lt;mtd&gt;&lt;mtext class=&quot;eqnarray&quot;&gt;(3)&lt;/mtext&gt;&lt;/mtd&gt;&lt;/mtr&gt;&lt;/mtable&gt;&lt;/math&gt;" role="presentation"><nobr aria-hidden="true"><span class="math" id="M29" style="width: 10.929em; display: inline-block;"><span style="display: inline-block; position: relative; width: 10.123em; height: 0px; font-size: 108%;"><span style="position: absolute; clip: rect(1.095em, 1009.86em, 2.726em, -1000em); top: -2.16em; left: 0em;"><span class="mrow" id="MathJax-Span-220"><span class="mtable" id="MathJax-Span-221" style="padding-right: 0.167em; padding-left: 0.167em;"><span style="display: inline-block; position: relative; width: 9.782em; height: 0px;"><span style="position: absolute; clip: rect(2.947em, 1007.64em, 4.578em, -1000em); top: -4.012em; left: 0em;"><span style="display: inline-block; position: relative; width: 7.704em; height: 0px;"><span style="position: absolute; clip: rect(2.821em, 1007.64em, 4.453em, -1000em); top: -3.887em; left: 50%; margin-left: -3.852em;"><span class="mtd" id="MathJax-Span-222"><span class="mrow" id="MathJax-Span-223"><span class="msub" id="MathJax-Span-224"><span style="display: inline-block; position: relative; width: 1.4em; height: 0px;"><span style="position: absolute; clip: rect(2.843em, 1000.76em, 4.22em, -1000em); top: -4.012em; left: 0em;"><span class="mrow" id="MathJax-Span-225"><span class="mover" id="MathJax-Span-226"><span style="display: inline-block; position: relative; width: 0.786em; height: 0px;"><span style="position: absolute; clip: rect(3.122em, 1000.76em, 4.22em, -1000em); top: -4.012em; left: 0em;"><span class="mrow" id="MathJax-Span-227"><span class="mi" id="MathJax-Span-228" style="font-family: MathJax_Math; font-style: italic;">G</span></span><span style="display: inline-block; width: 0px; height: 4.012em;"></span></span><span style="position: absolute; clip: rect(3.105em, 1000.55em, 3.601em, -1000em); top: -4.275em; left: 0.198em;"><span class="mo MathClass-op" id="MathJax-Span-229" style=""><span style="font-family: MathJax_Size1;">˜</span></span><span style="display: inline-block; width: 0px; height: 4.012em;"></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 4.012em;"></span></span><span style="position: absolute; top: -3.858em; left: 0.786em;"><span class="mrow" id="MathJax-Span-230"><span class="mi" id="MathJax-Span-231" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">i</span><span class="mi" id="MathJax-Span-232" style="font-size: 70.7%; font-family: MathJax_Main;">ℓ</span></span><span style="display: inline-block; width: 0px; height: 4.012em;"></span></span></span></span><span class="mo MathClass-rel" id="MathJax-Span-233" style="font-family: MathJax_Main; padding-left: 0.278em;">=</span><span class="msubsup" id="MathJax-Span-234" style="padding-left: 0.278em;"><span style="display: inline-block; position: relative; width: 1.252em; height: 0px;"><span style="position: absolute; clip: rect(1.484em, 1000.68em, 2.854em, -1000em); top: -2.469em; left: 0em;"><span class="mrow" id="MathJax-Span-235"><span class="mo MathClass-op" id="MathJax-Span-236"><span style="font-family: STIXGeneral, &quot;Arial Unicode MS&quot;, serif; font-size: 93%; font-style: normal; font-weight: normal;">Ũ</span></span></span><span style="display: inline-block; width: 0px; height: 2.469em;"></span></span><span style="position: absolute; clip: rect(3.348em, 1000.57em, 4.198em, -1000em); top: -4.539em; left: 0.679em;"><span class="mrow" id="MathJax-Span-237"><span class="mi" id="MathJax-Span-238" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">T<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.085em;"></span></span></span><span style="display: inline-block; width: 0px; height: 4.012em;"></span></span><span style="position: absolute; clip: rect(3.36em, 1000.32em, 4.205em, -1000em); top: -3.765em; left: 0.679em;"><span class="mrow" id="MathJax-Span-239"><span class="mi" id="MathJax-Span-240" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">i</span></span><span style="display: inline-block; width: 0px; height: 4.012em;"></span></span></span></span><span class="msub" id="MathJax-Span-241"><span style="display: inline-block; position: relative; width: 1.049em; height: 0px;"><span style="position: absolute; clip: rect(1.484em, 1000.68em, 2.854em, -1000em); top: -2.469em; left: 0em;"><span class="mrow" id="MathJax-Span-242"><span class="mo MathClass-op" id="MathJax-Span-243"><span style="font-family: STIXGeneral, &quot;Arial Unicode MS&quot;, serif; font-size: 93%; font-style: normal; font-weight: normal;">Ṽ</span></span></span><span style="display: inline-block; width: 0px; height: 2.469em;"></span></span><span style="position: absolute; top: -3.777em; left: 0.679em;"><span class="mrow" id="MathJax-Span-244"><span class="mi" id="MathJax-Span-245" style="font-size: 70.7%; font-family: MathJax_Main;">ℓ</span></span><span style="display: inline-block; width: 0px; height: 4.012em;"></span></span></span></span><span class="mo MathClass-bin" id="MathJax-Span-246" style="font-family: MathJax_Main; padding-left: 0.222em;">+</span><span class="msub" id="MathJax-Span-247" style="padding-left: 0.222em;"><span style="display: inline-block; position: relative; width: 1.169em; height: 0px;"><span style="position: absolute; clip: rect(3.139em, 1000.47em, 4.209em, -1000em); top: -4.012em; left: 0em;"><span class="mrow" id="MathJax-Span-248"><span class="mover" id="MathJax-Span-249"><span style="display: inline-block; position: relative; width: 0.556em; height: 0px;"><span style="position: absolute; clip: rect(3.396em, 1000.38em, 4.209em, -1000em); top: -4.012em; left: 0.047em;"><span class="mrow" id="MathJax-Span-250"><span class="mi" id="MathJax-Span-251" style="font-family: MathJax_Math; font-style: italic;">ϵ</span></span><span style="display: inline-block; width: 0px; height: 4.012em;"></span></span><span style="position: absolute; clip: rect(3.159em, 1000.42em, 3.633em, -1000em); top: -4.033em; left: 0.056em;"><span class="mo MathClass-op" id="MathJax-Span-252" style=""><span style="font-family: MathJax_Main;">˜</span></span><span style="display: inline-block; width: 0px; height: 4.012em;"></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 4.012em;"></span></span><span style="position: absolute; top: -3.858em; left: 0.556em;"><span class="mrow" id="MathJax-Span-253"><span class="mi" id="MathJax-Span-254" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">i</span><span class="mi" id="MathJax-Span-255" style="font-size: 70.7%; font-family: MathJax_Main;">ℓ</span></span><span style="display: inline-block; width: 0px; height: 4.012em;"></span></span></span></span><span class="mo MathClass-punc" id="MathJax-Span-256" style="font-family: MathJax_Main;">,</span></span></span><span style="display: inline-block; width: 0px; height: 4.012em;"></span></span></span><span style="display: inline-block; width: 0px; height: 4.012em;"></span></span><span style="position: absolute; clip: rect(3.203em, 1001.18em, 4.573em, -1000em); top: -4.012em; left: 8.504em;"><span style="display: inline-block; position: relative; width: 1.278em; height: 0px;"><span style="position: absolute; clip: rect(3.077em, 1001.18em, 4.448em, -1000em); top: -3.887em; left: 50%; margin-left: -0.639em;"><span class="mtd" id="MathJax-Span-257"><span class="mrow" id="MathJax-Span-258"><span class="mtext eqnarray" id="MathJax-Span-259" style="font-family: MathJax_Main;">(3)</span></span></span><span style="display: inline-block; width: 0px; height: 4.012em;"></span></span></span><span style="display: inline-block; width: 0px; height: 4.012em;"></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 2.16em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.477em; border-left: 0px solid; width: 0px; height: 1.495em;"></span></span></nobr><span class="MJX_Assistive_MathML" role="presentation"><math xmlns="http://www.w3.org/1998/Math/MathML" id="M29"><mtable><mtr><mtd><msub><mrow><mover accent="true"><mrow><mi>G</mi></mrow><mo class="MathClass-op">̃</mo></mover></mrow><mrow><mi>i</mi><mi>ℓ</mi></mrow></msub><mo class="MathClass-rel">=</mo><msubsup><mrow><mo class="MathClass-op">Ũ</mo></mrow><mrow><mi>i</mi></mrow><mrow><mi>T</mi></mrow></msubsup><msub><mrow><mo class="MathClass-op">Ṽ</mo></mrow><mrow><mi>ℓ</mi></mrow></msub><mo class="MathClass-bin">+</mo><msub><mrow><mover accent="true"><mrow><mi>ϵ</mi></mrow><mo class="MathClass-op">̃</mo></mover></mrow><mrow><mi>i</mi><mi>ℓ</mi></mrow></msub><mo class="MathClass-punc">,</mo></mtd><mtd><mtext class="eqnarray">(3)</mtext></mtd></mtr></mtable></math></span></span><script type="math/mml" id="MathJax-Element-8"><math id="M29"><mtable><mtr><mtd><msub><mrow><mover accent="true"><mrow><mi>G</mi></mrow><mo class="MathClass-op">̃</mo></mover></mrow><mrow><mi>i</mi><mi>ℓ</mi></mrow></msub><mo class="MathClass-rel">=</mo><msubsup><mrow><mo class="MathClass-op">Ũ</mo></mrow><mrow><mi>i</mi></mrow><mrow><mi>T</mi></mrow></msubsup><msub><mrow><mo class="MathClass-op">Ṽ</mo></mrow><mrow><mi>ℓ</mi></mrow></msub><mo class="MathClass-bin">+</mo><msub><mrow><mover accent="true"><mrow><mi>ϵ</mi></mrow><mo class="MathClass-op">̃</mo></mover></mrow><mrow><mi>i</mi><mi>ℓ</mi></mrow></msub><mo class="MathClass-punc">,</mo></mtd><mtd><mtext class="eqnarray">(3)</mtext></mtd></mtr></mtable></math></script>
         <div class="clear"></div>
<div class="DottedLine"></div>
      </div>
<p>where <span class="MathJax_Preview" style="color: inherit; display: none;"></span><span class="MathJax" id="MathJax-Element-9-Frame" tabindex="0" style="position: relative;" data-mathml="&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;M4&quot;&gt;&lt;mrow&gt;&lt;mover accent=&quot;true&quot;&gt;&lt;mrow&gt;&lt;mi&gt;G&lt;/mi&gt;&lt;/mrow&gt;&lt;mo class=&quot;MathClass-op&quot;&gt;&amp;#x303;&lt;/mo&gt;&lt;/mover&gt;&lt;mo class=&quot;MathClass-rel&quot;&gt;=&lt;/mo&gt;&lt;mi&gt;C&lt;/mi&gt;&lt;mi&gt;G&lt;/mi&gt;&lt;mo class=&quot;MathClass-punc&quot;&gt;,&lt;/mo&gt;&lt;/mrow&gt;&lt;/math&gt;" role="presentation"><nobr aria-hidden="true"><span class="math" id="M4" style="width: 4.552em; display: inline-block;"><span style="display: inline-block; position: relative; width: 3.927em; height: 0px; font-size: 116%;"><span style="position: absolute; clip: rect(1.589em, 1003.86em, 3.02em, -1000em); top: -2.682em; left: 0em;"><span class="mrow" id="MathJax-Span-261"><span class="mrow" id="MathJax-Span-262"><span class="mover" id="MathJax-Span-263"><span style="display: inline-block; position: relative; width: 0.786em; height: 0px;"><span style="position: absolute; clip: rect(3.174em, 1000.76em, 4.189em, -1000em); top: -4.023em; left: 0em;"><span class="mrow" id="MathJax-Span-264"><span class="mi" id="MathJax-Span-265" style="font-family: MathJax_Math; font-style: italic;">G</span></span><span style="display: inline-block; width: 0px; height: 4.023em;"></span></span><span style="position: absolute; clip: rect(3.157em, 1000.55em, 3.57em, -1000em); top: -4.251em; left: 0.198em;"><span class="mo MathClass-op" id="MathJax-Span-266" style=""><span style="font-family: MathJax_Size1;">˜</span></span><span style="display: inline-block; width: 0px; height: 4.023em;"></span></span></span></span><span class="mo MathClass-rel" id="MathJax-Span-267" style="font-family: MathJax_Main; padding-left: 0.278em;">=</span><span class="mi" id="MathJax-Span-268" style="font-family: MathJax_Math; font-style: italic; padding-left: 0.278em;">C<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.045em;"></span></span><span class="mi" id="MathJax-Span-269" style="font-family: MathJax_Math; font-style: italic;">G</span><span class="mo MathClass-punc" id="MathJax-Span-270" style="font-family: MathJax_Main;">,</span></span></span><span style="display: inline-block; width: 0px; height: 2.682em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.281em; border-left: 0px solid; width: 0px; height: 1.438em;"></span></span></nobr><span class="MJX_Assistive_MathML" role="presentation"><math xmlns="http://www.w3.org/1998/Math/MathML" id="M4"><mrow><mover accent="true"><mrow><mi>G</mi></mrow><mo class="MathClass-op">̃</mo></mover><mo class="MathClass-rel">=</mo><mi>C</mi><mi>G</mi><mo class="MathClass-punc">,</mo></mrow></math></span></span><script type="math/mml" id="MathJax-Element-9"><math id="M4"><mrow><mover accent="true"><mrow><mi>G</mi></mrow><mo class="MathClass-op">̃</mo></mover><mo class="MathClass-rel">=</mo><mi>C</mi><mi>G</mi><mo class="MathClass-punc">,</mo></mrow></math></script><span class="MathJax_Preview" style="color: inherit; display: none;"></span><span class="MathJax" id="MathJax-Element-10-Frame" tabindex="0" style="position: relative;" data-mathml="&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;M5&quot;&gt;&lt;mrow&gt;&lt;mi&gt;&amp;#x168;&lt;/mi&gt;&lt;mo class=&quot;MathClass-rel&quot;&gt;=&lt;/mo&gt;&lt;mi&gt;C&lt;/mi&gt;&lt;mi&gt;U&lt;/mi&gt;&lt;mo class=&quot;MathClass-punc&quot;&gt;,&lt;/mo&gt;&lt;/mrow&gt;&lt;/math&gt;" role="presentation"><nobr aria-hidden="true"><span class="math" id="M5" style="width: 4.361em; display: inline-block;"><span style="display: inline-block; position: relative; width: 3.736em; height: 0px; font-size: 116%;"><span style="position: absolute; clip: rect(1.738em, 1003.67em, 3.026em, -1000em); top: -2.682em; left: 0em;"><span class="mrow" id="MathJax-Span-272"><span class="mrow" id="MathJax-Span-273"><span class="mi" id="MathJax-Span-274"><span style="font-family: STIXGeneral, &quot;Arial Unicode MS&quot;, serif; font-size: 86%; font-style: italic; font-weight: normal;">Ũ</span></span><span class="mo MathClass-rel" id="MathJax-Span-275" style="font-family: MathJax_Main; padding-left: 0.278em;">=</span><span class="mi" id="MathJax-Span-276" style="font-family: MathJax_Math; font-style: italic; padding-left: 0.278em;">C<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.045em;"></span></span><span class="mi" id="MathJax-Span-277" style="font-family: MathJax_Math; font-style: italic;">U<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.084em;"></span></span><span class="mo MathClass-punc" id="MathJax-Span-278" style="font-family: MathJax_Main;">,</span></span></span><span style="display: inline-block; width: 0px; height: 2.682em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.288em; border-left: 0px solid; width: 0px; height: 1.271em;"></span></span></nobr><span class="MJX_Assistive_MathML" role="presentation"><math xmlns="http://www.w3.org/1998/Math/MathML" id="M5"><mrow><mi>Ũ</mi><mo class="MathClass-rel">=</mo><mi>C</mi><mi>U</mi><mo class="MathClass-punc">,</mo></mrow></math></span></span><script type="math/mml" id="MathJax-Element-10"><math id="M5"><mrow><mi>Ũ</mi><mo class="MathClass-rel">=</mo><mi>C</mi><mi>U</mi><mo class="MathClass-punc">,</mo></mrow></math></script><span class="MathJax_Preview" style="color: inherit; display: none;"></span><span class="MathJax" id="MathJax-Element-11-Frame" tabindex="0" style="position: relative;" data-mathml="&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;M6&quot;&gt;&lt;mrow&gt;&lt;mi&gt;&amp;#x1E7C;&lt;/mi&gt;&lt;mo class=&quot;MathClass-rel&quot;&gt;=&lt;/mo&gt;&lt;mi&gt;V&lt;/mi&gt;&lt;mo class=&quot;MathClass-punc&quot;&gt;,&lt;/mo&gt;&lt;/mrow&gt;&lt;/math&gt;" role="presentation"><nobr aria-hidden="true"><span class="math" id="M6" style="width: 3.451em; display: inline-block;"><span style="display: inline-block; position: relative; width: 2.969em; height: 0px; font-size: 116%;"><span style="position: absolute; clip: rect(1.738em, 1002.9em, 3.026em, -1000em); top: -2.682em; left: 0em;"><span class="mrow" id="MathJax-Span-280"><span class="mrow" id="MathJax-Span-281"><span class="mi" id="MathJax-Span-282"><span style="font-family: STIXGeneral, &quot;Arial Unicode MS&quot;, serif; font-size: 86%; font-style: italic; font-weight: normal;">Ṽ</span></span><span class="mo MathClass-rel" id="MathJax-Span-283" style="font-family: MathJax_Main; padding-left: 0.278em;">=</span><span class="mi" id="MathJax-Span-284" style="font-family: MathJax_Math; font-style: italic; text-rendering: optimizelegibility; padding-left: 0.278em;">V<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.186em;"></span></span><span class="mo MathClass-punc" id="MathJax-Span-285" style="font-family: MathJax_Main;">,</span></span></span><span style="display: inline-block; width: 0px; height: 2.682em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.288em; border-left: 0px solid; width: 0px; height: 1.271em;"></span></span></nobr><span class="MJX_Assistive_MathML" role="presentation"><math xmlns="http://www.w3.org/1998/Math/MathML" id="M6"><mrow><mi>Ṽ</mi><mo class="MathClass-rel">=</mo><mi>V</mi><mo class="MathClass-punc">,</mo></mrow></math></span></span><script type="math/mml" id="MathJax-Element-11"><math id="M6"><mrow><mi>Ṽ</mi><mo class="MathClass-rel">=</mo><mi>V</mi><mo class="MathClass-punc">,</mo></mrow></math></script> and where <span class="MathJax_Preview" style="color: inherit; display: none;"></span><span class="MathJax" id="MathJax-Element-12-Frame" tabindex="0" style="position: relative;" data-mathml="&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;M7&quot;&gt;&lt;mrow&gt;&lt;msub&gt;&lt;mrow&gt;&lt;mover accent=&quot;true&quot;&gt;&lt;mrow&gt;&lt;mi&gt;&amp;#x3F5;&lt;/mi&gt;&lt;/mrow&gt;&lt;mo class=&quot;MathClass-op&quot;&gt;&amp;#x303;&lt;/mo&gt;&lt;/mover&gt;&lt;/mrow&gt;&lt;mrow&gt;&lt;mi&gt;&amp;#x2113;&lt;/mi&gt;&lt;/mrow&gt;&lt;/msub&gt;&lt;/mrow&gt;&lt;/math&gt;" role="presentation"><nobr aria-hidden="true"><span class="math" id="M7" style="width: 1.056em; display: inline-block;"><span style="display: inline-block; position: relative; width: 0.91em; height: 0px; font-size: 116%;"><span style="position: absolute; clip: rect(1.358em, 1000.91em, 2.467em, -1000em); top: -2.155em; left: 0em;"><span class="mrow" id="MathJax-Span-287"><span class="mrow" id="MathJax-Span-288"><span class="msub" id="MathJax-Span-289"><span style="display: inline-block; position: relative; width: 0.925em; height: 0px;"><span style="position: absolute; clip: rect(3.226em, 1000.47em, 4.178em, -1000em); top: -4.023em; left: 0em;"><span class="mrow" id="MathJax-Span-290"><span class="mover" id="MathJax-Span-291"><span style="display: inline-block; position: relative; width: 0.556em; height: 0px;"><span style="position: absolute; clip: rect(3.448em, 1000.38em, 4.178em, -1000em); top: -4.023em; left: 0.047em;"><span class="mrow" id="MathJax-Span-292"><span class="mi" id="MathJax-Span-293" style="font-family: MathJax_Math; font-style: italic;">ϵ</span></span><span style="display: inline-block; width: 0px; height: 4.023em;"></span></span><span style="position: absolute; clip: rect(3.211em, 1000.42em, 3.602em, -1000em); top: -4.009em; left: 0.056em;"><span class="mo MathClass-op" id="MathJax-Span-294" style=""><span style="font-family: MathJax_Main;">˜</span></span><span style="display: inline-block; width: 0px; height: 4.023em;"></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 4.023em;"></span></span><span style="position: absolute; top: -3.869em; left: 0.556em;"><span class="mrow" id="MathJax-Span-295"><span class="mi" id="MathJax-Span-296" style="font-size: 70.7%; font-family: MathJax_Main;">ℓ</span></span><span style="display: inline-block; width: 0px; height: 4.023em;"></span></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 2.155em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.251em; border-left: 0px solid; width: 0px; height: 1.065em;"></span></span></nobr><span class="MJX_Assistive_MathML" role="presentation"><math xmlns="http://www.w3.org/1998/Math/MathML" id="M7"><mrow><msub><mrow><mover accent="true"><mrow><mi>ϵ</mi></mrow><mo class="MathClass-op">̃</mo></mover></mrow><mrow><mi>ℓ</mi></mrow></msub></mrow></math></span></span><script type="math/mml" id="MathJax-Element-12"><math id="M7"><mrow><msub><mrow><mover accent="true"><mrow><mi>ϵ</mi></mrow><mo class="MathClass-op">̃</mo></mover></mrow><mrow><mi>ℓ</mi></mrow></msub></mrow></math></script> are statistically independent Gaussian vectors of mean zero and covariance matrix equal to identity. The matrix <span class="MathJax_Preview" style="color: inherit; display: none;"></span><span class="MathJax" id="MathJax-Element-13-Frame" tabindex="0" style="position: relative;" data-mathml="&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;M8&quot;&gt;&lt;mrow&gt;&lt;mi&gt;&amp;#x168;&lt;/mi&gt;&lt;/mrow&gt;&lt;/math&gt;" role="presentation"><nobr aria-hidden="true"><span class="math" id="M8" style="width: 0.721em; display: inline-block;"><span style="display: inline-block; position: relative; width: 0.623em; height: 0px; font-size: 116%;"><span style="position: absolute; clip: rect(1.259em, 1000.62em, 2.547em, -1000em); top: -2.203em; left: 0em;"><span class="mrow" id="MathJax-Span-298"><span class="mrow" id="MathJax-Span-299"><span class="mi" id="MathJax-Span-300"><span style="font-family: STIXGeneral, &quot;Arial Unicode MS&quot;, serif; font-size: 86%; font-style: italic; font-weight: normal;">Ũ</span></span></span></span><span style="display: inline-block; width: 0px; height: 2.203em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.288em; border-left: 0px solid; width: 0px; height: 1.271em;"></span></span></nobr><span class="MJX_Assistive_MathML" role="presentation"><math xmlns="http://www.w3.org/1998/Math/MathML" id="M8"><mrow><mi>Ũ</mi></mrow></math></span></span><script type="math/mml" id="MathJax-Element-13"><math id="M8"><mrow><mi>Ũ</mi></mrow></math></script> and <span class="MathJax_Preview" style="color: inherit; display: none;"></span><span class="MathJax" id="MathJax-Element-14-Frame" tabindex="0" style="position: relative;" data-mathml="&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;M9&quot;&gt;&lt;mrow&gt;&lt;mi&gt;&amp;#x1E7C;&lt;/mi&gt;&lt;/mrow&gt;&lt;/math&gt;" role="presentation"><nobr aria-hidden="true"><span class="math" id="M9" style="width: 0.721em; display: inline-block;"><span style="display: inline-block; position: relative; width: 0.623em; height: 0px; font-size: 116%;"><span style="position: absolute; clip: rect(1.259em, 1000.62em, 2.547em, -1000em); top: -2.203em; left: 0em;"><span class="mrow" id="MathJax-Span-302"><span class="mrow" id="MathJax-Span-303"><span class="mi" id="MathJax-Span-304"><span style="font-family: STIXGeneral, &quot;Arial Unicode MS&quot;, serif; font-size: 86%; font-style: italic; font-weight: normal;">Ṽ</span></span></span></span><span style="display: inline-block; width: 0px; height: 2.203em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.288em; border-left: 0px solid; width: 0px; height: 1.271em;"></span></span></nobr><span class="MJX_Assistive_MathML" role="presentation"><math xmlns="http://www.w3.org/1998/Math/MathML" id="M9"><mrow><mi>Ṽ</mi></mrow></math></span></span><script type="math/mml" id="MathJax-Element-14"><math id="M9"><mrow><mi>Ṽ</mi></mrow></math></script> were obtained by applying a singular value decomposition of rank <i>K</i> to the transformed data matrix, <i>CG</i>. Then, <i>U</i> and <i>V</i> were obtained by applying a singular value decomposition of rank <i>K</i> to <span class="MathJax_Preview" style="color: inherit; display: none;"></span><span class="MathJax" id="MathJax-Element-15-Frame" tabindex="0" style="position: relative;" data-mathml="&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;M10&quot;&gt;&lt;mrow&gt;&lt;msup&gt;&lt;mrow&gt;&lt;mi&gt;C&lt;/mi&gt;&lt;/mrow&gt;&lt;mrow&gt;&lt;mo class=&quot;MathClass-bin&quot;&gt;-&lt;/mo&gt;&lt;mn&gt;1&lt;/mn&gt;&lt;/mrow&gt;&lt;/msup&gt;&lt;mi&gt;&amp;#x168;&lt;/mi&gt;&lt;mi&gt;&amp;#x1E7C;&lt;/mi&gt;&lt;/mrow&gt;&lt;/math&gt;" role="presentation"><nobr aria-hidden="true"><span class="math" id="M10" style="width: 3.594em; display: inline-block;"><span style="display: inline-block; position: relative; width: 3.065em; height: 0px; font-size: 116%;"><span style="position: absolute; clip: rect(1.156em, 1003.07em, 2.547em, -1000em); top: -2.203em; left: 0em;"><span class="mrow" id="MathJax-Span-306"><span class="mrow" id="MathJax-Span-307"><span class="msup" id="MathJax-Span-308"><span style="display: inline-block; position: relative; width: 1.802em; height: 0px;"><span style="position: absolute; clip: rect(3.174em, 1000.76em, 4.189em, -1000em); top: -4.023em; left: 0em;"><span class="mrow" id="MathJax-Span-309"><span class="mi" id="MathJax-Span-310" style="font-family: MathJax_Math; font-style: italic;">C<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.045em;"></span></span></span><span style="display: inline-block; width: 0px; height: 4.023em;"></span></span><span style="position: absolute; top: -4.455em; left: 0.824em;"><span class="mrow" id="MathJax-Span-311"><span class="mo MathClass-bin" id="MathJax-Span-312" style="font-size: 70.7%; font-family: MathJax_Main;">−</span><span class="mn" id="MathJax-Span-313" style="font-size: 70.7%; font-family: MathJax_Main;">1</span></span><span style="display: inline-block; width: 0px; height: 4.023em;"></span></span></span></span><span class="mi" id="MathJax-Span-314"><span style="font-family: STIXGeneral, &quot;Arial Unicode MS&quot;, serif; font-size: 86%; font-style: italic; font-weight: normal;">Ũ</span></span><span class="mi" id="MathJax-Span-315"><span style="font-family: STIXGeneral, &quot;Arial Unicode MS&quot;, serif; font-size: 86%; font-style: italic; font-weight: normal;">Ṽ</span></span></span></span><span style="display: inline-block; width: 0px; height: 2.203em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.288em; border-left: 0px solid; width: 0px; height: 1.39em;"></span></span></nobr><span class="MJX_Assistive_MathML" role="presentation"><math xmlns="http://www.w3.org/1998/Math/MathML" id="M10"><mrow><msup><mrow><mi>C</mi></mrow><mrow><mo class="MathClass-bin">-</mo><mn>1</mn></mrow></msup><mi>Ũ</mi><mi>Ṽ</mi></mrow></math></span></span><script type="math/mml" id="MathJax-Element-15"><math id="M10"><mrow><msup><mrow><mi>C</mi></mrow><mrow><mo class="MathClass-bin">-</mo><mn>1</mn></mrow></msup><mi>Ũ</mi><mi>Ṽ</mi></mrow></math></script>. To avoid multiple solutions, the orthogonality condition <i>VV<sup>T</sup></i> = <i>I<sub>K</sub></i>, where <i>I<sub>K</sub></i> is the identity matrix in <i>K</i> dimensions, was imposed to <i>V</i> (Figure <a href="#F1">1</a>). The time needed to compute spFA is the same order as the time needed to compute <i>K</i> scores and loadings for a standard PCA (<a href="#B21">Patterson et al., 2006</a>). For an example of implementation, see our R code<sup id="footnotesuper1"><a id="note1a"></a><a class="footnoteanchor" href="#note1">1</a></sup>.</p>
   <div class="DottedLine"></div>
   <div class="Imageheaders">FIGURE 1</div>
   <div class="FigureDesc">
      <a href="https://www.frontiersin.org/files/Articles/34644/fgene-03-00254-HTML/image_m/fgene-03-00254-g001.jpg" target="_blank">
         <img src="Frontiers%20|%20Correcting%20Principal%20Component%20Maps%20for%20Effects%20of%20Spatial%20Autocorrelation%20in%20Population%20Genetic%20Data%20|%20Genetics_fichiers/fgene-03-00254-g001.gif" id="F1" alt="www.frontiersin.org"></a>
      <p><strong>Figure 1. Algorithm for spFA</strong>. For a genotypic matrix <i>G</i> with individual geographic coordinates (<i>X<sub>i</sub></i>), and for scale parameter <i>θ</i> &gt; 0, the spFA steps summarize as follows.</p>
   </div>
   <div class="clear"></div>
   <div class="DottedLine"></div>
   <h3>Sparse Factor Analysis</h3>
   <p class="mb15">Sparse Factor Analysis (SFA) was introduced by <a href="#B8">Engelhardt and Stephens (2010)</a>
 as an alternative to admixture-based models, and this method can 
recapitulate the results of PCA when population structure is influenced 
by IBD patterns. To give a description of SFA, we considered a 
regression model of the following form</p>
<div style="font-size:100%" class="equationImageholder">
         <span class="MathJax_Preview" style="color: inherit; display: none;"></span><span class="MathJax" id="MathJax-Element-16-Frame" tabindex="0" style="position: relative;" data-mathml="&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;M30&quot;&gt;&lt;mtable&gt;&lt;mtr&gt;&lt;mtd&gt;&lt;msub&gt;&lt;mrow&gt;&lt;mi&gt;G&lt;/mi&gt;&lt;/mrow&gt;&lt;mrow&gt;&lt;mi&gt;i&lt;/mi&gt;&lt;mi&gt;&amp;#x2113;&lt;/mi&gt;&lt;/mrow&gt;&lt;/msub&gt;&lt;mo class=&quot;MathClass-rel&quot;&gt;=&lt;/mo&gt;&lt;msubsup&gt;&lt;mrow&gt;&lt;mi&gt;U&lt;/mi&gt;&lt;/mrow&gt;&lt;mrow&gt;&lt;mi&gt;i&lt;/mi&gt;&lt;/mrow&gt;&lt;mrow&gt;&lt;mi&gt;T&lt;/mi&gt;&lt;/mrow&gt;&lt;/msubsup&gt;&lt;msub&gt;&lt;mrow&gt;&lt;mi&gt;V&lt;/mi&gt;&lt;/mrow&gt;&lt;mrow&gt;&lt;mi&gt;&amp;#x2113;&lt;/mi&gt;&lt;/mrow&gt;&lt;/msub&gt;&lt;mo class=&quot;MathClass-bin&quot;&gt;+&lt;/mo&gt;&lt;msub&gt;&lt;mrow&gt;&lt;mi&gt;&amp;#x3F5;&lt;/mi&gt;&lt;/mrow&gt;&lt;mrow&gt;&lt;mi&gt;i&lt;/mi&gt;&lt;mi&gt;&amp;#x2113;&lt;/mi&gt;&lt;/mrow&gt;&lt;/msub&gt;&lt;/mtd&gt;&lt;mtd&gt;&lt;mtext class=&quot;eqnarray&quot;&gt;(4)&lt;/mtext&gt;&lt;/mtd&gt;&lt;/mtr&gt;&lt;/mtable&gt;&lt;/math&gt;" role="presentation"><nobr aria-hidden="true"><span class="math" id="M30" style="width: 10.559em; display: inline-block;"><span style="display: inline-block; position: relative; width: 9.753em; height: 0px; font-size: 108%;"><span style="position: absolute; clip: rect(1.153em, 1009.49em, 2.668em, -1000em); top: -2.16em; left: 0em;"><span class="mrow" id="MathJax-Span-317"><span class="mtable" id="MathJax-Span-318" style="padding-right: 0.167em; padding-left: 0.167em;"><span style="display: inline-block; position: relative; width: 9.421em; height: 0px;"><span style="position: absolute; clip: rect(3.005em, 1007.34em, 4.519em, -1000em); top: -4.012em; left: 0em;"><span style="display: inline-block; position: relative; width: 7.343em; height: 0px;"><span style="position: absolute; clip: rect(2.938em, 1007.34em, 4.453em, -1000em); top: -3.945em; left: 50%; margin-left: -3.672em;"><span class="mtd" id="MathJax-Span-319"><span class="mrow" id="MathJax-Span-320"><span class="msub" id="MathJax-Span-321"><span style="display: inline-block; position: relative; width: 1.4em; height: 0px;"><span style="position: absolute; clip: rect(3.122em, 1000.76em, 4.22em, -1000em); top: -4.012em; left: 0em;"><span class="mrow" id="MathJax-Span-322"><span class="mi" id="MathJax-Span-323" style="font-family: MathJax_Math; font-style: italic;">G</span></span><span style="display: inline-block; width: 0px; height: 4.012em;"></span></span><span style="position: absolute; top: -3.858em; left: 0.786em;"><span class="mrow" id="MathJax-Span-324"><span class="mi" id="MathJax-Span-325" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">i</span><span class="mi" id="MathJax-Span-326" style="font-size: 70.7%; font-family: MathJax_Main;">ℓ</span></span><span style="display: inline-block; width: 0px; height: 4.012em;"></span></span></span></span><span class="mo MathClass-rel" id="MathJax-Span-327" style="font-family: MathJax_Main; padding-left: 0.278em;">=</span><span class="msubsup" id="MathJax-Span-328" style="padding-left: 0.278em;"><span style="display: inline-block; position: relative; width: 1.415em; height: 0px;"><span style="position: absolute; clip: rect(3.144em, 1000.77em, 4.22em, -1000em); top: -4.012em; left: 0em;"><span class="mrow" id="MathJax-Span-329"><span class="mi" id="MathJax-Span-330" style="font-family: MathJax_Math; font-style: italic;">U<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.084em;"></span></span></span><span style="display: inline-block; width: 0px; height: 4.012em;"></span></span><span style="position: absolute; clip: rect(3.348em, 1000.57em, 4.198em, -1000em); top: -4.422em; left: 0.842em;"><span class="mrow" id="MathJax-Span-331"><span class="mi" id="MathJax-Span-332" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">T<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.085em;"></span></span></span><span style="display: inline-block; width: 0px; height: 4.012em;"></span></span><span style="position: absolute; clip: rect(3.36em, 1000.32em, 4.205em, -1000em); top: -3.765em; left: 0.683em;"><span class="mrow" id="MathJax-Span-333"><span class="mi" id="MathJax-Span-334" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">i</span></span><span style="display: inline-block; width: 0px; height: 4.012em;"></span></span></span></span><span class="msub" id="MathJax-Span-335"><span style="display: inline-block; position: relative; width: 0.953em; height: 0px;"><span style="position: absolute; clip: rect(3.144em, 1000.77em, 4.22em, -1000em); top: -4.012em; left: 0em;"><span class="mrow" id="MathJax-Span-336"><span class="mi" id="MathJax-Span-337" style="font-family: MathJax_Math; font-style: italic; text-rendering: optimizelegibility;">V<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.186em;"></span></span></span><span style="display: inline-block; width: 0px; height: 4.012em;"></span></span><span style="position: absolute; top: -3.858em; left: 0.583em;"><span class="mrow" id="MathJax-Span-338"><span class="mi" id="MathJax-Span-339" style="font-size: 70.7%; font-family: MathJax_Main;">ℓ</span></span><span style="display: inline-block; width: 0px; height: 4.012em;"></span></span></span></span><span class="mo MathClass-bin" id="MathJax-Span-340" style="font-family: MathJax_Main; padding-left: 0.222em;">+</span><span class="msub" id="MathJax-Span-341" style="padding-left: 0.222em;"><span style="display: inline-block; position: relative; width: 1.02em; height: 0px;"><span style="position: absolute; clip: rect(3.396em, 1000.38em, 4.209em, -1000em); top: -4.012em; left: 0em;"><span class="mrow" id="MathJax-Span-342"><span class="mi" id="MathJax-Span-343" style="font-family: MathJax_Math; font-style: italic;">ϵ</span></span><span style="display: inline-block; width: 0px; height: 4.012em;"></span></span><span style="position: absolute; top: -3.858em; left: 0.406em;"><span class="mrow" id="MathJax-Span-344"><span class="mi" id="MathJax-Span-345" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">i</span><span class="mi" id="MathJax-Span-346" style="font-size: 70.7%; font-family: MathJax_Main;">ℓ</span></span><span style="display: inline-block; width: 0px; height: 4.012em;"></span></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 4.012em;"></span></span></span><span style="display: inline-block; width: 0px; height: 4.012em;"></span></span><span style="position: absolute; clip: rect(3.144em, 1001.18em, 4.514em, -1000em); top: -4.012em; left: 8.143em;"><span style="display: inline-block; position: relative; width: 1.278em; height: 0px;"><span style="position: absolute; clip: rect(3.077em, 1001.18em, 4.448em, -1000em); top: -3.945em; left: 50%; margin-left: -0.639em;"><span class="mtd" id="MathJax-Span-347"><span class="mrow" id="MathJax-Span-348"><span class="mtext eqnarray" id="MathJax-Span-349" style="font-family: MathJax_Main;">(4)</span></span></span><span style="display: inline-block; width: 0px; height: 4.012em;"></span></span></span><span style="display: inline-block; width: 0px; height: 4.012em;"></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 2.16em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.414em; border-left: 0px solid; width: 0px; height: 1.369em;"></span></span></nobr><span class="MJX_Assistive_MathML" role="presentation"><math xmlns="http://www.w3.org/1998/Math/MathML" id="M30"><mtable><mtr><mtd><msub><mrow><mi>G</mi></mrow><mrow><mi>i</mi><mi>ℓ</mi></mrow></msub><mo class="MathClass-rel">=</mo><msubsup><mrow><mi>U</mi></mrow><mrow><mi>i</mi></mrow><mrow><mi>T</mi></mrow></msubsup><msub><mrow><mi>V</mi></mrow><mrow><mi>ℓ</mi></mrow></msub><mo class="MathClass-bin">+</mo><msub><mrow><mi>ϵ</mi></mrow><mrow><mi>i</mi><mi>ℓ</mi></mrow></msub></mtd><mtd><mtext class="eqnarray">(4)</mtext></mtd></mtr></mtable></math></span></span><script type="math/mml" id="MathJax-Element-16"><math id="M30"><mtable><mtr><mtd><msub><mrow><mi>G</mi></mrow><mrow><mi>i</mi><mi>ℓ</mi></mrow></msub><mo class="MathClass-rel">=</mo><msubsup><mrow><mi>U</mi></mrow><mrow><mi>i</mi></mrow><mrow><mi>T</mi></mrow></msubsup><msub><mrow><mi>V</mi></mrow><mrow><mi>ℓ</mi></mrow></msub><mo class="MathClass-bin">+</mo><msub><mrow><mi>ϵ</mi></mrow><mrow><mi>i</mi><mi>ℓ</mi></mrow></msub></mtd><mtd><mtext class="eqnarray">(4)</mtext></mtd></mtr></mtable></math></script>
         <div class="clear"></div>
<div class="DottedLine"></div>
      </div>
<p>in which the residual errors are independent Gaussian random variables, ε<i><sub>i,ℓ</sub></i> ∼ <i>N</i>(0,1/<i>ψ<sub>i</sub></i>), and where the prior distribution on the precision parameter, <i>ψ<sub>i</sub></i>, is a Gamma distribution. In the SFA model, an automatic relevance determination prior is considered for the score vectors, <span class="MathJax_Preview" style="color: inherit; display: none;"></span><span class="MathJax" id="MathJax-Element-17-Frame" tabindex="0" style="position: relative;" data-mathml="&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;M11&quot;&gt;&lt;mrow&gt;&lt;msub&gt;&lt;mrow&gt;&lt;mi&gt;U&lt;/mi&gt;&lt;/mrow&gt;&lt;mrow&gt;&lt;mi&gt;i&lt;/mi&gt;&lt;mi&gt;k&lt;/mi&gt;&lt;/mrow&gt;&lt;/msub&gt;&lt;mo class=&quot;MathClass-rel&quot;&gt;~&lt;/mo&gt;&lt;mi&gt;N&lt;/mi&gt;&lt;mrow&gt;&lt;mo class=&quot;MathClass-open&quot;&gt;(&lt;/mo&gt;&lt;mrow&gt;&lt;mn&gt;0&lt;/mn&gt;&lt;mo class=&quot;MathClass-punc&quot;&gt;,&lt;/mo&gt;&lt;msubsup&gt;&lt;mrow&gt;&lt;mi&gt;&amp;#x3C3;&lt;/mi&gt;&lt;/mrow&gt;&lt;mrow&gt;&lt;mi&gt;i&lt;/mi&gt;&lt;mi&gt;k&lt;/mi&gt;&lt;/mrow&gt;&lt;mrow&gt;&lt;mn&gt;2&lt;/mn&gt;&lt;/mrow&gt;&lt;/msubsup&gt;&lt;/mrow&gt;&lt;mo class=&quot;MathClass-close&quot;&gt;)&lt;/mo&gt;&lt;/mrow&gt;&lt;mo class=&quot;MathClass-punc&quot;&gt;,&lt;/mo&gt;&lt;/mrow&gt;&lt;/math&gt;" role="presentation"><nobr aria-hidden="true"><span class="math" id="M11" style="width: 7.569em; display: inline-block;"><span style="display: inline-block; position: relative; width: 6.513em; height: 0px; font-size: 116%;"><span style="position: absolute; clip: rect(1.785em, 1006.45em, 3.271em, -1000em); top: -2.778em; left: 0em;"><span class="mrow" id="MathJax-Span-351"><span class="mrow" id="MathJax-Span-352"><span class="msub" id="MathJax-Span-353"><span style="display: inline-block; position: relative; width: 1.37em; height: 0px;"><span style="position: absolute; clip: rect(3.196em, 1000.77em, 4.189em, -1000em); top: -4.023em; left: 0em;"><span class="mrow" id="MathJax-Span-354"><span class="mi" id="MathJax-Span-355" style="font-family: MathJax_Math; font-style: italic;">U<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.084em;"></span></span></span><span style="display: inline-block; width: 0px; height: 4.023em;"></span></span><span style="position: absolute; top: -3.873em; left: 0.683em;"><span class="mrow" id="MathJax-Span-356"><span class="mi" id="MathJax-Span-357" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">i</span><span class="mi" id="MathJax-Span-358" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">k</span></span><span style="display: inline-block; width: 0px; height: 4.023em;"></span></span></span></span><span class="mo MathClass-rel" id="MathJax-Span-359" style="font-family: MathJax_Main;">~</span><span class="mi" id="MathJax-Span-360" style="font-family: MathJax_Math; font-style: italic;">N<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.085em;"></span></span><span class="mrow" id="MathJax-Span-361" style="padding-left: 0.167em;"><span class="mo MathClass-open" id="MathJax-Span-362" style="vertical-align: 0em;"><span style="font-family: MathJax_Size1;">(</span></span><span class="mrow" id="MathJax-Span-363"><span class="mn" id="MathJax-Span-364" style="font-family: MathJax_Main;">0</span><span class="mo MathClass-punc" id="MathJax-Span-365" style="font-family: MathJax_Main;">,</span><span class="msubsup" id="MathJax-Span-366" style="padding-left: 0.167em;"><span style="display: inline-block; position: relative; width: 1.258em; height: 0px;"><span style="position: absolute; clip: rect(3.448em, 1000.57em, 4.178em, -1000em); top: -4.023em; left: 0em;"><span class="mrow" id="MathJax-Span-367"><span class="mi" id="MathJax-Span-368" style="font-family: MathJax_Math; font-style: italic;">σ<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.001em;"></span></span></span><span style="display: inline-block; width: 0px; height: 4.023em;"></span></span><span style="position: absolute; clip: rect(3.408em, 1000.43em, 4.167em, -1000em); top: -4.367em; left: 0.622em;"><span class="mrow" id="MathJax-Span-369"><span class="mn" id="MathJax-Span-370" style="font-size: 70.7%; font-family: MathJax_Main;">2</span></span><span style="display: inline-block; width: 0px; height: 4.023em;"></span></span><span style="position: absolute; clip: rect(3.389em, 1000.69em, 4.174em, -1000em); top: -3.697em; left: 0.571em;"><span class="mrow" id="MathJax-Span-371"><span class="mi" id="MathJax-Span-372" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">i</span><span class="mi" id="MathJax-Span-373" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">k</span></span><span style="display: inline-block; width: 0px; height: 4.023em;"></span></span></span></span></span><span class="mo MathClass-close" id="MathJax-Span-374" style="vertical-align: 0em;"><span style="font-family: MathJax_Size1;">)</span></span></span><span class="mo MathClass-punc" id="MathJax-Span-375" style="font-family: MathJax_Main; padding-left: 0.167em;">,</span></span></span><span style="display: inline-block; width: 0px; height: 2.778em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.461em; border-left: 0px solid; width: 0px; height: 1.502em;"></span></span></nobr><span class="MJX_Assistive_MathML" role="presentation"><math xmlns="http://www.w3.org/1998/Math/MathML" id="M11"><mrow><msub><mrow><mi>U</mi></mrow><mrow><mi>i</mi><mi>k</mi></mrow></msub><mo class="MathClass-rel">~</mo><mi>N</mi><mrow><mo class="MathClass-open">(</mo><mrow><mn>0</mn><mo class="MathClass-punc">,</mo><msubsup><mrow><mi>σ</mi></mrow><mrow><mi>i</mi><mi>k</mi></mrow><mrow><mn>2</mn></mrow></msubsup></mrow><mo class="MathClass-close">)</mo></mrow><mo class="MathClass-punc">,</mo></mrow></math></span></span><script type="math/mml" id="MathJax-Element-17"><math id="M11"><mrow><msub><mrow><mi>U</mi></mrow><mrow><mi>i</mi><mi>k</mi></mrow></msub><mo class="MathClass-rel">~</mo><mi>N</mi><mrow><mo class="MathClass-open">(</mo><mrow><mn>0</mn><mo class="MathClass-punc">,</mo><msubsup><mrow><mi>σ</mi></mrow><mrow><mi>i</mi><mi>k</mi></mrow><mrow><mn>2</mn></mrow></msubsup></mrow><mo class="MathClass-close">)</mo></mrow><mo class="MathClass-punc">,</mo></mrow></math></script> where some <span class="MathJax_Preview" style="color: inherit; display: none;"></span><span class="MathJax" id="MathJax-Element-18-Frame" tabindex="0" style="position: relative;" data-mathml="&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;M12&quot;&gt;&lt;mrow&gt;&lt;msubsup&gt;&lt;mrow&gt;&lt;mi&gt;&amp;#x3C3;&lt;/mi&gt;&lt;/mrow&gt;&lt;mrow&gt;&lt;mi&gt;i&lt;/mi&gt;&lt;mi&gt;k&lt;/mi&gt;&lt;/mrow&gt;&lt;mrow&gt;&lt;mn&gt;2&lt;/mn&gt;&lt;/mrow&gt;&lt;/msubsup&gt;&lt;/mrow&gt;&lt;/math&gt;" role="presentation"><nobr aria-hidden="true"><span class="math" id="M12" style="width: 1.439em; display: inline-block;"><span style="display: inline-block; position: relative; width: 1.245em; height: 0px; font-size: 116%;"><span style="position: absolute; clip: rect(1.196em, 1001.25em, 2.633em, -1000em); top: -2.155em; left: 0em;"><span class="mrow" id="MathJax-Span-377"><span class="mrow" id="MathJax-Span-378"><span class="msubsup" id="MathJax-Span-379"><span style="display: inline-block; position: relative; width: 1.258em; height: 0px;"><span style="position: absolute; clip: rect(3.448em, 1000.57em, 4.178em, -1000em); top: -4.023em; left: 0em;"><span class="mrow" id="MathJax-Span-380"><span class="mi" id="MathJax-Span-381" style="font-family: MathJax_Math; font-style: italic;">σ<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.001em;"></span></span></span><span style="display: inline-block; width: 0px; height: 4.023em;"></span></span><span style="position: absolute; clip: rect(3.408em, 1000.43em, 4.167em, -1000em); top: -4.367em; left: 0.622em;"><span class="mrow" id="MathJax-Span-382"><span class="mn" id="MathJax-Span-383" style="font-size: 70.7%; font-family: MathJax_Main;">2</span></span><span style="display: inline-block; width: 0px; height: 4.023em;"></span></span><span style="position: absolute; clip: rect(3.389em, 1000.69em, 4.174em, -1000em); top: -3.697em; left: 0.571em;"><span class="mrow" id="MathJax-Span-384"><span class="mi" id="MathJax-Span-385" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">i</span><span class="mi" id="MathJax-Span-386" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">k</span></span><span style="display: inline-block; width: 0px; height: 4.023em;"></span></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 2.155em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.443em; border-left: 0px solid; width: 0px; height: 1.444em;"></span></span></nobr><span class="MJX_Assistive_MathML" role="presentation"><math xmlns="http://www.w3.org/1998/Math/MathML" id="M12"><mrow><msubsup><mrow><mi>σ</mi></mrow><mrow><mi>i</mi><mi>k</mi></mrow><mrow><mn>2</mn></mrow></msubsup></mrow></math></span></span><script type="math/mml" id="MathJax-Element-18"><math id="M12"><mrow><msubsup><mrow><mi>σ</mi></mrow><mrow><mi>i</mi><mi>k</mi></mrow><mrow><mn>2</mn></mrow></msubsup></mrow></math></script> are constrained to be equal to zero. We implemented SFA using the code distributed in <a href="#B8">Engelhardt and Stephens (2010)</a>, and we used 1,000 iterations. Eigenvectors in spFA and in SFA are also referred to as <i>factors</i> or <i>axes</i>.</p>
   <h3>Simulated Data</h3>
   <p class="mb0">We generated simulated data for two diverging populations using coalescent models implemented in the computer program <i>ms</i> (<a href="#B11">Hudson, 2002</a>).
 In these models, each population was simulated according to a linear 
stepping-stone model with 50 demes. To reproduce the simulation settings
 of <a href="#B20">Novembre and Stephens (2008)</a>, the effective migration rate between pairs of adjacent demes was set to the value 4<i>Nm</i>
 = 1. The divergence time τ between the two populations was varied 
within the range of values τ = (0,100) measured in coalescent units. We 
sampled 100 individuals, one from each deme both side of a (fictive) 
geographic barrier. For each simulation, we evaluated Wilks’ Λ, a 
statistic used in multivariate analysis of variance to test whether 
there are differences between the means of identified groups of 
individuals on the combination of genotypes (<a href="#B17">Mardia et al., 1979</a>).</p>
   <a id="h4" class="reset-hash-position"></a><h2>Results</h2>
   <h3 class="pt0">Pure Isolation-by-Distance Patterns</h3>
   <p class="mb0">In a first series of experiments, we used simulations 
of one-dimensional stepping-stone models reproducing the patterns of IBD
 described in <a href="#B20">Novembre and Stephens (2008)</a>. In those 
simulated data, the divergence time between the two populations was thus
 set to τ = 0, and the populations were connected by recurrent gene flow
 (4<i>Nm</i> = 1). As expected from theoretical results for PCA and for other ordination methods (<a href="#B1">Ahmed et al., 1974</a>; <a href="#B6">Dray et al., 2006</a>; <a href="#B20">Novembre and Stephens, 2008</a>),
 the first PC maps displayed oscillating patterns. In addition, the 
frequency of oscillation increased as we examined axes of higher orders 
(Figure <a href="#F2">2</a>A). When we used sPCA, the first three positive components were almost identical to those obtained with PCA (not reported).</p>
   <div class="DottedLine"></div>
   <div class="Imageheaders">FIGURE 2</div>
   <div class="FigureDesc">
      <a href="https://www.frontiersin.org/files/Articles/34644/fgene-03-00254-HTML/image_m/fgene-03-00254-g002.jpg" target="_blank">
         <img src="Frontiers%20|%20Correcting%20Principal%20Component%20Maps%20for%20Effects%20of%20Spatial%20Autocorrelation%20in%20Population%20Genetic%20Data%20|%20Genetics_fichiers/fgene-03-00254-g002.gif" id="F2" alt="www.frontiersin.org"></a>
      <p><strong>Figure 2. PC and spFA factor maps for data simulated under an IBD model</strong>. <strong>(A)</strong> PC maps, <strong>(B)</strong> spFA factor maps for <span class="MathJax_Preview" style="color: inherit; display: none;"></span><span class="MathJax" id="MathJax-Element-19-Frame" tabindex="0" style="position: relative;" data-mathml="&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;M13&quot;&gt;&lt;mrow&gt;&lt;mi&gt;&amp;#x3B8;&lt;/mi&gt;&lt;mo class=&quot;MathClass-bin&quot;&gt;&amp;#x2215;&lt;/mo&gt;&lt;mover accent=&quot;true&quot;&gt;&lt;mrow&gt;&lt;mi&gt;d&lt;/mi&gt;&lt;/mrow&gt;&lt;mo class=&quot;MathClass-op&quot;&gt;&amp;#x304;&lt;/mo&gt;&lt;/mover&gt;&lt;mo class=&quot;MathClass-rel&quot;&gt;=&lt;/mo&gt;&lt;mn&gt;0&lt;/mn&gt;&lt;mo class=&quot;MathClass-punc&quot;&gt;.&lt;/mo&gt;&lt;mn&gt;1&lt;/mn&gt;&lt;mo class=&quot;MathClass-punc&quot;&gt;,&lt;/mo&gt;&lt;/mrow&gt;&lt;/math&gt;" role="presentation"><nobr aria-hidden="true"><span class="math" id="M13" style="width: 6.099em; display: inline-block;"><span style="display: inline-block; position: relative; width: 5.234em; height: 0px; font-size: 116%;"><span style="position: absolute; clip: rect(1.588em, 1005.17em, 3.156em, -1000em); top: -2.709em; left: 0em;"><span class="mrow" id="MathJax-Span-388"><span class="mrow" id="MathJax-Span-389"><span class="mi" id="MathJax-Span-390" style="font-family: MathJax_Math; font-style: italic;">θ</span><span class="mo MathClass-bin" id="MathJax-Span-391" style="vertical-align: 0.087em; padding-left: 0.222em;"><span style="font-family: MathJax_Size1;">/</span></span><span class="mover" id="MathJax-Span-392" style="padding-left: 0.222em;"><span style="display: inline-block; position: relative; width: 0.636em; height: 0px;"><span style="position: absolute; clip: rect(3.124em, 1000.52em, 4.197em, -1000em); top: -4.002em; left: 0em;"><span class="mrow" id="MathJax-Span-393"><span class="mi" id="MathJax-Span-394" style="font-family: MathJax_Math; font-style: italic;">d<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span></span><span style="display: inline-block; width: 0px; height: 4.002em;"></span></span><span style="position: absolute; top: -4.306em; left: 0.275em;"><span style="height: 0em; vertical-align: 0em; width: 0.431em; display: inline-block; overflow: hidden;"></span><span class="mo MathClass-op" id="MathJax-Span-395" style="font-family: MathJax_Main;">̄&nbsp;<span style="height: 0em; vertical-align: 0em; margin-left: -0.246em;"></span></span><span style="display: inline-block; overflow: hidden; height: 1px; width: 0em;"></span><span style="display: inline-block; width: 0px; height: 4.002em;"></span></span></span></span><span class="mo MathClass-rel" id="MathJax-Span-396" style="font-family: MathJax_Main; padding-left: 0.278em;">=</span><span class="mn" id="MathJax-Span-397" style="font-family: MathJax_Main; padding-left: 0.278em;">0</span><span class="mo MathClass-punc" id="MathJax-Span-398" style="font-family: MathJax_Main;">.</span><span class="mn" id="MathJax-Span-399" style="font-family: MathJax_Main; padding-left: 0.167em;">1</span><span class="mo MathClass-punc" id="MathJax-Span-400" style="font-family: MathJax_Main;">,</span></span></span><span style="display: inline-block; width: 0px; height: 2.709em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.375em; border-left: 0px solid; width: 0px; height: 1.534em;"></span></span></nobr><span class="MJX_Assistive_MathML" role="presentation"><math xmlns="http://www.w3.org/1998/Math/MathML" id="M13"><mrow><mi>θ</mi><mo class="MathClass-bin">∕</mo><mover accent="true"><mrow><mi>d</mi></mrow><mo class="MathClass-op">̄</mo></mover><mo class="MathClass-rel">=</mo><mn>0</mn><mo class="MathClass-punc">.</mo><mn>1</mn><mo class="MathClass-punc">,</mo></mrow></math></span></span><script type="math/mml" id="MathJax-Element-19"><math id="M13"><mrow><mi>θ</mi><mo class="MathClass-bin">∕</mo><mover accent="true"><mrow><mi>d</mi></mrow><mo class="MathClass-op">̄</mo></mover><mo class="MathClass-rel">=</mo><mn>0</mn><mo class="MathClass-punc">.</mo><mn>1</mn><mo class="MathClass-punc">,</mo></mrow></math></script><strong>(C)</strong> spFA factor maps for <span class="MathJax_Preview" style="color: inherit; display: none;"></span><span class="MathJax" id="MathJax-Element-20-Frame" tabindex="0" style="position: relative;" data-mathml="&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;M14&quot;&gt;&lt;mrow&gt;&lt;mi&gt;&amp;#x3B8;&lt;/mi&gt;&lt;mo class=&quot;MathClass-bin&quot;&gt;&amp;#x2215;&lt;/mo&gt;&lt;mover accent=&quot;true&quot;&gt;&lt;mrow&gt;&lt;mi&gt;d&lt;/mi&gt;&lt;/mrow&gt;&lt;mo class=&quot;MathClass-op&quot;&gt;&amp;#x304;&lt;/mo&gt;&lt;/mover&gt;&lt;mo class=&quot;MathClass-rel&quot;&gt;=&lt;/mo&gt;&lt;mn&gt;0&lt;/mn&gt;&lt;mo class=&quot;MathClass-punc&quot;&gt;.&lt;/mo&gt;&lt;mn&gt;2&lt;/mn&gt;&lt;mo class=&quot;MathClass-punc&quot;&gt;,&lt;/mo&gt;&lt;/mrow&gt;&lt;/math&gt;" role="presentation"><nobr aria-hidden="true"><span class="math" id="M14" style="width: 6.099em; display: inline-block;"><span style="display: inline-block; position: relative; width: 5.234em; height: 0px; font-size: 116%;"><span style="position: absolute; clip: rect(1.588em, 1005.17em, 3.156em, -1000em); top: -2.709em; left: 0em;"><span class="mrow" id="MathJax-Span-402"><span class="mrow" id="MathJax-Span-403"><span class="mi" id="MathJax-Span-404" style="font-family: MathJax_Math; font-style: italic;">θ</span><span class="mo MathClass-bin" id="MathJax-Span-405" style="vertical-align: 0.087em; padding-left: 0.222em;"><span style="font-family: MathJax_Size1;">/</span></span><span class="mover" id="MathJax-Span-406" style="padding-left: 0.222em;"><span style="display: inline-block; position: relative; width: 0.636em; height: 0px;"><span style="position: absolute; clip: rect(3.124em, 1000.52em, 4.197em, -1000em); top: -4.002em; left: 0em;"><span class="mrow" id="MathJax-Span-407"><span class="mi" id="MathJax-Span-408" style="font-family: MathJax_Math; font-style: italic;">d<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span></span><span style="display: inline-block; width: 0px; height: 4.002em;"></span></span><span style="position: absolute; top: -4.306em; left: 0.275em;"><span style="height: 0em; vertical-align: 0em; width: 0.431em; display: inline-block; overflow: hidden;"></span><span class="mo MathClass-op" id="MathJax-Span-409" style="font-family: MathJax_Main;">̄&nbsp;<span style="height: 0em; vertical-align: 0em; margin-left: -0.246em;"></span></span><span style="display: inline-block; overflow: hidden; height: 1px; width: 0em;"></span><span style="display: inline-block; width: 0px; height: 4.002em;"></span></span></span></span><span class="mo MathClass-rel" id="MathJax-Span-410" style="font-family: MathJax_Main; padding-left: 0.278em;">=</span><span class="mn" id="MathJax-Span-411" style="font-family: MathJax_Main; padding-left: 0.278em;">0</span><span class="mo MathClass-punc" id="MathJax-Span-412" style="font-family: MathJax_Main;">.</span><span class="mn" id="MathJax-Span-413" style="font-family: MathJax_Main; padding-left: 0.167em;">2</span><span class="mo MathClass-punc" id="MathJax-Span-414" style="font-family: MathJax_Main;">,</span></span></span><span style="display: inline-block; width: 0px; height: 2.709em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.375em; border-left: 0px solid; width: 0px; height: 1.534em;"></span></span></nobr><span class="MJX_Assistive_MathML" role="presentation"><math xmlns="http://www.w3.org/1998/Math/MathML" id="M14"><mrow><mi>θ</mi><mo class="MathClass-bin">∕</mo><mover accent="true"><mrow><mi>d</mi></mrow><mo class="MathClass-op">̄</mo></mover><mo class="MathClass-rel">=</mo><mn>0</mn><mo class="MathClass-punc">.</mo><mn>2</mn><mo class="MathClass-punc">,</mo></mrow></math></span></span><script type="math/mml" id="MathJax-Element-20"><math id="M14"><mrow><mi>θ</mi><mo class="MathClass-bin">∕</mo><mover accent="true"><mrow><mi>d</mi></mrow><mo class="MathClass-op">̄</mo></mover><mo class="MathClass-rel">=</mo><mn>0</mn><mo class="MathClass-punc">.</mo><mn>2</mn><mo class="MathClass-punc">,</mo></mrow></math></script> 
         <strong>(D)</strong> spFA factor maps for <span class="MathJax_Preview" style="color: inherit; display: none;"></span><span class="MathJax" id="MathJax-Element-21-Frame" tabindex="0" style="position: relative;" data-mathml="&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;M15&quot;&gt;&lt;mrow&gt;&lt;mi&gt;&amp;#x3B8;&lt;/mi&gt;&lt;mo class=&quot;MathClass-bin&quot;&gt;&amp;#x2215;&lt;/mo&gt;&lt;mover accent=&quot;true&quot;&gt;&lt;mrow&gt;&lt;mi&gt;d&lt;/mi&gt;&lt;/mrow&gt;&lt;mo class=&quot;MathClass-op&quot;&gt;&amp;#x304;&lt;/mo&gt;&lt;/mover&gt;&lt;mo class=&quot;MathClass-rel&quot;&gt;=&lt;/mo&gt;&lt;mn&gt;0&lt;/mn&gt;&lt;mo class=&quot;MathClass-punc&quot;&gt;.&lt;/mo&gt;&lt;mn&gt;3&lt;/mn&gt;&lt;mo class=&quot;MathClass-punc&quot;&gt;.&lt;/mo&gt;&lt;/mrow&gt;&lt;/math&gt;" role="presentation"><nobr aria-hidden="true"><span class="math" id="M15" style="width: 6.099em; display: inline-block;"><span style="display: inline-block; position: relative; width: 5.234em; height: 0px; font-size: 116%;"><span style="position: absolute; clip: rect(1.588em, 1005.16em, 3.156em, -1000em); top: -2.709em; left: 0em;"><span class="mrow" id="MathJax-Span-416"><span class="mrow" id="MathJax-Span-417"><span class="mi" id="MathJax-Span-418" style="font-family: MathJax_Math; font-style: italic;">θ</span><span class="mo MathClass-bin" id="MathJax-Span-419" style="vertical-align: 0.087em; padding-left: 0.222em;"><span style="font-family: MathJax_Size1;">/</span></span><span class="mover" id="MathJax-Span-420" style="padding-left: 0.222em;"><span style="display: inline-block; position: relative; width: 0.636em; height: 0px;"><span style="position: absolute; clip: rect(3.124em, 1000.52em, 4.197em, -1000em); top: -4.002em; left: 0em;"><span class="mrow" id="MathJax-Span-421"><span class="mi" id="MathJax-Span-422" style="font-family: MathJax_Math; font-style: italic;">d<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span></span><span style="display: inline-block; width: 0px; height: 4.002em;"></span></span><span style="position: absolute; top: -4.306em; left: 0.275em;"><span style="height: 0em; vertical-align: 0em; width: 0.431em; display: inline-block; overflow: hidden;"></span><span class="mo MathClass-op" id="MathJax-Span-423" style="font-family: MathJax_Main;">̄&nbsp;<span style="height: 0em; vertical-align: 0em; margin-left: -0.246em;"></span></span><span style="display: inline-block; overflow: hidden; height: 1px; width: 0em;"></span><span style="display: inline-block; width: 0px; height: 4.002em;"></span></span></span></span><span class="mo MathClass-rel" id="MathJax-Span-424" style="font-family: MathJax_Main; padding-left: 0.278em;">=</span><span class="mn" id="MathJax-Span-425" style="font-family: MathJax_Main; padding-left: 0.278em;">0</span><span class="mo MathClass-punc" id="MathJax-Span-426" style="font-family: MathJax_Main;">.</span><span class="mn" id="MathJax-Span-427" style="font-family: MathJax_Main; padding-left: 0.167em;">3</span><span class="mo MathClass-punc" id="MathJax-Span-428" style="font-family: MathJax_Main;">.</span></span></span><span style="display: inline-block; width: 0px; height: 2.709em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.375em; border-left: 0px solid; width: 0px; height: 1.534em;"></span></span></nobr><span class="MJX_Assistive_MathML" role="presentation"><math xmlns="http://www.w3.org/1998/Math/MathML" id="M15"><mrow><mi>θ</mi><mo class="MathClass-bin">∕</mo><mover accent="true"><mrow><mi>d</mi></mrow><mo class="MathClass-op">̄</mo></mover><mo class="MathClass-rel">=</mo><mn>0</mn><mo class="MathClass-punc">.</mo><mn>3</mn><mo class="MathClass-punc">.</mo></mrow></math></span></span><script type="math/mml" id="MathJax-Element-21"><math id="M15"><mrow><mi>θ</mi><mo class="MathClass-bin">∕</mo><mover accent="true"><mrow><mi>d</mi></mrow><mo class="MathClass-op">̄</mo></mover><mo class="MathClass-rel">=</mo><mn>0</mn><mo class="MathClass-punc">.</mo><mn>3</mn><mo class="MathClass-punc">.</mo></mrow></math></script></p>
   </div>
   <div class="clear"></div>
   <div class="DottedLine"></div>
   <p class="mb15 w100pc float_left mt15">Running spFA with <i>K</i> = 3 and with 3 distinct values of the scale parameter (<span class="MathJax_Preview" style="color: inherit; display: none;"></span><span class="MathJax" id="MathJax-Element-22-Frame" tabindex="0" style="position: relative;" data-mathml="&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;M16&quot;&gt;&lt;mrow&gt;&lt;mrow&gt;&lt;mi&gt;&amp;#x3B8;&lt;/mi&gt;&lt;mo&gt;/&lt;/mo&gt;&lt;mover accent=&quot;true&quot;&gt;&lt;mi&gt;d&lt;/mi&gt;&lt;mo&gt;&amp;#xAF;&lt;/mo&gt;&lt;/mover&gt;&lt;/mrow&gt;&lt;mo&gt;=&lt;/mo&gt;&lt;mn&gt;0.1&lt;/mn&gt;&lt;/mrow&gt;&lt;/math&gt;" role="presentation"><nobr aria-hidden="true"><span class="math" id="M16" style="width: 4.887em; display: inline-block;"><span style="display: inline-block; position: relative; width: 4.215em; height: 0px; font-size: 116%;"><span style="position: absolute; clip: rect(1.679em, 1004.14em, 3.076em, -1000em); top: -2.682em; left: 0em;"><span class="mrow" id="MathJax-Span-430"><span class="mrow" id="MathJax-Span-431"><span class="mrow" id="MathJax-Span-432"><span class="mi" id="MathJax-Span-433" style="font-family: MathJax_Math; font-style: italic;">θ</span><span class="mo" id="MathJax-Span-434" style="font-family: MathJax_Main;">/</span><span class="mover" id="MathJax-Span-435"><span style="display: inline-block; position: relative; width: 0.705em; height: 0px;"><span style="position: absolute; clip: rect(3.185em, 1000.52em, 4.177em, -1000em); top: -4.023em; left: 0em;"><span class="mi" id="MathJax-Span-436" style="font-family: MathJax_Math; font-style: italic;">d<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span><span style="display: inline-block; width: 0px; height: 4.023em;"></span></span><span style="position: absolute; clip: rect(3.289em, 1000.43em, 3.623em, -1000em); top: -4.293em; left: 0.205em;"><span class="mo" id="MathJax-Span-437" style=""><span style="font-family: MathJax_Main;">¯</span></span><span style="display: inline-block; width: 0px; height: 4.023em;"></span></span></span></span></span><span class="mo" id="MathJax-Span-438" style="font-family: MathJax_Main; padding-left: 0.278em;">=</span><span class="mn" id="MathJax-Span-439" style="font-family: MathJax_Main; padding-left: 0.278em;">0.1</span></span></span><span style="display: inline-block; width: 0px; height: 2.682em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.346em; border-left: 0px solid; width: 0px; height: 1.398em;"></span></span></nobr><span class="MJX_Assistive_MathML" role="presentation"><math xmlns="http://www.w3.org/1998/Math/MathML" id="M16"><mrow><mrow><mi>θ</mi><mo>/</mo><mover accent="true"><mi>d</mi><mo>¯</mo></mover></mrow><mo>=</mo><mn>0.1</mn></mrow></math></span></span><script type="math/mml" id="MathJax-Element-22"><math id="M16"><mrow><mrow><mi>θ</mi><mo>/</mo><mover accent="true"><mi>d</mi><mo>¯</mo></mover></mrow><mo>=</mo><mn>0.1</mn></mrow></math></script>, 0.2, and 0.3) led to different interpretations of the genetic data (Figures <a href="#F2">2</a>B–D). Gradually varying <i>θ</i>
 allowed us to evaluate the scales at which the IBD effects were 
apparent, and also allowed us to remove those effects sequentially. For <span class="MathJax_Preview" style="color: inherit; display: none;"></span><span class="MathJax" id="MathJax-Element-23-Frame" tabindex="0" style="position: relative;" data-mathml="&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;M17&quot;&gt;&lt;mrow&gt;&lt;mrow&gt;&lt;mi&gt;&amp;#x3B8;&lt;/mi&gt;&lt;mo&gt;/&lt;/mo&gt;&lt;mover accent=&quot;true&quot;&gt;&lt;mi&gt;d&lt;/mi&gt;&lt;mo&gt;&amp;#xAF;&lt;/mo&gt;&lt;/mover&gt;&lt;/mrow&gt;&lt;mo&gt;=&lt;/mo&gt;&lt;mn&gt;0.1&lt;/mn&gt;&lt;/mrow&gt;&lt;/math&gt;" role="presentation"><nobr aria-hidden="true"><span class="math" id="M17" style="width: 4.887em; display: inline-block;"><span style="display: inline-block; position: relative; width: 4.215em; height: 0px; font-size: 116%;"><span style="position: absolute; clip: rect(1.679em, 1004.14em, 3.076em, -1000em); top: -2.682em; left: 0em;"><span class="mrow" id="MathJax-Span-441"><span class="mrow" id="MathJax-Span-442"><span class="mrow" id="MathJax-Span-443"><span class="mi" id="MathJax-Span-444" style="font-family: MathJax_Math; font-style: italic;">θ</span><span class="mo" id="MathJax-Span-445" style="font-family: MathJax_Main;">/</span><span class="mover" id="MathJax-Span-446"><span style="display: inline-block; position: relative; width: 0.705em; height: 0px;"><span style="position: absolute; clip: rect(3.185em, 1000.52em, 4.177em, -1000em); top: -4.023em; left: 0em;"><span class="mi" id="MathJax-Span-447" style="font-family: MathJax_Math; font-style: italic;">d<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span><span style="display: inline-block; width: 0px; height: 4.023em;"></span></span><span style="position: absolute; clip: rect(3.289em, 1000.43em, 3.623em, -1000em); top: -4.293em; left: 0.205em;"><span class="mo" id="MathJax-Span-448" style=""><span style="font-family: MathJax_Main;">¯</span></span><span style="display: inline-block; width: 0px; height: 4.023em;"></span></span></span></span></span><span class="mo" id="MathJax-Span-449" style="font-family: MathJax_Main; padding-left: 0.278em;">=</span><span class="mn" id="MathJax-Span-450" style="font-family: MathJax_Main; padding-left: 0.278em;">0.1</span></span></span><span style="display: inline-block; width: 0px; height: 2.682em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.346em; border-left: 0px solid; width: 0px; height: 1.398em;"></span></span></nobr><span class="MJX_Assistive_MathML" role="presentation"><math xmlns="http://www.w3.org/1998/Math/MathML" id="M17"><mrow><mrow><mi>θ</mi><mo>/</mo><mover accent="true"><mi>d</mi><mo>¯</mo></mover></mrow><mo>=</mo><mn>0.1</mn></mrow></math></span></span><script type="math/mml" id="MathJax-Element-23"><math id="M17"><mrow><mrow><mi>θ</mi><mo>/</mo><mover accent="true"><mi>d</mi><mo>¯</mo></mover></mrow><mo>=</mo><mn>0.1</mn></mrow></math></script>,
 the maps corresponding to factor 1 and 2 displayed sinusoidal curves 
similar to PC1 and PC2, whereas the map for factor 3 was flat as 
expected if the effect of IBD is removed (Figure <a href="#F2">2</a>B). For <span class="MathJax_Preview" style="color: inherit; display: none;"></span><span class="MathJax" id="MathJax-Element-24-Frame" tabindex="0" style="position: relative;" data-mathml="&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;M18&quot;&gt;&lt;mrow&gt;&lt;mrow&gt;&lt;mi&gt;&amp;#x3B8;&lt;/mi&gt;&lt;mo&gt;/&lt;/mo&gt;&lt;mover accent=&quot;true&quot;&gt;&lt;mi&gt;d&lt;/mi&gt;&lt;mo&gt;&amp;#xAF;&lt;/mo&gt;&lt;/mover&gt;&lt;/mrow&gt;&lt;mo&gt;=&lt;/mo&gt;&lt;mn&gt;0.2&lt;/mn&gt;&lt;/mrow&gt;&lt;/math&gt;" role="presentation"><nobr aria-hidden="true"><span class="math" id="M18" style="width: 4.887em; display: inline-block;"><span style="display: inline-block; position: relative; width: 4.215em; height: 0px; font-size: 116%;"><span style="position: absolute; clip: rect(1.679em, 1004.16em, 3.076em, -1000em); top: -2.682em; left: 0em;"><span class="mrow" id="MathJax-Span-452"><span class="mrow" id="MathJax-Span-453"><span class="mrow" id="MathJax-Span-454"><span class="mi" id="MathJax-Span-455" style="font-family: MathJax_Math; font-style: italic;">θ</span><span class="mo" id="MathJax-Span-456" style="font-family: MathJax_Main;">/</span><span class="mover" id="MathJax-Span-457"><span style="display: inline-block; position: relative; width: 0.705em; height: 0px;"><span style="position: absolute; clip: rect(3.185em, 1000.52em, 4.177em, -1000em); top: -4.023em; left: 0em;"><span class="mi" id="MathJax-Span-458" style="font-family: MathJax_Math; font-style: italic;">d<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span><span style="display: inline-block; width: 0px; height: 4.023em;"></span></span><span style="position: absolute; clip: rect(3.289em, 1000.43em, 3.623em, -1000em); top: -4.293em; left: 0.205em;"><span class="mo" id="MathJax-Span-459" style=""><span style="font-family: MathJax_Main;">¯</span></span><span style="display: inline-block; width: 0px; height: 4.023em;"></span></span></span></span></span><span class="mo" id="MathJax-Span-460" style="font-family: MathJax_Main; padding-left: 0.278em;">=</span><span class="mn" id="MathJax-Span-461" style="font-family: MathJax_Main; padding-left: 0.278em;">0.2</span></span></span><span style="display: inline-block; width: 0px; height: 2.682em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.346em; border-left: 0px solid; width: 0px; height: 1.398em;"></span></span></nobr><span class="MJX_Assistive_MathML" role="presentation"><math xmlns="http://www.w3.org/1998/Math/MathML" id="M18"><mrow><mrow><mi>θ</mi><mo>/</mo><mover accent="true"><mi>d</mi><mo>¯</mo></mover></mrow><mo>=</mo><mn>0.2</mn></mrow></math></span></span><script type="math/mml" id="MathJax-Element-24"><math id="M18"><mrow><mrow><mi>θ</mi><mo>/</mo><mover accent="true"><mi>d</mi><mo>¯</mo></mover></mrow><mo>=</mo><mn>0.2</mn></mrow></math></script> the map corresponding to factor 1 remained similar to PC1, but the maps for factor 2 and factor 3 were flat (Figure <a href="#F2">2</a>C). For <span class="MathJax_Preview" style="color: inherit; display: none;"></span><span class="MathJax" id="MathJax-Element-25-Frame" tabindex="0" style="position: relative;" data-mathml="&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;M19&quot;&gt;&lt;mrow&gt;&lt;mrow&gt;&lt;mi&gt;&amp;#x3B8;&lt;/mi&gt;&lt;mo&gt;/&lt;/mo&gt;&lt;mover accent=&quot;true&quot;&gt;&lt;mi&gt;d&lt;/mi&gt;&lt;mo&gt;&amp;#xAF;&lt;/mo&gt;&lt;/mover&gt;&lt;/mrow&gt;&lt;mo&gt;=&lt;/mo&gt;&lt;mn&gt;0.3&lt;/mn&gt;&lt;/mrow&gt;&lt;/math&gt;" role="presentation"><nobr aria-hidden="true"><span class="math" id="M19" style="width: 4.887em; display: inline-block;"><span style="display: inline-block; position: relative; width: 4.215em; height: 0px; font-size: 116%;"><span style="position: absolute; clip: rect(1.679em, 1004.17em, 3.076em, -1000em); top: -2.682em; left: 0em;"><span class="mrow" id="MathJax-Span-463"><span class="mrow" id="MathJax-Span-464"><span class="mrow" id="MathJax-Span-465"><span class="mi" id="MathJax-Span-466" style="font-family: MathJax_Math; font-style: italic;">θ</span><span class="mo" id="MathJax-Span-467" style="font-family: MathJax_Main;">/</span><span class="mover" id="MathJax-Span-468"><span style="display: inline-block; position: relative; width: 0.705em; height: 0px;"><span style="position: absolute; clip: rect(3.185em, 1000.52em, 4.177em, -1000em); top: -4.023em; left: 0em;"><span class="mi" id="MathJax-Span-469" style="font-family: MathJax_Math; font-style: italic;">d<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span><span style="display: inline-block; width: 0px; height: 4.023em;"></span></span><span style="position: absolute; clip: rect(3.289em, 1000.43em, 3.623em, -1000em); top: -4.293em; left: 0.205em;"><span class="mo" id="MathJax-Span-470" style=""><span style="font-family: MathJax_Main;">¯</span></span><span style="display: inline-block; width: 0px; height: 4.023em;"></span></span></span></span></span><span class="mo" id="MathJax-Span-471" style="font-family: MathJax_Main; padding-left: 0.278em;">=</span><span class="mn" id="MathJax-Span-472" style="font-family: MathJax_Main; padding-left: 0.278em;">0.3</span></span></span><span style="display: inline-block; width: 0px; height: 2.682em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.346em; border-left: 0px solid; width: 0px; height: 1.398em;"></span></span></nobr><span class="MJX_Assistive_MathML" role="presentation"><math xmlns="http://www.w3.org/1998/Math/MathML" id="M19"><mrow><mrow><mi>θ</mi><mo>/</mo><mover accent="true"><mi>d</mi><mo>¯</mo></mover></mrow><mo>=</mo><mn>0.3</mn></mrow></math></span></span><script type="math/mml" id="MathJax-Element-25"><math id="M19"><mrow><mrow><mi>θ</mi><mo>/</mo><mover accent="true"><mi>d</mi><mo>¯</mo></mover></mrow><mo>=</mo><mn>0.3</mn></mrow></math></script> the effects of isolation-by-distance were corrected in all axes (Figure <a href="#F2">2</a>D).</p>
   <p class="mb0">When we ran SFA with <i>K</i> = 3 factors, the 
resulting maps also emphasized aspects of the data different from the 
ones described by PC maps and spatial factor maps (Figure <a href="#F3">3</a>).
 Maps for SFA are interpreted in terms of clusters, similar to those 
obtained in non-spatial Bayesian assignment programs like <small>STRUCTURE</small> (<a href="#B24">Pritchard et al., 2000</a>). Clusters created by clustering programs under IBD models are often reported as being undesirable (<a href="#B10">François and Durand, 2010</a>; <a href="#B19">Meirmans, 2012</a>).</p>
   <div class="DottedLine"></div>
   <div class="Imageheaders">FIGURE 3</div>
   <div class="FigureDesc">
      <a href="https://www.frontiersin.org/files/Articles/34644/fgene-03-00254-HTML/image_m/fgene-03-00254-g003.jpg" target="_blank">
         <img src="Frontiers%20|%20Correcting%20Principal%20Component%20Maps%20for%20Effects%20of%20Spatial%20Autocorrelation%20in%20Population%20Genetic%20Data%20|%20Genetics_fichiers/fgene-03-00254-g003.gif" id="F3" alt="www.frontiersin.org"></a>
      <p><strong>Figure 3. SFA factor maps for data simulated under an IBD model</strong>. Plots of the first three Factor maps for SFA.</p>
   </div>
   <div class="clear"></div>
   <div class="DottedLine"></div>
   <h3>Two Diverging Populations with IBD Patterns</h3>
   <p class="mb15">In a second series of experiments, we used 
simulations of a two-population model, where each population consisted 
of a linear network of 50 demes. In these experiments, the two 
populations were separated by a geographic barrier to gene flow.</p>
   <p class="mb0">First the divergence time was set to τ = 10 coalescent
 units. Using PCA, the first 2 components displayed oscillating 
patterns, similar to those obtained with τ = 0 (pure IBD simulations; 
Figure <a href="#F4">4</a>A). The PC1-PC2 plot exhibited a clear 
horseshoe pattern. Differentiation between the two populations was 
visible in the PC1 map, where a discontinuity was observed at the center
 of the habitat. This discontinuity corresponded to the localization of 
the geographic barrier. Results for the positive eigenvectors of sPCA 
strongly resembled those obtained for the first PCs (Figure <a href="#F4">4</a>B).</p>
   <div class="DottedLine"></div>
   <div class="Imageheaders">FIGURE 4</div>
   <div class="FigureDesc">
      <a href="https://www.frontiersin.org/files/Articles/34644/fgene-03-00254-HTML/image_m/fgene-03-00254-g004.jpg" target="_blank">
         <img src="Frontiers%20|%20Correcting%20Principal%20Component%20Maps%20for%20Effects%20of%20Spatial%20Autocorrelation%20in%20Population%20Genetic%20Data%20|%20Genetics_fichiers/fgene-03-00254-g004.gif" id="F4" alt="www.frontiersin.org"></a>
      <p><strong>Figure 4. Two discrete populations under equilibrium IBD</strong>. Plots of the first 2 maps for <strong>(A)</strong> PCA, <strong>(B)</strong> sPCA, <strong>(C)</strong> spFA, <strong>(D)</strong> SFA.</p>
   </div>
   <div class="clear"></div>
   <div class="DottedLine"></div>
   <p class="mt15 w100pc float_left">Turning to spFA, we argued for a particular choice of <span class="MathJax_Preview" style="color: inherit; display: none;"></span><span class="MathJax" id="MathJax-Element-26-Frame" tabindex="0" style="position: relative;" data-mathml="&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;M20&quot;&gt;&lt;mrow&gt;&lt;mi&gt;&amp;#x3B8;&lt;/mi&gt;&lt;mo class=&quot;MathClass-bin&quot;&gt;&amp;#x2215;&lt;/mo&gt;&lt;mover accent=&quot;true&quot;&gt;&lt;mrow&gt;&lt;mi&gt;d&lt;/mi&gt;&lt;/mrow&gt;&lt;mo class=&quot;MathClass-op&quot;&gt;&amp;#x304;&lt;/mo&gt;&lt;/mover&gt;&lt;/mrow&gt;&lt;/math&gt;" role="presentation"><nobr aria-hidden="true"><span class="math" id="M20" style="width: 2.397em; display: inline-block;"><span style="display: inline-block; position: relative; width: 2.059em; height: 0px; font-size: 116%;"><span style="position: absolute; clip: rect(1.679em, 1001.95em, 3.051em, -1000em); top: -2.682em; left: 0em;"><span class="mrow" id="MathJax-Span-474"><span class="mrow" id="MathJax-Span-475"><span class="mi" id="MathJax-Span-476" style="font-family: MathJax_Math; font-style: italic;">θ</span><span class="mo MathClass-bin" id="MathJax-Span-477" style="vertical-align: 0.025em; padding-left: 0.222em;"><span style="font-family: MathJax_Main;">/</span></span><span class="mover" id="MathJax-Span-478" style="padding-left: 0.222em;"><span style="display: inline-block; position: relative; width: 0.636em; height: 0px;"><span style="position: absolute; clip: rect(3.185em, 1000.52em, 4.177em, -1000em); top: -4.023em; left: 0em;"><span class="mrow" id="MathJax-Span-479"><span class="mi" id="MathJax-Span-480" style="font-family: MathJax_Math; font-style: italic;">d<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span></span><span style="display: inline-block; width: 0px; height: 4.023em;"></span></span><span style="position: absolute; top: -4.293em; left: 0.275em;"><span style="height: 0em; vertical-align: 0em; width: 0.431em; display: inline-block; overflow: hidden;"></span><span class="mo MathClass-op" id="MathJax-Span-481" style="font-family: MathJax_Main;">̄&nbsp;<span style="height: 0em; vertical-align: 0em; margin-left: -0.239em;"></span></span><span style="display: inline-block; overflow: hidden; height: 1px; width: 0em;"></span><span style="display: inline-block; width: 0px; height: 4.023em;"></span></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 2.682em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.317em; border-left: 0px solid; width: 0px; height: 1.37em;"></span></span></nobr><span class="MJX_Assistive_MathML" role="presentation"><math xmlns="http://www.w3.org/1998/Math/MathML" id="M20"><mrow><mi>θ</mi><mo class="MathClass-bin">∕</mo><mover accent="true"><mrow><mi>d</mi></mrow><mo class="MathClass-op">̄</mo></mover></mrow></math></span></span><script type="math/mml" id="MathJax-Element-26"><math id="M20"><mrow><mi>θ</mi><mo class="MathClass-bin">∕</mo><mover accent="true"><mrow><mi>d</mi></mrow><mo class="MathClass-op">̄</mo></mover></mrow></math></script>
 based on Wilks’ Λ statistic, a standard measure of separation of groups
 in discriminant analysis, and computed this statistic for <span class="MathJax_Preview" style="color: inherit; display: none;"></span><span class="MathJax" id="MathJax-Element-27-Frame" tabindex="0" style="position: relative;" data-mathml="&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;M21&quot;&gt;&lt;mrow&gt;&lt;mi&gt;&amp;#x3B8;&lt;/mi&gt;&lt;mo class=&quot;MathClass-bin&quot;&gt;&amp;#x2215;&lt;/mo&gt;&lt;mover accent=&quot;true&quot;&gt;&lt;mrow&gt;&lt;mi&gt;d&lt;/mi&gt;&lt;/mrow&gt;&lt;mo class=&quot;MathClass-op&quot;&gt;&amp;#x304;&lt;/mo&gt;&lt;/mover&gt;&lt;/mrow&gt;&lt;/math&gt;" role="presentation"><nobr aria-hidden="true"><span class="math" id="M21" style="width: 2.397em; display: inline-block;"><span style="display: inline-block; position: relative; width: 2.059em; height: 0px; font-size: 116%;"><span style="position: absolute; clip: rect(1.679em, 1001.95em, 3.051em, -1000em); top: -2.682em; left: 0em;"><span class="mrow" id="MathJax-Span-483"><span class="mrow" id="MathJax-Span-484"><span class="mi" id="MathJax-Span-485" style="font-family: MathJax_Math; font-style: italic;">θ</span><span class="mo MathClass-bin" id="MathJax-Span-486" style="vertical-align: 0.025em; padding-left: 0.222em;"><span style="font-family: MathJax_Main;">/</span></span><span class="mover" id="MathJax-Span-487" style="padding-left: 0.222em;"><span style="display: inline-block; position: relative; width: 0.636em; height: 0px;"><span style="position: absolute; clip: rect(3.185em, 1000.52em, 4.177em, -1000em); top: -4.023em; left: 0em;"><span class="mrow" id="MathJax-Span-488"><span class="mi" id="MathJax-Span-489" style="font-family: MathJax_Math; font-style: italic;">d<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span></span><span style="display: inline-block; width: 0px; height: 4.023em;"></span></span><span style="position: absolute; top: -4.293em; left: 0.275em;"><span style="height: 0em; vertical-align: 0em; width: 0.431em; display: inline-block; overflow: hidden;"></span><span class="mo MathClass-op" id="MathJax-Span-490" style="font-family: MathJax_Main;">̄&nbsp;<span style="height: 0em; vertical-align: 0em; margin-left: -0.239em;"></span></span><span style="display: inline-block; overflow: hidden; height: 1px; width: 0em;"></span><span style="display: inline-block; width: 0px; height: 4.023em;"></span></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 2.682em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.317em; border-left: 0px solid; width: 0px; height: 1.37em;"></span></span></nobr><span class="MJX_Assistive_MathML" role="presentation"><math xmlns="http://www.w3.org/1998/Math/MathML" id="M21"><mrow><mi>θ</mi><mo class="MathClass-bin">∕</mo><mover accent="true"><mrow><mi>d</mi></mrow><mo class="MathClass-op">̄</mo></mover></mrow></math></span></span><script type="math/mml" id="MathJax-Element-27"><math id="M21"><mrow><mi>θ</mi><mo class="MathClass-bin">∕</mo><mover accent="true"><mrow><mi>d</mi></mrow><mo class="MathClass-op">̄</mo></mover></mrow></math></script>
 ranging between 0.01 and 10. As spatial factor analysis provided 
different interpretations of the data depending on the scale at which 
the data were analyzed, the choice of <i>θ</i> was crucial to the method. Figure <a href="#F5">5</a> reports the value of Wilks’ Λ as a function of the logarithm of <span class="MathJax_Preview" style="color: inherit; display: none;"></span><span class="MathJax" id="MathJax-Element-28-Frame" tabindex="0" style="position: relative;" data-mathml="&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;M22&quot;&gt;&lt;mrow&gt;&lt;mi&gt;&amp;#x3B8;&lt;/mi&gt;&lt;mo class=&quot;MathClass-bin&quot;&gt;&amp;#x2215;&lt;/mo&gt;&lt;mover accent=&quot;true&quot;&gt;&lt;mrow&gt;&lt;mi&gt;d&lt;/mi&gt;&lt;/mrow&gt;&lt;mo class=&quot;MathClass-op&quot;&gt;&amp;#x304;&lt;/mo&gt;&lt;/mover&gt;&lt;mo class=&quot;MathClass-punc&quot;&gt;.&lt;/mo&gt;&lt;/mrow&gt;&lt;/math&gt;" role="presentation"><nobr aria-hidden="true"><span class="math" id="M22" style="width: 2.732em; display: inline-block;"><span style="display: inline-block; position: relative; width: 2.347em; height: 0px; font-size: 116%;"><span style="position: absolute; clip: rect(1.726em, 1002.27em, 3.099em, -1000em); top: -2.73em; left: 0em;"><span class="mrow" id="MathJax-Span-492"><span class="mrow" id="MathJax-Span-493"><span class="mi" id="MathJax-Span-494" style="font-family: MathJax_Math; font-style: italic;">θ</span><span class="mo MathClass-bin" id="MathJax-Span-495" style="vertical-align: 0.025em; padding-left: 0.222em;"><span style="font-family: MathJax_Main;">/</span></span><span class="mover" id="MathJax-Span-496" style="padding-left: 0.222em;"><span style="display: inline-block; position: relative; width: 0.636em; height: 0px;"><span style="position: absolute; clip: rect(3.185em, 1000.52em, 4.177em, -1000em); top: -4.023em; left: 0em;"><span class="mrow" id="MathJax-Span-497"><span class="mi" id="MathJax-Span-498" style="font-family: MathJax_Math; font-style: italic;">d<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span></span><span style="display: inline-block; width: 0px; height: 4.023em;"></span></span><span style="position: absolute; top: -4.293em; left: 0.275em;"><span style="height: 0em; vertical-align: 0em; width: 0.431em; display: inline-block; overflow: hidden;"></span><span class="mo MathClass-op" id="MathJax-Span-499" style="font-family: MathJax_Main;">̄&nbsp;<span style="height: 0em; vertical-align: 0em; margin-left: -0.239em;"></span></span><span style="display: inline-block; overflow: hidden; height: 1px; width: 0em;"></span><span style="display: inline-block; width: 0px; height: 4.023em;"></span></span></span></span><span class="mo MathClass-punc" id="MathJax-Span-500" style="font-family: MathJax_Main;">.</span></span></span><span style="display: inline-block; width: 0px; height: 2.73em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.317em; border-left: 0px solid; width: 0px; height: 1.37em;"></span></span></nobr><span class="MJX_Assistive_MathML" role="presentation"><math xmlns="http://www.w3.org/1998/Math/MathML" id="M22"><mrow><mi>θ</mi><mo class="MathClass-bin">∕</mo><mover accent="true"><mrow><mi>d</mi></mrow><mo class="MathClass-op">̄</mo></mover><mo class="MathClass-punc">.</mo></mrow></math></span></span><script type="math/mml" id="MathJax-Element-28"><math id="M22"><mrow><mi>θ</mi><mo class="MathClass-bin">∕</mo><mover accent="true"><mrow><mi>d</mi></mrow><mo class="MathClass-op">̄</mo></mover><mo class="MathClass-punc">.</mo></mrow></math></script> Values of <span class="MathJax_Preview" style="color: inherit; display: none;"></span><span class="MathJax" id="MathJax-Element-29-Frame" tabindex="0" style="position: relative;" data-mathml="&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;M23&quot;&gt;&lt;mrow&gt;&lt;mi&gt;&amp;#x3B8;&lt;/mi&gt;&lt;mo class=&quot;MathClass-bin&quot;&gt;&amp;#x2215;&lt;/mo&gt;&lt;mover accent=&quot;true&quot;&gt;&lt;mrow&gt;&lt;mi&gt;d&lt;/mi&gt;&lt;/mrow&gt;&lt;mo class=&quot;MathClass-op&quot;&gt;&amp;#x304;&lt;/mo&gt;&lt;/mover&gt;&lt;/mrow&gt;&lt;/math&gt;" role="presentation"><nobr aria-hidden="true"><span class="math" id="M23" style="width: 2.397em; display: inline-block;"><span style="display: inline-block; position: relative; width: 2.059em; height: 0px; font-size: 116%;"><span style="position: absolute; clip: rect(1.679em, 1001.95em, 3.051em, -1000em); top: -2.682em; left: 0em;"><span class="mrow" id="MathJax-Span-502"><span class="mrow" id="MathJax-Span-503"><span class="mi" id="MathJax-Span-504" style="font-family: MathJax_Math; font-style: italic;">θ</span><span class="mo MathClass-bin" id="MathJax-Span-505" style="vertical-align: 0.025em; padding-left: 0.222em;"><span style="font-family: MathJax_Main;">/</span></span><span class="mover" id="MathJax-Span-506" style="padding-left: 0.222em;"><span style="display: inline-block; position: relative; width: 0.636em; height: 0px;"><span style="position: absolute; clip: rect(3.185em, 1000.52em, 4.177em, -1000em); top: -4.023em; left: 0em;"><span class="mrow" id="MathJax-Span-507"><span class="mi" id="MathJax-Span-508" style="font-family: MathJax_Math; font-style: italic;">d<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span></span><span style="display: inline-block; width: 0px; height: 4.023em;"></span></span><span style="position: absolute; top: -4.293em; left: 0.275em;"><span style="height: 0em; vertical-align: 0em; width: 0.431em; display: inline-block; overflow: hidden;"></span><span class="mo MathClass-op" id="MathJax-Span-509" style="font-family: MathJax_Main;">̄&nbsp;<span style="height: 0em; vertical-align: 0em; margin-left: -0.239em;"></span></span><span style="display: inline-block; overflow: hidden; height: 1px; width: 0em;"></span><span style="display: inline-block; width: 0px; height: 4.023em;"></span></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 2.682em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.317em; border-left: 0px solid; width: 0px; height: 1.37em;"></span></span></nobr><span class="MJX_Assistive_MathML" role="presentation"><math xmlns="http://www.w3.org/1998/Math/MathML" id="M23"><mrow><mi>θ</mi><mo class="MathClass-bin">∕</mo><mover accent="true"><mrow><mi>d</mi></mrow><mo class="MathClass-op">̄</mo></mover></mrow></math></span></span><script type="math/mml" id="MathJax-Element-29"><math id="M23"><mrow><mi>θ</mi><mo class="MathClass-bin">∕</mo><mover accent="true"><mrow><mi>d</mi></mrow><mo class="MathClass-op">̄</mo></mover></mrow></math></script> minimizing Wilks’ statistic and providing the best assignment of our data into clusters were about 0.32 (Figure <a href="#F5">5</a>). When spFA was applied with <i>K</i>
 = 2, the first factor map grouped demes at the left and the right of 
the geographic barrier in two main clusters, while simultaneously 
correcting for IBD patterns within the two clusters (Figure <a href="#F4">4</a>C).
 The spFA Axis1-Axis2 plot removed the horseshoe effect observed in PCA 
and sPCA plots. The resulting figure emphasized a discontinuous 
population structure consisting of two differentiated genetic clusters. 
Running SFA with <i>K</i> = 2 also led to a description of the data in 
two genetic clusters, located both sides of the geographic barrier, but 
the method failed to describe the two clusters as discontinuous entities
 (Figure <a href="#F4">4</a>D).</p>
   <div class="DottedLine"></div>
   <div class="Imageheaders">FIGURE 5</div>
   <div class="FigureDesc">
      <a href="https://www.frontiersin.org/files/Articles/34644/fgene-03-00254-HTML/image_m/fgene-03-00254-g005.jpg" target="_blank">
         <img src="Frontiers%20|%20Correcting%20Principal%20Component%20Maps%20for%20Effects%20of%20Spatial%20Autocorrelation%20in%20Population%20Genetic%20Data%20|%20Genetics_fichiers/fgene-03-00254-g005.gif" id="F5" alt="www.frontiersin.org"></a>
      <p><strong>Figure 5. Wilks’ Λ statistic as a function of the scale parameter <span class="MathJax_Preview" style="color: inherit; display: none;"></span><span class="MathJax" id="MathJax-Element-30-Frame" tabindex="0" style="position: relative;" data-mathml="&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;M24&quot;&gt;&lt;mrow&gt;&lt;mi&gt;&amp;#x3B8;&lt;/mi&gt;&lt;mo class=&quot;MathClass-bin&quot;&gt;&amp;#x2215;&lt;/mo&gt;&lt;mover accent=&quot;true&quot;&gt;&lt;mrow&gt;&lt;mi&gt;d&lt;/mi&gt;&lt;/mrow&gt;&lt;mo class=&quot;MathClass-op&quot;&gt;&amp;#x304;&lt;/mo&gt;&lt;/mover&gt;&lt;/mrow&gt;&lt;/math&gt;" role="presentation"><nobr aria-hidden="true"><span class="math" id="M24" style="width: 2.528em; display: inline-block;"><span style="display: inline-block; position: relative; width: 2.155em; height: 0px; font-size: 116%;"><span style="position: absolute; clip: rect(1.588em, 1002.04em, 3.156em, -1000em); top: -2.709em; left: 0em;"><span class="mrow" id="MathJax-Span-511"><span class="mrow" id="MathJax-Span-512"><span class="mi" id="MathJax-Span-513" style="font-family: MathJax_Math; font-style: italic;">θ</span><span class="mo MathClass-bin" id="MathJax-Span-514" style="vertical-align: 0.087em; padding-left: 0.222em;"><span style="font-family: MathJax_Size1;">/</span></span><span class="mover" id="MathJax-Span-515" style="padding-left: 0.222em;"><span style="display: inline-block; position: relative; width: 0.636em; height: 0px;"><span style="position: absolute; clip: rect(3.124em, 1000.52em, 4.197em, -1000em); top: -4.002em; left: 0em;"><span class="mrow" id="MathJax-Span-516"><span class="mi" id="MathJax-Span-517" style="font-family: MathJax_Math; font-style: italic;">d<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span></span><span style="display: inline-block; width: 0px; height: 4.002em;"></span></span><span style="position: absolute; top: -4.306em; left: 0.275em;"><span style="height: 0em; vertical-align: 0em; width: 0.431em; display: inline-block; overflow: hidden;"></span><span class="mo MathClass-op" id="MathJax-Span-518" style="font-family: MathJax_Main;">̄&nbsp;<span style="height: 0em; vertical-align: 0em; margin-left: -0.246em;"></span></span><span style="display: inline-block; overflow: hidden; height: 1px; width: 0em;"></span><span style="display: inline-block; width: 0px; height: 4.002em;"></span></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 2.709em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.375em; border-left: 0px solid; width: 0px; height: 1.534em;"></span></span></nobr><span class="MJX_Assistive_MathML" role="presentation"><math xmlns="http://www.w3.org/1998/Math/MathML" id="M24"><mrow><mi>θ</mi><mo class="MathClass-bin">∕</mo><mover accent="true"><mrow><mi>d</mi></mrow><mo class="MathClass-op">̄</mo></mover></mrow></math></span></span><script type="math/mml" id="MathJax-Element-30"><math id="M24"><mrow><mi>θ</mi><mo class="MathClass-bin">∕</mo><mover accent="true"><mrow><mi>d</mi></mrow><mo class="MathClass-op">̄</mo></mover></mrow></math></script> in spFA</strong>.</p>
   </div>
   <div class="clear"></div>
   <div class="DottedLine"></div>
   <p class="mt15 w100pc float_left">Based on PC and factor plots, we 
next computed Wilks’ Λ statistic for all methods, and for divergence 
times τ ranging between 0 and 100 (Figure <a href="#F6">6</a>). Lower 
values of Λ generally indicated better discrimination of the 2 divergent
 populations in PC or factor plots. For all methods, the Λ statistic 
decreased as the divergence time between the 2 populations increased (<a href="#B18">McVean, 2009</a>).
 In our spatially explicit framework, SFA (green curve) detected the 
existence of diverging populations earlier than PCA (red curve) and than
 sPCA (not shown, similar to PCA). SpFA was the most sensitive method, 
and provided an earlier detection of divergent clusters than SFA and PCA
 (blue curve).</p>
   <div class="DottedLine"></div>
   <div class="Imageheaders">FIGURE 6</div>
   <div class="FigureDesc">
      <a href="https://www.frontiersin.org/files/Articles/34644/fgene-03-00254-HTML/image_m/fgene-03-00254-g006.jpg" target="_blank">
         <img src="Frontiers%20|%20Correcting%20Principal%20Component%20Maps%20for%20Effects%20of%20Spatial%20Autocorrelation%20in%20Population%20Genetic%20Data%20|%20Genetics_fichiers/fgene-03-00254-g006.gif" id="F6" alt="www.frontiersin.org"></a>
      <p><strong>Figure 6. Wilks’ Λ statistic as a function of the divergence time, τ, ranging between 1 and 100</strong>.</p>
   </div>
   <div class="clear"></div>
   <div class="DottedLine"></div>
   <h3>Human Data Analysis</h3>
   <p class="mb0">Next we applied PCA, sPCA, spFA, and SFA to a 
worldwide sample of genomic DNA from 418 individuals in 27 Asian 
populations, from the Harvard Human Genome Diversity Project - Centre 
Etude Polymorphism Humain (Harvard HGDP-CEPH)<sup id="footnotesuper2"><a id="note2a"></a><a class="footnoteanchor" href="#note2">2</a></sup>.
 In those data, each marker has been ascertained in samples of Mongolian
 ancestry (referenced population HGDP01224). We selected all samples 
from Central and East-Asia with the exception of Xibe, which originated 
in northeastern China, but migrated to northwestern China only recently (<a href="#B23">Powell et al., 2007</a>) (Figure <a href="#F7">7</a>A). The data set used a panel of 10,664 SNPs<sup id="footnotesuper3"><a id="note3a"></a><a class="footnoteanchor" href="#note3">3</a></sup> (see <a href="#B22">Patterson et al., 2012</a>).</p>
   <div class="DottedLine"></div>
   <div class="Imageheaders">FIGURE 7</div>
   <div class="FigureDesc">
      <a href="https://www.frontiersin.org/files/Articles/34644/fgene-03-00254-HTML/image_m/fgene-03-00254-g007.jpg" target="_blank">
         <img src="Frontiers%20|%20Correcting%20Principal%20Component%20Maps%20for%20Effects%20of%20Spatial%20Autocorrelation%20in%20Population%20Genetic%20Data%20|%20Genetics_fichiers/fgene-03-00254-g007.gif" id="F7" alt="www.frontiersin.org"></a>
      <p><strong>Figure 7. (A)</strong> Map of Asia with geographic locations of HGDP populations. PC and factor plots for <strong>(B)</strong> PCA, <strong>(C)</strong> spFA, <strong>(D)</strong> SFA.</p>
   </div>
   <div class="clear"></div>
   <div class="DottedLine"></div>
   <p class="mt15 w100pc float_left">In our analysis, samples from 
Central Asia, west to the Tibetan plateau, were represented with 
red/orange colors, whereas populations from East-Asia were represented 
with blue colors (Figure <a href="#F7">7</a>A). For those samples, the 
PC plot exhibited a horseshoe pattern, which was a signature of the 
presence of IBD patterns in the data (Figure <a href="#F7">7</a>B). PCA led to a continuum of samples without observable genetic discontinuities. Running spFA with <i>K</i> = 2 and setting <span class="MathJax_Preview" style="color: inherit; display: none;"></span><span class="MathJax" id="MathJax-Element-31-Frame" tabindex="0" style="position: relative;" data-mathml="&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;M25&quot;&gt;&lt;mrow&gt;&lt;mi&gt;&amp;#x3B8;&lt;/mi&gt;&lt;mo class=&quot;MathClass-bin&quot;&gt;&amp;#x2215;&lt;/mo&gt;&lt;mover accent=&quot;true&quot;&gt;&lt;mrow&gt;&lt;mi&gt;d&lt;/mi&gt;&lt;/mrow&gt;&lt;mo class=&quot;MathClass-op&quot;&gt;&amp;#x304;&lt;/mo&gt;&lt;/mover&gt;&lt;mo class=&quot;MathClass-rel&quot;&gt;=&lt;/mo&gt;&lt;mn&gt;1&lt;/mn&gt;&lt;msup&gt;&lt;mrow&gt;&lt;mn&gt;0&lt;/mn&gt;&lt;/mrow&gt;&lt;mrow&gt;&lt;mo class=&quot;MathClass-bin&quot;&gt;-&lt;/mo&gt;&lt;mn&gt;2&lt;/mn&gt;&lt;/mrow&gt;&lt;/msup&gt;&lt;/mrow&gt;&lt;/math&gt;" role="presentation"><nobr aria-hidden="true"><span class="math" id="M25" style="width: 6.181em; display: inline-block;"><span style="display: inline-block; position: relative; width: 5.316em; height: 0px; font-size: 116%;"><span style="position: absolute; clip: rect(1.722em, 1005.32em, 3.098em, -1000em); top: -2.73em; left: 0em;"><span class="mrow" id="MathJax-Span-520"><span class="mrow" id="MathJax-Span-521"><span class="mi" id="MathJax-Span-522" style="font-family: MathJax_Math; font-style: italic;">θ</span><span class="mo MathClass-bin" id="MathJax-Span-523" style="vertical-align: 0.026em; padding-left: 0.222em;"><span style="font-family: MathJax_Main;">/</span></span><span class="mover" id="MathJax-Span-524" style="padding-left: 0.222em;"><span style="display: inline-block; position: relative; width: 0.636em; height: 0px;"><span style="position: absolute; clip: rect(3.185em, 1000.52em, 4.177em, -1000em); top: -4.023em; left: 0em;"><span class="mrow" id="MathJax-Span-525"><span class="mi" id="MathJax-Span-526" style="font-family: MathJax_Math; font-style: italic;">d<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.003em;"></span></span></span><span style="display: inline-block; width: 0px; height: 4.023em;"></span></span><span style="position: absolute; top: -4.293em; left: 0.275em;"><span style="height: 0em; vertical-align: 0em; width: 0.431em; display: inline-block; overflow: hidden;"></span><span class="mo MathClass-op" id="MathJax-Span-527" style="font-family: MathJax_Main;">̄&nbsp;<span style="height: 0em; vertical-align: 0em; margin-left: -0.239em;"></span></span><span style="display: inline-block; overflow: hidden; height: 1px; width: 0em;"></span><span style="display: inline-block; width: 0px; height: 4.023em;"></span></span></span></span><span class="mo MathClass-rel" id="MathJax-Span-528" style="font-family: MathJax_Main; padding-left: 0.278em;">=</span><span class="mn" id="MathJax-Span-529" style="font-family: MathJax_Main; padding-left: 0.278em;">1</span><span class="msup" id="MathJax-Span-530"><span style="display: inline-block; position: relative; width: 1.479em; height: 0px;"><span style="position: absolute; clip: rect(3.213em, 1000.46em, 4.189em, -1000em); top: -4.023em; left: 0em;"><span class="mrow" id="MathJax-Span-531"><span class="mn" id="MathJax-Span-532" style="font-family: MathJax_Main;">0</span></span><span style="display: inline-block; width: 0px; height: 4.023em;"></span></span><span style="position: absolute; top: -4.416em; left: 0.5em;"><span class="mrow" id="MathJax-Span-533"><span class="mo MathClass-bin" id="MathJax-Span-534" style="font-size: 70.7%; font-family: MathJax_Main;">−</span><span class="mn" id="MathJax-Span-535" style="font-size: 70.7%; font-family: MathJax_Main;">2</span></span><span style="display: inline-block; width: 0px; height: 4.023em;"></span></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 2.73em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.316em; border-left: 0px solid; width: 0px; height: 1.374em;"></span></span></nobr><span class="MJX_Assistive_MathML" role="presentation"><math xmlns="http://www.w3.org/1998/Math/MathML" id="M25"><mrow><mi>θ</mi><mo class="MathClass-bin">∕</mo><mover accent="true"><mrow><mi>d</mi></mrow><mo class="MathClass-op">̄</mo></mover><mo class="MathClass-rel">=</mo><mn>1</mn><msup><mrow><mn>0</mn></mrow><mrow><mo class="MathClass-bin">-</mo><mn>2</mn></mrow></msup></mrow></math></span></span><script type="math/mml" id="MathJax-Element-31"><math id="M25"><mrow><mi>θ</mi><mo class="MathClass-bin">∕</mo><mover accent="true"><mrow><mi>d</mi></mrow><mo class="MathClass-op">̄</mo></mover><mo class="MathClass-rel">=</mo><mn>1</mn><msup><mrow><mn>0</mn></mrow><mrow><mo class="MathClass-bin">-</mo><mn>2</mn></mrow></msup></mrow></math></script> on the basis of Wilks’ statistic analysis, spFA corrected for the effects of IBD in axes 1 and 2 (Figure <a href="#F7">7</a>C).
 The spFA method provided evidence of a major discontinuity separating 
two clusters, one in Central Asia and one in East-Asia. In addition, 
Uyghur and Hazara population samples aligned with the two main clusters 
and were placed in an intermediate position, suggesting genetic 
admixture from ancestral Central Asian and East-Asian gene pools. 
Essentially the same patterns emerged when spFA was applied with <i>K</i> = 3 at the same scale (Figures <a href="#F8">8</a>C,D).</p>
   <div class="DottedLine"></div>
   <div class="Imageheaders">FIGURE 8</div>
   <div class="FigureDesc">
      <a href="https://www.frontiersin.org/files/Articles/34644/fgene-03-00254-HTML/image_m/fgene-03-00254-g008.jpg" target="_blank">
         <img src="Frontiers%20|%20Correcting%20Principal%20Component%20Maps%20for%20Effects%20of%20Spatial%20Autocorrelation%20in%20Population%20Genetic%20Data%20|%20Genetics_fichiers/fgene-03-00254-g008.gif" id="F8" alt="www.frontiersin.org"></a>
      <p><strong>Figure 8. Factor plots for (A,B) SFA and (C,D) spFA with <i>K</i> = 3 clusters</strong>.</p>
   </div>
   <div class="clear"></div>
   <div class="DottedLine"></div>
   <p class="mb0 w100pc float_left mt15">Using SFA with <i>K</i> = 2, 
factors 1 and 2 confirmed the main discontinuity, in a representation of
 clusters closer to Bayesian clustering methods than to PCA (Figure <a href="#F7">7</a>D). Uyghur and Hazara population samples were also placed between the main clusters. When we used SFA with <i>K</i> = 3, we obtained shapes without natural interpretations (Figures <a href="#F8">8</a>A,B).
 SFA detected additional discontinuities whereas the other methods 
suggested that continuous genetic variation in geographic space was 
predominant.</p>
   <a id="h5" class="reset-hash-position"></a><h2>Discussion</h2>
   <p class="mb15">Principal component analysis and related methods used
 to describe genomic variation among large population samples are known 
to produce results that can be distorted by IBD, and that may thus be 
difficult to interpret. The horseshoe effect is one of the distortions 
observed in PC plots that arises when covariance between allele 
frequencies decays exponentially with geographic distance. In this case,
 there is an established mathematical correspondence between the 
eigenvectors of the covariance matrix and the columns of a discrete 
cosine-transform (<a href="#B1">Ahmed et al., 1974</a>; <a href="#B5">Diaconis et al., 2008</a>).
 In this study, we used this correspondence to propose a new approach 
based on spatial models for the covariance structure of residual errors 
in factor analysis. In spFA, IBD effects were modeled through the 
introduction of a covariance matrix that accounts for the geographic 
distance between individuals explicitly.</p>
   <p class="mb15">We compared spFA to PCA and to two recent methods 
that also attempt to correct for IBD effects: spatial Principal 
Component Analysis (sPCA, <a href="#B13">Jombart et al., 2008</a>) and sparse factor analysis (SFA, <a href="#B8">Engelhardt and Stephens, 2010</a>).
 When we applied PCA to simulated data from spatial coalescent models, 
PC maps displayed sinusoidal curves as observed in previous studies (<a href="#B20">Novembre and Stephens, 2008</a>).
 We observed that sPCA, which includes several distance matrices within 
Moran eigenvector maps of genetic data, produced results similar to 
those of PCA, and did not correct for IBD effects. When we applied SFA 
to spatial coalescent simulations, the algorithm clustered individuals 
in several small groups depending on the number of latent factors used 
in the method. SFA factor maps actually displayed outcomes closer to 
discrete clusters than to continuous variation. After adjusting for the 
spatial scale in the covariance model, spFA was able to remove the 
oscillating shapes observed in the first PCs sequentially.</p>
   <p class="mb15">When PCA was applied to spatially explicit 
simulations of two diverging populations, PC maps failed to firmly 
identify genetic discontinuities between populations. Despite a 
relatively long period of isolation in simulations, the populations were
 not strongly separated in PC maps due to the horseshoe effect. Compared
 to PCA and sPCA, the spFA method had increased power to identify 
genetic discontinuities where they were masked by spurious 
autocorrelation effects. When we applied SFA, we found that, up to 
normalization of outputs, the results were similar to those generated by
 clustering algorithms like <small>STRUCTURE</small>. For simulations of
 two diverging populations, SFA detected a main separation between two 
differentiated populations, but this approach did not correct for IBD 
effects within the main genetic clusters. Similarly to <small>STRUCTURE</small>,
 the results of SFA were influenced by the presence of IBD patterns in 
the samples. We found that spFA alleviated this issue, and that it 
produced results more robust to the choice of the number of factors than
 SFA.</p>
   <p class="mb15">The methods used in this study provided quite 
distinct descriptions of the data when they were applied to human 
population samples from Central and East-Asia, and they underlined 
several aspects of the data. With PCA, a typical horseshoe pattern was 
observed, but no obvious genetic discontinuities were observed. In 
contrast, SFA provided evidence for two main clusters which were also 
confirmed by spFA. When we used SFA with <i>K</i> = 3, we obtained shapes without natural interpretations (Figure <a href="#F8">8</a>).
 SFA detected additional discontinuities whereas the other methods 
suggested that continuous genetic variation in geographic space was 
predominant. We observed that SFA behaves like clustering algorithms and
 did not correct for spurious clusters created by IBD patterns. This 
issue makes the SFA results difficult to interpret in terms of admixture
 and ancestral populations. The spFA method corrected for the horseshoe 
pattern observed in PC plots by removing autocorrelation effects from 
the second and third axes. The method suggested that Asian population 
structure is strongly influenced by IBD patterns. In the spFA plot, 
Hazara of Pakistan and Uygur of northwestern China grouped together, and
 were placed between Pakistani and East-Asian populations (<a href="#B25">Rosenberg et al., 2002</a>).
 These results either support the presence of admixed genomes in Hazara 
and Uygur populations, or favor the hypothesis of a central Asian 
migration route of modern humans in East-Asia (<a href="#B29">Zhang et al., 2007</a>).
 The public availability of data sets other than the HGDP will enable us
 to further assess the utility of the method for analyzing human genetic
 data.</p>
   <p class="mb0">A potential limitation of the spFA approach is it’s sensitivity to the choice of the scale parameter, <i>θ</i>. The <i>θ</i>
 parameter actually determines the scale of the spatial effects that 
could be removed by spFA. Note that spFA is essentially performing a 
standard principal component analysis when it is applied with small 
values of the scale parameter. In this study, we recommended exploring a
 grid of <i>θ</i> values so that IBD effects could be removed at distinct scales sequentially. The choice of the number of factors, <i>K</i>, in spFA is also tied to the particular value of <i>θ</i> implemented in the model. One way to determine <i>K</i> is by using Tracy-Widom tests on the matrix of genotypes, <span class="MathJax_Preview" style="color: inherit; display: none;"></span><span class="MathJax" id="MathJax-Element-32-Frame" tabindex="0" style="position: relative;" data-mathml="&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot; id=&quot;M32&quot;&gt;&lt;mover accent=&quot;true&quot;&gt;&lt;mi&gt;G&lt;/mi&gt;&lt;mo&gt;&amp;#x2DC;&lt;/mo&gt;&lt;/mover&gt;&lt;/math&gt;" role="presentation"><nobr aria-hidden="true"><span class="math" id="M32" style="width: 0.912em; display: inline-block;"><span style="display: inline-block; position: relative; width: 0.766em; height: 0px; font-size: 116%;"><span style="position: absolute; clip: rect(1.062em, 1000.74em, 2.321em, -1000em); top: -2.155em; left: 0em;"><span class="mrow" id="MathJax-Span-537"><span class="mover" id="MathJax-Span-538"><span style="display: inline-block; position: relative; width: 0.786em; height: 0px;"><span style="position: absolute; clip: rect(3.174em, 1000.76em, 4.189em, -1000em); top: -4.023em; left: 0em;"><span class="mi" id="MathJax-Span-539" style="font-family: MathJax_Math; font-style: italic;">G</span><span style="display: inline-block; width: 0px; height: 4.023em;"></span></span><span style="position: absolute; clip: rect(3.157em, 1000.55em, 3.57em, -1000em); top: -4.251em; left: 0.198em;"><span class="mo" id="MathJax-Span-540" style=""><span style="font-family: MathJax_Size1;">˜</span></span><span style="display: inline-block; width: 0px; height: 4.023em;"></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 2.155em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.081em; border-left: 0px solid; width: 0px; height: 1.238em;"></span></span></nobr><span class="MJX_Assistive_MathML" role="presentation"><math xmlns="http://www.w3.org/1998/Math/MathML" id="M32"><mover accent="true"><mi>G</mi><mo>˜</mo></mover></math></span></span><script type="math/mml" id="MathJax-Element-32"><math id="M32"><mover accent="true"><mi>G</mi><mo>˜</mo></mover></math></script> (<a href="#B21">Patterson et al., 2006</a>). Gradually increasing the value of <i>θ</i>
 enabled a fine grain analysis of genetic discontinuities in human data,
 and allowed us to study IBD patterns within genetic clusters. The 
computational complexity of spFA increases linearly as a function of the
 number of markers. Since it is equivalent to the computation of a low 
rank approximation of the genotypic matrix (lower than a standard PCA, a
 few seconds on standard computer systems), applying spFA at multiple 
scales was not overly time-consuming.</p>
   <a id="h6" class="reset-hash-position"></a><h2>Conclusion</h2>
   <p class="mb0">This study provided a comparison of existing methods 
that attempt to correct for IBD effects in population genetic analyses, 
and showed that each of studied approaches provided different insights 
on the data. Under equilibrium IBD, PCA was confounded by continuous 
variation and the main genetic discontinuities may be missed or 
misinterpreted. For the same data, SFA over-estimated the number of 
clusters in the genetic data, creating spurious clusters from continuous
 patterns. In the presence of IBD patterns, spatial factor analysis 
provided clearer interpretations of the data than PCA and SFA. In a 
spatially explicit framework, we found that spFA identified genetic 
discontinuities more efficiently than did PCA or SFA when these 
discontinuities are blurred by noise from IBD patterns in the genetic 
data.</p>
   <a id="h7" class="reset-hash-position"></a><h2>Conflict of Interest Statement</h2>
   <p class="mb0">The authors declare that the research was conducted in
 the absence of any commercial or financial relationships that could be 
construed as a potential conflict of interest.</p>
   <a id="h8" class="reset-hash-position"></a><h2>Acknowledgments</h2>
   <p class="mb0">We thank Nicolas Duforet-Frebourg for his help with the software <code>ms</code>.
 This work was supported by a grant from la Région Rhône-Alpes to Eric 
Frichot and Olivier François, and by an NSF grant to Sean Schoville 
(OISE-0965038). Olivier François acknowledges support from Grenoble INP.</p>
   <a id="h9" class="reset-hash-position"></a><h2>Footnotes</h2>
   <div id="footnotetext" class="fulltextdescription">
      <ol>
         <li id="note1"><strong><a class="footnotetextanchor" href="#note1a" style="color:#707173; text-decoration:none;" title="">^</a></strong><a href="http://membres-timc.imag.fr/Olivier.Francois/spfa.R" style="color:#707173; text-decoration:none;">http://membres-timc.imag.fr/Olivier.Francois/spfa.R</a></li>
         <li id="note2"><strong><a class="footnotetextanchor" href="#note2a" title="" style="color:#707173; text-decoration:none;">^</a></strong><a href="ftp://ftp.cephb.fr/hgdp_v3/" style="color:#707173; text-decoration:none;">ftp://ftp.cephb.fr/hgdp_v3/</a></li>
         <li id="note3"><strong><a class="footnotetextanchor" href="#note3a" title="" style="color:#707173; text-decoration:none;">^</a></strong><a href="ftp://ftp.cephb.fr/hgdp_supp10/" style="color:#707173; text-decoration:none;">ftp://ftp.cephb.fr/hgdp_supp10/</a></li>
      </ol>
   </div>
   <a id="h10" class="reset-hash-position"></a><h2>References</h2>
   <div class="References">
      <p class="ReferencesCopy1"><a id="B1"></a>Ahmed, N., Natarajan, T., and Rao, K. R. (1974). Discrete cosine transform. <i>IEEE Trans. Comput</i>. C–23, 90–93.</p>
      <p class="ReferencesCopy2"><a href="http://dx.doi.org/10.1109/T-C.1974.223784" target="_blank">CrossRef Full Text</a></p></div>
   <div class="References">
      <p class="ReferencesCopy1"><a id="B2"></a>Borcard, D., and 
Legendre, P. (2002). All-scale spatial analysis of ecological data by 
means of principal coordinates of neighbour matrices. <i>Ecol. Modell.</i> 153, 51–68.</p>
      <p class="ReferencesCopy2"><a href="http://dx.doi.org/10.1016/S0304-3800(01)00501-4" target="_blank">CrossRef Full Text</a></p></div>
   <div class="References">
      <p class="ReferencesCopy1"><a id="B3"></a>Borcard, D., Legendre, 
P., Avois-Jacquet, C., and Tuomisto, H. (2004). Dissecting the spatial 
structure of ecological data at multiple scales. <i>Ecology</i> 85, 1826–1832.</p>
      <p class="ReferencesCopy2"><a href="http://dx.doi.org/10.1890/03-3111" target="_blank">CrossRef Full Text</a></p></div>
   <div class="References">
      <p class="ReferencesCopy1"><a id="B4"></a>Cressie, N. A. C. (1993). <i>Statistics for Spatial Data, Revised Edition</i>. New York: Wiley.</p>
   </div>
   <div class="References">
      <p class="ReferencesCopy1"><a id="B5"></a>Diaconis, P., Goel, S., and Holmes, S. (2008). Horseshoes in multidimensional scaling and local kernel methods. <i>Ann. Appl. Stat.</i> 2, 777–807.</p>
      <p class="ReferencesCopy2"><a href="http://dx.doi.org/10.1214/08-AOAS165" target="_blank">CrossRef Full Text</a></p></div>
   <div class="References">
      <p class="ReferencesCopy1"><a id="B6"></a>Dray, S., Legendre, P., 
and Peresneto, P. (2006). Spatial modelling: a comprehensive framework 
for principal coordinate analysis of neighbour matrices (PCNM). <i>Ecol. Modell.</i> 196, 483–493.</p>
      <p class="ReferencesCopy2"><a href="http://dx.doi.org/10.1016/j.ecolmodel.2006.02.015" target="_blank">CrossRef Full Text</a></p></div>
   <div class="References">
      <p class="ReferencesCopy1"><a id="B7"></a>Durand, E., Jay, F., 
Gaggiotti, O. E., and François, O. (2009). Spatial inference of 
admixture proportions and secondary contact zones. <i>Mol. Biol. Evol.</i> 26, 1963–1973.</p>
      <p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&amp;Cmd=ShowDetailView&amp;TermToSearch=19461114" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=19461114" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1093/molbev/msp106" target="_blank">CrossRef Full Text</a></p></div>
   <div class="References">
      <p class="ReferencesCopy1"><a id="B8"></a>Engelhardt, B. E., and 
Stephens, M. (2010). Analysis of population structure: a unifying 
framework and novel methods based on sparse factor analysis. <i>PLoS Genet.</i> 6, e1001117. doi:10.1371/journal.pgen.1001117</p>
      <p class="ReferencesCopy2"><a href="http://dx.doi.org/10.1371/journal.pgen.1001117" target="_blank">CrossRef Full Text</a></p></div>
   <div class="References">
      <p class="ReferencesCopy1"><a id="B9"></a>François, O., Currat, 
M., Ray, N., Han, E., Excoffier, L., and Novembre, J. (2010). Principal 
component analysis under population genetic models of range expansion 
and admixture. <i>Mol. Biol. Evol.</i> 27, 1257–1268.</p>
      <p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&amp;Cmd=ShowDetailView&amp;TermToSearch=20097660" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=20097660" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1093/molbev/msq010" target="_blank">CrossRef Full Text</a></p></div>
   <div class="References">
      <p class="ReferencesCopy1"><a id="B10"></a>François, O., and Durand, E. (2010). Spatially explicit bayesian clustering models in population genetics. <i>Mol. Ecol. Resour.</i> 10, 773–784.</p>
      <p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&amp;Cmd=ShowDetailView&amp;TermToSearch=21565089" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=21565089" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1111/j.1755-0998.2010.02868.x" target="_blank">CrossRef Full Text</a></p></div>
   <div class="References">
      <p class="ReferencesCopy1"><a id="B11"></a>Hudson, R. R. (2002). Generating samples under a Wright-Fisher neutral model of genetic variation. <i>Bioinformatics</i> 18, 337–338.</p>
      <p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&amp;Cmd=ShowDetailView&amp;TermToSearch=11847089" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=11847089" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1093/bioinformatics/18.suppl_1.S337" target="_blank">CrossRef Full Text</a></p></div>
   <div class="References">
      <p class="ReferencesCopy1"><a id="B12"></a>Jolliffe, I. T. (1986). <i>Principal Component Analysis</i>. New York: Springer Verlag.</p>
   </div>
   <div class="References">
      <p class="ReferencesCopy1"><a id="B13"></a>Jombart, T., Devillard,
 S., Dufour, A.-B., and Pontier, D. (2008). Revealing cryptic spatial 
patterns in genetic variability by a new multivariate method. <i>Heredity</i> 101, 92–103.</p>
      <p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&amp;Cmd=ShowDetailView&amp;TermToSearch=18446182" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=18446182" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1038/hdy.2008.34" target="_blank">CrossRef Full Text</a></p></div>
   <div class="References">
      <p class="ReferencesCopy1"><a id="B14"></a>Kimura, M., and Weiss, 
G. H. (1964). The stepping stone model of population structure and the 
decrease of genetic correlation with distance. <i>Genetics</i> 49, 561–576.</p>
      <p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&amp;Cmd=ShowDetailView&amp;TermToSearch=17248204" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=17248204" target="_blank">Pubmed Full Text</a></p></div>
   <div class="References">
      <p class="ReferencesCopy1"><a id="B15"></a>Legendre, P., and Gallagher, E. (2001). Ecologically meaningful transformations for ordination of species data. <i>Oecologia</i> 129, 271–280.</p>
      <p class="ReferencesCopy2"><a href="http://dx.doi.org/10.1007/s004420100716" target="_blank">CrossRef Full Text</a></p></div>
   <div class="References">
      <p class="ReferencesCopy1"><a id="B16"></a>Malécot, G. (1948). <i>Les Mathématiques de l’Hérédité</i>. Paris: Masson.</p>
   </div>
   <div class="References">
      <p class="ReferencesCopy1"><a id="B17"></a>Mardia, K. V., Kent, J. T., and Bibby, J. M. (1979). <i>Multivariate Analysis</i>. London: Academic Press.</p>
   </div>
   <div class="References">
      <p class="ReferencesCopy1"><a id="B18"></a>McVean, G. (2009). A genealogical interpretation of principal components analysis. <i>PLoS Genet.</i> 5, e1000686. doi:10.1371/journal.pgen.1000686</p>
      <p class="ReferencesCopy2"><a href="http://dx.doi.org/10.1371/journal.pgen.1000686" target="_blank">CrossRef Full Text</a></p></div>
   <div class="References">
      <p class="ReferencesCopy1"><a id="B19"></a>Meirmans, P. G. (2012). The trouble with isolation by distance. <i>Mol. Ecol.</i> 21, 2839–2846.</p>
      <p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&amp;Cmd=ShowDetailView&amp;TermToSearch=22574758" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=22574758" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1111/j.1365-294X.2012.05578.x" target="_blank">CrossRef Full Text</a></p></div>
   <div class="References">
      <p class="ReferencesCopy1"><a id="B20"></a>Novembre, J., and Stephens, M. (2008). Interpreting principal component analyses of spatial population genetic variation. <i>Nat. Genet.</i> 40, 646–649.</p>
      <p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&amp;Cmd=ShowDetailView&amp;TermToSearch=18425127" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=18425127" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1038/ng.139" target="_blank">CrossRef Full Text</a></p></div>
   <div class="References">
      <p class="ReferencesCopy1"><a id="B21"></a>Patterson, N., Price, A. L., and Reich, D. (2006). Population structure and eigenanalysis. <i>PLoS Genet.</i> 2, e190. doi:10.1371/journal.pgen.0020190</p>
      <p class="ReferencesCopy2"><a href="http://dx.doi.org/10.1371/journal.pgen.0020190" target="_blank">CrossRef Full Text</a></p></div>
   <div class="References">
      <p class="ReferencesCopy1"><a id="B22"></a>Patterson, N. J., Moorjani, P., Luo, Y., Mallick, S., Rohland, N., Zhan, Y., et al. (2012). Ancient admixture in human history. <i>Genetics</i> 192, 1065–1093.</p>
      <p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&amp;Cmd=ShowDetailView&amp;TermToSearch=22960212" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=22960212" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1534/genetics.112.145037" target="_blank">CrossRef Full Text</a></p></div>
   <div class="References">
      <p class="ReferencesCopy1"><a id="B23"></a>Powell, G. T., Yang, 
H., Tyler-Smith, C., and Xue, Y. (2007). The population history of the 
Xibe in northern China: a comparison of autosomal, mtDNA and 
Y-chromosomal analyses of migration and gene flow. <i>Forensic Sci. Int. Genet.</i> 1, 115–119.</p>
      <p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&amp;Cmd=ShowDetailView&amp;TermToSearch=19083740" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=19083740" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1016/j.fsigen.2007.01.015" target="_blank">CrossRef Full Text</a></p></div>
   <div class="References">
      <p class="ReferencesCopy1"><a id="B24"></a>Pritchard, J. K., Stephens, M., and Donnelly, P. (2000). Inference of population structure using multilocus genotype data. <i>Genetics</i> 155, 945–959.</p>
      <p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&amp;Cmd=ShowDetailView&amp;TermToSearch=10835412" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=10835412" target="_blank">Pubmed Full Text</a></p></div>
   <div class="References">
      <p class="ReferencesCopy1"><a id="B25"></a>Rosenberg, N., 
Pritchard, J., Weber, J., Cann, H., Kidd, K., Zhivotovsky, L., et al. 
(2002). Genetic structure of human populations. <i>Science</i> 298, 2381–2385.</p>
      <p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&amp;Cmd=ShowDetailView&amp;TermToSearch=12493913" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=12493913" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1126/science.1078311" target="_blank">CrossRef Full Text</a></p></div>
   <div class="References">
      <p class="ReferencesCopy1"><a id="B26"></a>Rousset, F. (1997). Genetic differentiation and estimation of gene flow from F-statistics under isolation by distance. <i>Genetics</i> 145, 1219–1228.</p>
      <p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&amp;Cmd=ShowDetailView&amp;TermToSearch=9093870" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=9093870" target="_blank">Pubmed Full Text</a></p></div>
   <div class="References">
      <p class="ReferencesCopy1"><a id="B27"></a>Tipping, M. E., and Bishop, C. M. (1999). Probabilistic principal component analysis. <i>J. R. Stat. Soc. B</i> 61, 611–622. doi:10.1111/1467-9868.00196</p>
      <p class="ReferencesCopy2"><a href="http://dx.doi.org/10.1111/1467-9868.00196" target="_blank">CrossRef Full Text</a></p></div>
   <div class="References">
      <p class="ReferencesCopy1"><a id="B28"></a>Wright, S. (1943). Isolation by distance. <i>Genetics</i> 28, 114–138.</p>
      <p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&amp;Cmd=ShowDetailView&amp;TermToSearch=17247074" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=17247074" target="_blank">Pubmed Full Text</a></p></div>
   <div class="References">
      <p class="ReferencesCopy1"><a id="B29"></a>Zhang, F., Su, B., Zhang, Y., and Jin, L. (2007). Genetic studies of human diversity in East Asia. <i>Philos. Trans. R. Soc. Lond. B Biol. Sci.</i> 362, 987–996.</p>
      <p class="ReferencesCopy2"><a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&amp;Cmd=ShowDetailView&amp;TermToSearch=17317646" target="_blank">Pubmed Abstract</a> | <a href="http://eutils.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?db=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=17317646" target="_blank">Pubmed Full Text</a> | <a href="http://dx.doi.org/10.1098/rstb.2007.2028" target="_blank">CrossRef Full Text</a></p></div>
</div>
<div class="thinLineM20"></div>
<div class="AbstractSummary">
   <p>
      <span>Keywords:</span> principal component analysis, isolation-by-distance, spatial autocorrelation, spatial factor analysis</p>
   <p>
      <span>Citation:</span> Frichot E, Schoville S, Bouchard G and 
François O (2012) Correcting principal component maps for effects of 
spatial autocorrelation in population genetic data. <i>Front. Gene.</i> <b>3</b>:254. doi: 10.3389/fgene.2012.00254</p>
   <p id="timestamps">
      <span>Received:</span> 21 August 2012; <span>Accepted:</span> 29  October  2012;<br> <span>Published online:</span> 20 November  2012.</p>
   <div>
      <p>Edited by:</p>
 <a href="http://www.frontiersin.org/Community/WhosWhoActivity.aspx?sname=ChrisAmos&amp;UID=50708">Chris Amos</a>, The University of Texas MD Anderson Cancer Center, USA</div>
   <div>
      <p>Reviewed by:</p>
 <a href="http://www.frontiersin.org/Community/WhosWhoActivity.aspx?sname=YiranGuo&amp;UID=27284">Yiran Guo</a>, Children’s Hospital of Philadelphia, USA<br> <a href="http://www.frontiersin.org/Community/WhosWhoActivity.aspx?sname=BamideleTayo&amp;UID=29364">Bamidele Tayo</a>, Loyola University Chicago, USA</div>
   <p>
      <span>Copyright:</span>  © 2012 Frichot, Schoville, Bouchard and François. This is an open-access article distributed under the terms of the <a href="http://creativecommons.org/licenses/by/3.0/">Creative Commons Attribution License</a>,
 which permits use, distribution and reproduction in other forums, 
provided the original authors and source are credited and subject to any
 copyright notices concerning any third-party graphics etc.</p>
   <p>
      <span>*Correspondence:</span> Olivier François, Laboratoire 
TIMC-IMAG, Faculty of Medicine, Grenoble INP, La Tronche, Grenoble 
F38706, France. e-mail: olivier.francois@imag.fr</p>
   <div class="clear"></div>
</div>
                
                <div class="thin-line-dark"></div>
                    <div class="social-feed"></div>
                    <div class="container-bordered comment-list social-feed">
                        <div class="container-comments">

    <div class="w100pc float_left CommentsHolder" id="div_article_comment_holder">
        

        
    </div>

    <div id="div_add_comment" class="commentBox">
        <div class="imgTn32" style="display: none;">
            
            <img src="Frontiers%20|%20Correcting%20Principal%20Component%20Maps%20for%20Effects%20of%20Spatial%20Autocorrelation%20in%20Population%20Genetic%20Data%20|%20Genetics_fichiers/default_profile_32.jpg" alt="" onerror="this.src='/Images/Frontiers/Common/Profile/default_profile_32.jpg';">
            

        </div>
        <div class="wrapper pt4 pb4 wAuto">
            <textarea onfocus="showUserImageOnAdd(this)" onblur="hideUserImage(this)" id="txt_comment" cols="" rows="1" style="overflow: hidden; word-wrap: break-word; resize: horizontal;" data-test-id="article_comment_add_textarea" class="write setfocus autoext w535 frUIResizeTextArea comment-textarea is-ui animated-resize grey_90">Write a comment...</textarea>
        </div>
        <div class="commentAddWrap">
            <div>
                <a class="addBtnComment" data-test-id="article_comment_add_button" onclick="javascript:AddComments(this);" href="javascript:void(0);" id="btnAddcom"><span>Add</span></a>
            </div>
        </div>
    </div>

</div>

                                               
                    </div>

            </div>
        </div>
    </div>
</main>

            <div class="clearfix visible-sm"></div>

            
    <div class="c col-xs-12 col-sm-12 col-lg-2 col-md-3 pull-right side-article related right-container-articles" style="clear: right !important">
        
        <article class="widget-listing commentary-article hidden">
            <h5 class="like-h4">COMMENTARY</h5>
        </article>

        <article class="widget-listing original-article hidden">
            <h5 class="like-h4">ORIGINAL ARTICLE</h5>
        </article>

        <article class="widget-listing people-also-looked-at side-article-related">
            <h5 class="like-h4">People also looked at</h5>
        <div class="teaser"><div class="heading-container"><h2 class="teaser-heading"><a href="https://www.frontiersin.org/articles/10.3389/fgene.2018.00097/full">A Meta-Analysis of Genome-Wide Association Studies of Growth Differentiation Factor-15 Concentration in Blood</a></h2><p class="teaser-authors"><a href="https://loop.frontiersin.org/people/478241/overview">Jiyang Jiang</a>, Anbupalam Thalamuthu, Jennifer E. Ho, Anubha Mahajan, Weronica E. Ek, <a href="https://loop.frontiersin.org/people/258204/overview">David A. Brown</a>, Samuel N. Breit, Thomas J. Wang, Ulf Gyllensten, Ming-Huei Chen, <a href="https://loop.frontiersin.org/people/237467/overview">Stefan Enroth</a>, James L. Januzzi, Lars Lind, <a href="https://loop.frontiersin.org/people/334133/overview">Nicola J. Armstrong</a>, John B. Kwok, <a href="https://loop.frontiersin.org/people/345918/overview">Peter R. Schofield</a>, <a href="https://loop.frontiersin.org/people/481719/overview">Wei Wen</a>, <a href="https://loop.frontiersin.org/people/92787/overview">Julian N. Trollor</a>, Åsa Johansson, Andrew P. Morris, Ramachandran S. Vasan, <a href="https://loop.frontiersin.org/people/7050/overview">Perminder S. Sachdev</a> and <a href="https://loop.frontiersin.org/people/138659/overview">Karen A. Mather</a></p></div></div><div class="teaser"><div class="heading-container"><h2 class="teaser-heading"><a href="https://www.frontiersin.org/articles/10.3389/fnsys.2017.00016/full">Accelerated
 Recovery of Consciousness after General Anesthesia Is Associated with 
Increased Functional Brain Connectivity in the High-Gamma Bandwidth</a></h2><p class="teaser-authors"><a href="https://loop.frontiersin.org/people/403633/overview">Duan Li</a>, Viviane S. Hambrecht-Wiedbusch and <a href="https://loop.frontiersin.org/people/9706/overview">George A. Mashour</a></p></div></div><div class="teaser"><div class="heading-container"><h2 class="teaser-heading"><a href="https://www.frontiersin.org/articles/10.3389/fneur.2015.00010/full">Multimodal Imaging of Dynamic Functional Connectivity</a></h2><p class="teaser-authors"><a href="https://loop.frontiersin.org/people/45003/overview">Enzo Tagliazucchi</a> and <a href="https://loop.frontiersin.org/people/61439/overview">Helmut Laufs</a></p></div></div><div class="teaser"><div class="heading-container"><h2 class="teaser-heading"><a href="https://www.frontiersin.org/articles/10.3389/fgene.2012.00076/full">Transgenerational Epigenetic Inheritance in Animals</a></h2><p class="teaser-authors"><a href="https://loop.frontiersin.org/people/27066/overview">Igor Kovalchuk</a></p></div></div><div class="teaser"><div class="heading-container"><h2 class="teaser-heading"><a href="https://www.frontiersin.org/articles/10.3389/fgene.2012.00300/full">Double
 genomic control is not effective to correct for population 
stratification in meta-analysis for genome-wide association studies</a></h2><p class="teaser-authors"><a href="https://loop.frontiersin.org/people/35865/overview">Shudong Wang</a>, Wenan Chen, <a href="https://loop.frontiersin.org/people/14298/overview">Xiangning Chen</a>, Fengjiao Hu, <a href="https://loop.frontiersin.org/people/321324/overview">Kellie J. Archer</a>, <a href="https://loop.frontiersin.org/people/23370/overview">Nianjun Liu</a>, Shumei Sun and <a href="https://loop.frontiersin.org/people/24583/overview">Guimin Gao</a></p></div></div></article>
            <div class="side-article-mrk clearfix hidden-md hidden-lg">
                <style type="text/css"><!--
.spotlight-title:hover,
.spotlight-title:focus {
 color: #555 !important; 
}
.spotlight-link {
 color: #f2b12f !important;
}
.spotlight-link:hover,
.spotlight-link:focus {
 color: #e5a812 !important;
}
--></style>
<div class="spotlight"><a href="https://spotlight.frontiersin.org/how-to-enter/?utm_source=FWEB&amp;utm_medium=FMAIN&amp;utm_campaign=SPOTLIGHT&amp;utm_term=BANNER&amp;utm_content=WEB"><img src="Frontiers%20|%20Correcting%20Principal%20Component%20Maps%20for%20Effects%20of%20Spatial%20Autocorrelation%20in%20Population%20Genetic%20Data%20|%20Genetics_fichiers/article-marketing-message-1f3be380-af93-4b04-a218-32e22eb3bd.png" alt=""></a><br>
<h3 style="font-weight: 500;"><a class="spotlight-title" tabindex="0" href="https://spotlight.frontiersin.org/how-to-enter/?utm_source=FWEB&amp;utm_medium=FMAIN&amp;utm_campaign=SPOTLIGHT&amp;utm_term=BANNER&amp;utm_content=WEB" target="_blank">Want to win $100,000 to host your own conference?</a></h3>
<a class="spotlight-link" href="https://spotlight.frontiersin.org/how-to-enter/?utm_source=FWEB&amp;utm_medium=FMAIN&amp;utm_campaign=SPOTLIGHT&amp;utm_term=BANNER&amp;utm_content=WEB" target="_blank">Suggest a Research Topic</a></div>
            </div>


    </div>


        </div>
    </div>
</div>
<div id="divTemplates">
    <div class="modal fade modal-container impact-modal-container" data-backdrop="static" id="impactModal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true">
</div>






<script type="text/template" id="template-impact-modal">

    <div class="container">
        <div class="row">
            <div style="padding-bottom: 100px">
                <div class="modal-dialog">
                    <div class="modal-content">
                        <div class="modal-header clearfix">
                            <button type="button" class="close btn-close" data-dismiss="modal" aria-hidden="true">&times;</button>
                            <h3 class="modal-title" id="myModalLabel">Impact</h3>
                            <h6>
                                <a href="javascript:void(0)" class="article-title" data-dismiss="modal" aria-hidden="true">
                                    Correcting Principal Component Maps for Effects of Spatial Autocorrelation in Population Genetic Data
                                </a>
                            </h6>
                        </div>
                        <div class="modal-body">
                            <img class="loader" src="/areas/journals/images/icon/loading.gif" alt="Loading.." />

                            <div class="row article-impact-tabs hidden">
                                <!-- Nav tabs -->
                                <ul class="nav nav-tabs impact-tabs clearfix">
                                    <li class="active nav-tab-views"><a href="#views-tab" data-toggle="tab" data-tab-type="views"><span class="total-views" data-tab-type="views"></span>Views</a></li>
                                    <li class="nav-tab-citations"><a href="#citations-tab" data-toggle="tab" data-tab-type="citations"><span class="highest-Citation" data-tab-type="citations"></span>Citations</a></li>
                                    <li class="nav-tab-demographics"><a href="#demographics-tab" data-toggle="tab" data-tab-type="demographics">Demographics</a></li>
                                        <li class="nav-tab-social-buzz"><a href="#social-buzz-tab" data-toggle="tab" data-tab-type="socialbuzz">Social Buzz</a></li>
                                </ul>
                                <div class="tab-content impact-tabs">
                                    <!-- VIEWS TAB -->
                                    <div class="tab-pane active" id="views-tab">
                                        <h5 style="padding-left: 68px; display: none;" class="sincebeginning">Since beginning</h5>
                                        <h5 style="padding-left: 68px; display: none;" class="thisyear">Last 12 months</h5>
                                        <h5 style="padding-left: 68px; display: none;" class="thismonth">Last 30 days</h5>


                                        <p style="display: none;height:60px" class="views-empty-message lead">No records found</p>
                                        <div class="chart-container">
                                            <p class="info sincebeginning" style="display: none;"><strong class="begin-views-downloads download-font-styles"> </strong> views and downloads <strong class="begin-views year-font-styles"> </strong> views <strong class="begin-downloads year-font-styles"> </strong> downloads</p>
                                            <p class="info thisyear" style="display: none;"><strong class="year-views-downloads download-font-styles"> </strong> views and downloads <strong class="year-views year-font-styles"> </strong> views <strong class="year-downloads year-font-styles"> </strong> downloads</p>
                                            <p class="info thismonth" style="display: none;"><strong class="month-views-downloads download-font-styles"> </strong> views and downloads <strong class="month-views year-font-styles"> </strong> views <strong class="month-downloads year-font-styles"> </strong> downloads</p>
                                            <div id="views-timeline-chart" class="views-bar-charts"></div>
                                            <div id="views-bar-chart" class="views-timeline-charts"></div>
                                        </div>
                                        <div id="views-chart-controls" class="input-block-level" style="display: none;">
                                            <!-- Split button -->
                                            <div class="btn-group pull-right">
                                                <button type="button" class="btn btn-flat dropdown-toggle" data-toggle="dropdown" >
                                                    <span id="Period" data-bind="label">Select a time period</span>&nbsp;<span class="caret"></span>
                                                </button>
                                                <ul id="PeriodDrop" class="dropdown-menu" role="menu">
                                                    <li class="beginning" ><a>Since beginning</a></li>
                                                    <li class="year"><a>Last 12 months</a></li>
                                                    <li class="month"><a>Last 30 days</a></li>
                                                </ul>
                                            </div>
                                            <div style="padding-left: 72px;" class="btn-group pull-left" data-toggle="tooltip" title="Toggle between timeline and bar chart views">
                                                <button type="button" class="btn btn-flat btn-line active" id="views-timeline-button">
                                                    <span style="padding: 0 4px">&nbsp;&nbsp;</span>
                                                </button>
                                                <button type="button" class="btn btn-flat btn-line" id="views-bar-button">
                                                    <span style="padding: 0 4px">&nbsp;&nbsp;</span>
                                                </button>

                                            </div>
                                            <div class="clearfix"></div>
                                        </div><!-- /#views-chart-controls -->

                                    </div><!-- /#views-tab -->
                                    <!-- CITATIONS TAB -->
                                    <div class="tab-pane " id="citations-tab">
                                        <img class="citation-loader" src="/Images/Frontiers/Common/Icon/loading.gif" alt="Loading.." />
                                        <div style="width: 350px; margin: 0 auto;" class="citation-container hidden">
                                            <h5 style="text-align:center; margin-bottom: 0; padding-bottom: 0;">Article Citations</h5>
                                            <div class="scopus" style="width: 50%; float: left; text-align: center;">
                                                <a href= "http://www.scopus.com/inward/citedby.url?partnerID=HzOxMe3b&amp;doi=10.3389/fgene.2012.00254&amp;origin=inward"><h5><span class="repository">by Scopus</span></h5></a>
                                                <!-- Split button -->
                                                <div class="citation-count-container">
                                                    <div class="citation-panel">
                                                        <p class="citation-count"><span class="citaion-scopus"></span></p>
                                                    </div><!-- /.citation-panel -->
                                                </div><!-- /.well -->
                                            </div>

                                            <div class="crossref" style="width: 50%; float: right; text-align: center;">
                                                <a href="http://www.crossref.org/"><h5><span class="repository">by CrossRef</span></h5></a>
                                                <!-- Split button -->
                                                <div class="clearfix"></div>
                                                <div class="citation-count-container">
                                                    <div class="citation-panel">
                                                        <p class="citation-count"><span class="citation-crossref"></span></p>
                                                    </div><!-- /.citation-panel -->

                                                </div><!-- /.well -->
                                            </div>
                                            <div class="clearfix"></div>
                                        </div>
                                    </div><!-- /#citations-tab -->
                                    <!-- DEMOGRAPHICS TAB -->
                                    <div class="tab-pane " id="demographics-tab">

                                        <div class="btn-group pull-right" id="demographics-dropdown" style="margin-right: 10px;">
                                            &nbsp;
                                        </div>

                                        <div id="map-container">

                                            <div id="map">

                                            </div>
                                            <img class="map-loading" src="/Images/Frontiers/Common/Icon/loading.gif" alt="Loading.." />
                                        </div>
                                        <div class="clearfix"></div>

                                        <div class="row">
                                            <div class="col-xs-6">
                                                <h5>Top countries</h5>
                                                <div id="demographics-top-countries"></div>
                                            </div>
                                            <div class="col-xs-6">
                                                <h5>Top referring sites</h5>
                                                <div id="demographics-top-sites"></div>
                                            </div>
                                        </div>
                                        <hr>
                                        <div class="row">
                                            <div class="col-xs-4">
                                                <h5 style="text-align: center;">Domain</h5>
                                                <div id="demographics-domain"></div>
                                            </div>
                                            <div class="col-xs-4">
                                                <h5 style="text-align: center;">Field</h5>
                                                <div id="demographics-field"></div>
                                            </div>
                                            <div class="col-xs-4">
                                                <h5 style="text-align: center;">Specialty</h5>
                                                <div id="demographics-specialty"></div>
                                            </div>
                                        </div>
                                        <hr>
                                        <div class="row">
                                            <div class="col-xs-4">
                                                <h5 style="text-align: center;">Industry</h5>
                                                <div id="demographics-industry"></div>
                                            </div>

                                            <div class="col-xs-4">
                                                <h5 style="text-align: center;">Education</h5>
                                                <div id="demographics-education"></div>
                                            </div>

                                            <div class="col-xs-4">
                                                <h5 style="text-align: center;">Position</h5>
                                                <div id="demographics-position"></div>
                                            </div>
                                          
                                        </div>
                                        <div class="clearfix"></div>

                                        <div class="demographics-age-gender">
                                            <hr>
                                            <h5 style="margin: 35px 0px;">Age and Gender</h5>
                                            <div id="demographics-age" style="min-width: 800px; height: 400px; margin: 0 auto"></div>
                                        </div>
                                    </div><!-- /#demographics-tab -->
                                        <!-- SOCIAL BUZZ TAB -->
                                        <div class="tab-pane " id="social-buzz-tab">
                                            <h5 class="altmetric-title">Altmetric</h5>
                                            <div id="altmetric-donut" data-badge-details="right" data-badge-type="medium-donut" data-doi="<%- doi %>" data-hide-no-mentions="false" class="altmetric-embed"></div>

                                            <div id='main' class="social-buzz-content-area hidden">
                                                <div id='content'>
                                                    <div id='details'>
                                                        <div class="row">
                                                            <div class="col-xs-6">
                                                                <h5>Sources</h5>
                                                            </div>
                                                            <div class="col-xs-6">
                                                                <ul class="nav nav-tabs pull-right">
                                                                    <li class='active'><a href="#altmetric-all-tab"  data-toggle="tab">All</a></li>
                                                                    <li id='tab_for_msm'><a href="#details_msm" data-toggle="tab">News</a></li>
                                                                    <li id='tab_for_blogs'><a href="#details_blogs" data-toggle="tab">Blogs</a></li>
                                                                </ul>
                                                            </div>
                                                        </div>

                                                        <div class="tabbable">
                                                            <div class="tab-content">

                                                                <div class='details tab-pane ' id='details_blogs'>

                                                                    <div class="blogs-posts"></div>
                                                                </div> <!--Blog-->

                                                                <div class='details tab-pane' id='details_msm'>

                                                                    <div class="news-posts"></div>
                                                                </div> <!--News-->

                                                                <div class='details tab-pane active' id='altmetric-all-tab'>
                                                                    <div class="all-posts"></div>
                                                                </div> <!--All-->

                                                            </div>
                                                        </div>

                                                    </div>

                                                    <div class='clearfix'></div>
                                                </div>
                                            </div>

                                        </div>
                                        <!-- /#social-buzz-tab -->
                                </div>
                            </div><!-- /.row -->

                        </div><!-- /.modal-body -->
                        <div class="modal-footer">
                            <span class="modal-footer-pmc">The displayed data aggregates results from <a href="https://www.frontiersin.org">Frontiers</a> and <a href="http://www.ncbi.nlm.nih.gov/pmc/">PubMed Central®</a>.</span>
                            <button type="button" class="btn btn-new-orange btn-flat btn-close" data-dismiss="modal">Close</button>
                        </div>
                    </div><!-- /.modal-content -->
                </div><!-- /.modal-dialog -->
            </div><!-- /.col-xs-9 -->
        </div><!-- /.row -->
    </div><!-- /.container -->

</script>



<script type="text/template" id="template-social-buzz-blogs-posts">
    <div class='posts_info'>So far Altmetric has seen <b><%- blogsCount %></b> posts.</div>
    <% _.each( socialBuzzPosts, function( socialBuzzPost ){ %>

    <div class='postbox'>
        <div class='title'><a target='_blank' href="<%- socialBuzzPost.Url%>" ><%- socialBuzzPost.Title %></a></div>
        <div class='source'><a target='_blank' href="<%- socialBuzzPost.Author.Url %>" ><%- socialBuzzPost.Author.Name %></a></div>
        <div class='snippet'>
            <p><%- socialBuzzPost.Summary %></p>
        </div>
        <div class="footer blog">
            <%-  FRArticleImpact.formattedPostedDate(socialBuzzPost.PostedOn) %>
        </div>
    </div>

    <% }); %>

</script>


<script type="text/template" id="template-social-buzz-news-posts">
    <div class='posts_info'>So far Altmetric has seen <b><%- newsCount %></b> stories.</div>
    <% _.each( socialBuzzPosts, function( socialBuzzPost ){ %>
    <div class='postbox'>
        <div class='author source' style='width: 80px;'>
            <div style='max-width: 80px; overflow: hidden;'>
                <img width='80' src="<%- socialBuzzPost.Author.Image %>" class="photo" />
            </div>
        </div>

        <div class="entry-content" style="margin-left: 0px;">
            <div class='title'>
                <a target='_blank' href="<%- socialBuzzPost.Url%>"><%- socialBuzzPost.Title %> </a>
            </div>
            <div class='source'><%- socialBuzzPost.Author.Name %></div>
            <div class='snippet'>
                <p><%- socialBuzzPost.Summary %></p>
            </div>
        </div>
        <div class="footer">
            <%-  FRArticleImpact.formattedPostedDate(socialBuzzPost.PostedOn) %>
        </div>
    </div>

    <% }); %>

</script>


<script type="text/template" id="template-social-buzz-all-posts">

    <div class="posts_info">So far Altmetric has seen <b><%- allCount %></b> stories.</div>

    <% _.each( socialBuzzPosts, function( socialBuzzPost ){ %>
    <% if (socialBuzzPost.Type == "Blogs") {%>

    <div class='postbox'>
        <div class='title'><a target='_blank' href="<%- socialBuzzPost.Url%>" ><%- socialBuzzPost.Title %></a></div>
        <div class='source'><a target='_blank' href="<%- socialBuzzPost.Author.Url %>" ><%- socialBuzzPost.Author.Name %></a></div>
        <div class='snippet'>
            <p><%- socialBuzzPost.Summary %></p>
        </div>
        <div class="footer blog">
            <%-  FRArticleImpact.formattedPostedDate(socialBuzzPost.PostedOn) %>
        </div>
    </div>

    <% } %>
    <%  if(socialBuzzPost.Type == "News"){ %>
    <div class='postbox'>
        <div class='author source' style='width: 80px;'>
            <div style='max-width: 80px; overflow: hidden;'>
                <img width='80' src="<%- socialBuzzPost.Author.Image %>" class="photo" />
            </div>
        </div>

        <div class="entry-content" style="margin-left: 0px;">
            <div class='title'>
                <a target='_blank' href="<%- socialBuzzPost.Url%>"><%- socialBuzzPost.Title %> </a>
            </div>
            <div class='source'><%- socialBuzzPost.Author.Name %></div>
            <div class='snippet'>
                <p><%- socialBuzzPost.Summary %></p>
            </div>
        </div>
        <div class="footer">
            <%-  FRArticleImpact.formattedPostedDate(socialBuzzPost.PostedOn)%>
        </div>
    </div>
    <% } %>
    <% }); %>

</script>






    <div class="modal fade modal-container supplementary-modal-container" data-backdrop="static" data-keyboard="false" id="supplementaryFilesModal" tabindex="-1" role="dialog" aria-labelledby="mySupplementaryModalLabel" aria-hidden="true">
</div>




<script type="text/template" id="template-supplementary-files-modal">
    <div class="supplementary-material-wrapper modal-dialog" role="document">
        <div class="modal-content">
            <div class="modal-body">
                <a class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">×</span></a>
                <h4>Supplementary Material</h4><br>
                <p class="supplementary-empty-message">There is no supplementary material currently available for this article</p>
                <div class="loading-wrapper"
                     id="loader"
                     style="display:none">
                    Loading supplemental data...
                    <img style="" src="/Areas/Articles/Images/Icon/loading.gif" alt="Loading.." />
                </div>
                <br />
                <div class="table-responsive" id="localFiles">
                </div>
                <div id="figshare-widget-container">

                </div>
            </div>
            <br />
            <div class="modal-footer bottom-links">
                <a class="btn btn-default" data-dismiss="modal">Close</a>
            </div>
        </div>

    </div>
</script>





<script type="text/template" id="template-local-files-modal">
    <table class="table table-striped supplementary-content" id="localFilesTable">
        <thead>
            <tr>
                <th>&nbsp;</th>
                <th>File Name</th>
                <th>&nbsp;</th>
            </tr>
        </thead>
        <tbody class="file-content">
            <% _.each(supplimentalFileDetails.FileDetails, function(fileViewModel){   %>
            <tr>
                <th scope="row"><a data-test-id="<%= fileViewModel.TestId %>" href="<%= fileViewModel.FileDownloadUrl %>"><img class="img-figure" src="<%= fileViewModel.ImageUrl %>"></a></th>
                <td>
                    <a  href="<%= fileViewModel.FileDownloadUrl %>">
                        <%= fileViewModel.FileName %>
                    </a>
                </td>
                <td><a  href="<%= fileViewModel.FileDownloadUrl %>" class="btn-link"><i class="fa fa-download" aria-hidden="true"></i></a></td>
            </tr>
            <% }); %>
        </tbody>
    </table>
</script>







    <div class="modal fade modal-container notifyme-modal-container" data-backdrop="static" id="notifyModal" tabindex="-1" role="dialog" aria-labelledby="Notify on publication" aria-hidden="true">
</div>



<script type="text/template" id="template-notify-me-modal">
    <div class="modal-f page-container simple-modal" role="dialog" aria-labelledby="NotfifyMeModalLabel">
            <div class="modal-dialog" role="document">
                <div class="modal-content">
                    <form onsubmit="return false" id="modalnotifyme">
                        <div class="modal-header">
                            <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">×</span></button>
                            <h3>Notify me on publication</h3>
                        </div>
                        <div class="modal-body">

                            <div class="form-group">
                                <label>Please enter your email address:</label>
                            <input type="email" required class="form-control" id="txt_notification_email_id" placeholder="Email">
                            </div>

                        <div class="g-recaptcha" id="recaptcha"></div><br />

                        <p class="small" style="color: #555;">If you already have an account, please <a href="<%=login.FrontiersLoginUrl%>?returnUrl=<%=currentPage%>" class="text-blue">login</a>.</p>
                        <p class="small" style="color: #555;">You don't have a Frontiers account ? You can <a href="<%=login.FrontiersRegistrationUrl%>" class="text-blue">register here</a>.</p>
                        </div>
                        <div class="modal-footer">
                        <button type="button" id="article_notify_non_registered_user" class="btn btn-default btn-progress" style="width: 118px; visibility:hidden;">Notify me</button>
                        </div>
                    </form>
                </div>
            </div>
        </div>
</script>

<script type="text/template" id="template-notifyme-mailselection-modal">
    <div class="modal-f page-container simple-modal" tabindex="-1" role="dialog" aria-labelledby="NotfifyMeModalLabel">
        <div class="modal-dialog" role="document">
            <div class="modal-content">
                <div class="modal-header">
                    <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">×</span></button>
                    <h3>Select one of your emails</h3>
                </div>
                <div class="modal-body">
                    <form>
                        <div class="form-group">
                            <label>You have multiple emails registered with Frontiers:</label>
                        </div>
                                           
                        <% _.each(userEmailList, function(user){   %>
                        <input type="radio" value="<%=user%>" name="notify_Emailcollection"> <%=user%><br>
                        <% }); %>

                    </form>
                </div>
                <div class="modal-footer">
                    <button type="button" id="article_notify_loggedin_user" class="btn btn-default btn-progress" style="width: 118px;">Notify me</button>
                </div>
            </div>

        </div>

    </div>
</script>

<script type="text/template" id="template_notifyme_error_modal">
    <div class="modal-f page-container simple-modal" tabindex="-1" role="dialog" aria-labelledby="NotfifyMeModalLabel">
        <div class="modal-dialog" role="document">
            <div class="modal-content">
                <div class="modal-header">
                    <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">×</span></button>
                    <h3>Notify me on publication</h3>
                </div>
                <div class="modal-body">
                    <form>
                        <div class="form-group">
                            <p class="text-red"><strong><%=message%></strong></p>
                        </div>
                    </form>
                </div>
                <div class="modal-footer">
                    <button type="button" data-dismiss="modal" class="btn btn-default btn-progress" style="width: 118px;">Cancel</button>
                </div>
            </div>
        </div>
    </div>
</script>







    

<script type="text/template" id="template-article-comments">

    <div class="w100pc float_left CommentsHolder" id="div_article_comment_holder">
        <% if(journalComments.Count !=0) { %>
        <div id="divActivityBox" class="commentBox bubbleInfo " style="z-index: 300;">
            <div class="float_left w100pc divActivityBox">
                <p id="pAllUsers" class="pb0 float_left w581 pAllUsers">
                    <a id="aAllUsers" href="javascript:void(0);" class="fontnormal trigger">
                        <%- journalComments.Count %> Comments
                    </a> <%= journalComments.CommentedUsers %>
                </p>
            </div>
        </div>
        <% } %>

        <% _.each( journalComments.Comments, function( comment ){ %>

        <div class="commentBox">

            <% if (comment.IsDeleted) {%>

            <div class="wrapper pt4 pb4">
                <div class="imgHolder mr10"> <!--<a  class="fontnormal">--><img width="32" src="/Design/Images/default_profile_32.jpg" onerror="this.src='/Design/Images/default_profile_32.jpg';" alt=""><!--</a>--></div>
                <div class="content w555">
                    <!-- <p class="mtm3" style='word-wrap:break-word'><a  class="fontnormal">User</a> </p>-->
                    <p class="mtm3">User</p><ul class="toolbar mt4 disblk w100pc">
                        <li class="pl0">
                            <p class="date">Comment deleted on <%- GetFormattedCommentDeletedDate(comment.CommentedDate) %></p>
                        </li>
                        <li class="float_right pr0"></li>
                    </ul>
                </div>
            </div>

            <% } else {%>

            <div class="wrapper pt4 pb4">
                <div class="imgHolder mr10"> <a class="fontnormal" data-test-id="article_commenteduser_profileimage_link" href="<%- comment.CommentedUser.ProfileUrl %>"><img src="<%- comment.CommentedUser.ThumbnailUrl %>" onerror="this.src='/Images/Frontiers/Common/Profile/default_profile_32.jpg';" alt=""></a></div>
                <div class="content w555 comment-body">
                    <p class="mtm3" style="word-wrap:break-word">
                        <a class="fontnormal" data-test-id="article_commenteduser_profile_link" href="<%- comment.CommentedUser.ProfileUrl %>">
                            <%- comment.CommentedUser.FullName %>
                        </a>
                        <span class="comment-text"><%= comment.Comment %></span>
                    </p>
                    <ul class="toolbar mt4 disblk w100pc">
                        <li class="pl0"><p class="date"><%- GetFormattedCommentDate(comment.CommentedDate) %></p></li>
                        <% if (comment.IsEditVisible)
                        { %>
                        <li>|</li>
                        <li><a href="javascript:void(0);" data-test-id="article_comment_edit_link" class="fontnormal btn-edit-comment" id="btnEditcom" data-comment-id="<%- comment.CommentId %>">Edit</a></li>
                        <% } %>
                        <% if (comment.IsDeleteVisible)
                        { %>
                        <li>|</li>
                        <li><a href="javascript:void(0);" data-test-id="article_comment_delete_link" class="fontnormal btn-delete-comment" id="btnDeletecom" data-comment-id="<%- comment.CommentId %>">Delete</a></li>
                        <% }%>
                        <li class="float_right pr0"></li>
                    </ul>
                </div>
            </div>
            <% }%>
        </div>

        <%  }); %>
    </div>

    <div id="div_add_comment" class="commentBox">
        <div class="imgTn32"
             style="display: none;">
            <% if(journalComments.LoginUserId !=0) { %>
            <img src="<%- journalComments.LoggedInUserInfo.ThumbnailUrl %>" alt=""
                 onerror="this.src='/Images/Frontiers/Common/Profile/default_profile_32.jpg';">
            <% } else {%>
            <img src="/Design/Images/default_profile_32.jpg" alt=""
                 onerror="this.src='/Images/Frontiers/Common/Profile/default_profile_32.jpg';">
            <% }%>

        </div>
        <div class="wrapper pt4 pb4 wAuto">
            <textarea onfocus="showUserImageOnAdd(this)"
                      onblur="hideUserImage(this)"
                      id="txt_comment"
                      cols=""
                      rows="1"
                      style="overflow: hidden; word-wrap: break-word; resize: horizontal;"
                      data-test-id="article_comment_add_textarea" 
                      class="write setfocus autoext w535 frUIResizeTextArea comment-textarea is-ui animated-resize grey_90">Write a comment...</textarea>
        </div>
        <div class="commentAddWrap">
            <div>
                <a class="addBtnComment" data-test-id="article_comment_add_button"  onclick="javascript:AddComments(this);" href="javascript:void(0);" id="btnAddcom"><span>Add</span></a>
            </div>
        </div>
    </div>

</script>

<script type="text/template" id="template-article-edit-comments">


    <div id="edit-comment-form" class="wrapper pt4 pb4 wAuto edit-comments">
        <textarea class="txt-comment"
                  onfocus="showUserImageOnEdit(this)"
                  onblur="hideUserImage(this)"
                    data-test-id="article_comment_edit_textarea" 
                  rows="3"><%= commentText %> </textarea>
        <div>
            <a class="btn btn-add-comment addBtnComment" data-test-id="article_comment_edit_button" data-comment-id="<%- commentId %>">Add</a>
        </div>
    </div>

</script>



</div>

    </div>

    
<footer class="footer-new">
    <div class="container-fluid main-container-xxl">
        <div class="col-lg-2 col-lg-offset-2 col-md-3 col-sm-3">
            <ul class="list-unstyled">
                <li><a href="https://www.frontiersin.org/Journal/Frontiers.aspx">Home</a></li>
                <li><a href="https://www.frontiersin.org/about">About Frontiers</a></li>
                <li><a href="https://www.frontiersin.org/about/journalseries">Journals A-Z</a></li>
                <li><a href="https://www.frontiersin.org/about/Institutional_Membership">Institutional Membership</a></li>

            </ul>
        </div>
        <div class="col-lg-2 col-md-3 col-sm-3">
            <ul class="list-unstyled">
                <li><a href="https://www.frontiersin.org/about/contact">Contact</a></li>
                
                <li><a href="https://blog.frontiersin.org/">News</a></li>
            </ul>
        </div>
        <div class="col-lg-2 col-md-3 col-sm-3">
            <ul class="list-unstyled">
                <li><a href="https://www.frontiersin.org/submissioninfo">Submit</a></li>
                <li><a target="_blank" href="https://frontiers.zendesk.com/hc/en-us">FAQs</a></li>
                <li><a href="https://www.frontiersin.org/TermsandConditions.aspx">Terms &amp; Conditions</a></li>
                <li><a href="https://www.frontiersin.org/legal/privacy-policy">Privacy Policy</a></li>
            </ul>
        </div>
        <div class="col-lg-2 col-md-3 col-sm-3">
            <ul class="list-unstyled">
                <li><a href="http://connect.frontiersin.org/subscriptions/subscribe">Newsletters</a></li>
                <li><a href="https://blog.frontiersin.org/2013/11/01/frontiers-social-media-and-rss/">RSS/Twitter</a></li>
                <li><a href="https://www.frontiersin.org/Team.aspx">Team</a></li>
                <li><a href="https://www.frontiersin.org/Careers">Careers</a></li>
            </ul>
        </div>
    </div>
</footer>


<div class="footer-copyright-new">
    <div class="container-fluid main-container-xxl">
        <div class="col-md-12 text-center"> © 2007 - 2019 Frontiers Media S.A. All Rights Reserved </div>
    </div>
</div>


    <script src="Frontiers%20|%20Correcting%20Principal%20Component%20Maps%20for%20Effects%20of%20Spatial%20Autocorrelation%20in%20Population%20Genetic%20Data%20|%20Genetics_fichiers/jquery-2.js"></script>

    <script src="Frontiers%20|%20Correcting%20Principal%20Component%20Maps%20for%20Effects%20of%20Spatial%20Autocorrelation%20in%20Population%20Genetic%20Data%20|%20Genetics_fichiers/frontiers_003.js"></script>



    <script src="Frontiers%20|%20Correcting%20Principal%20Component%20Maps%20for%20Effects%20of%20Spatial%20Autocorrelation%20in%20Population%20Genetic%20Data%20|%20Genetics_fichiers/bootstrap.js"></script>

    <script src="Frontiers%20|%20Correcting%20Principal%20Component%20Maps%20for%20Effects%20of%20Spatial%20Autocorrelation%20in%20Population%20Genetic%20Data%20|%20Genetics_fichiers/vendors"></script>

    <script src="Frontiers%20|%20Correcting%20Principal%20Component%20Maps%20for%20Effects%20of%20Spatial%20Autocorrelation%20in%20Population%20Genetic%20Data%20|%20Genetics_fichiers/MathJax.js"></script>
 

    <script src="Frontiers%20|%20Correcting%20Principal%20Component%20Maps%20for%20Effects%20of%20Spatial%20Autocorrelation%20in%20Population%20Genetic%20Data%20|%20Genetics_fichiers/embed.js"></script>
 

    <script src="Frontiers%20|%20Correcting%20Principal%20Component%20Maps%20for%20Effects%20of%20Spatial%20Autocorrelation%20in%20Population%20Genetic%20Data%20|%20Genetics_fichiers/frontiers"></script>


    <script type="text/javascript">

        var FRArticle = (function() {
            return {
                ArticleId: '34644',
                DOI:'10.3389/fgene.2012.00254',
                LoginUserId: '0',
                DomainId: '1',
                FieldId: '30',
                SpecialityId: '1265',
                IsPreview: FRSafe.boolean('False'),
                IsViewImpactFromLoop: FRSafe.boolean('False'),
                IsPublished: 'True',
                FigShareApiUrl: 'https://api.figshare.com/v2/collections/search',
                FigShareTimeOut: '3000'
            };
        })();

        var FRSocial = (function() {
            return {
                itemId: 14,
                itemTypeId: 1,
                entityId: '34644',
                ownerId: '65620',
                sanPath: 'http://www.frontiersin.org/files/',
                subItemId: '8',
                loginUserId: '0',
                ownerNWDBId: '1',
                pageType: 1,
                loopUrl:'https://loop.frontiersin.org'
            };
        })();
    </script>

    <script>
        var FRAjaxSettings = (function () {

            function urlLowercase() {
                $.ajaxSetup({
                    beforeSend: function (jqXHR, settings) {
                        settings.url = settings.url.toLowerCase();
                    }
                });
            }

            return { urlLowercase: urlLowercase };
        })();

        FRAjaxSettings.urlLowercase();
    </script>

    <script src="Frontiers%20|%20Correcting%20Principal%20Component%20Maps%20for%20Effects%20of%20Spatial%20Autocorrelation%20in%20Population%20Genetic%20Data%20|%20Genetics_fichiers/frontiers.js"></script><script src="Frontiers%20|%20Correcting%20Principal%20Component%20Maps%20for%20Effects%20of%20Spatial%20Autocorrelation%20in%20Population%20Genetic%20Data%20|%20Genetics_fichiers/taxonomy"></script><script src="Frontiers%20|%20Correcting%20Principal%20Component%20Maps%20for%20Effects%20of%20Spatial%20Autocorrelation%20in%20Population%20Genetic%20Data%20|%20Genetics_fichiers/userprofile"></script>

    <script type="text/javascript">

        if (!FRArticle.IsPreview) {

            (function(i, s, o, g, r, a, m) {
                i['GoogleAnalyticsObject'] = r;
                i[r] = i[r] || function() {
                    (i[r].q = i[r].q || []).push(arguments);
                }, i[r].l = 1 * new Date();
                a = s.createElement(o),
                m = s.getElementsByTagName(o)[0];
                a.async = 1;
                a.src = g;
                m.parentNode.insertBefore(a, m);
            })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');

            ga('create', 'UA-9164039-1', 'www.frontiersin.org');
         
        }
    </script>

    
    <script src="Frontiers%20|%20Correcting%20Principal%20Component%20Maps%20for%20Effects%20of%20Spatial%20Autocorrelation%20in%20Population%20Genetic%20Data%20|%20Genetics_fichiers/app"></script>

    <script src="Frontiers%20|%20Correcting%20Principal%20Component%20Maps%20for%20Effects%20of%20Spatial%20Autocorrelation%20in%20Population%20Genetic%20Data%20|%20Genetics_fichiers/figshare.js"></script>




<script type="text/html" id="frontiers-cookie-banner-template">
    <div class="container-cookies">
        <div class="message-cookies">
            We use cookies on this website to improve your user experience. 
            <a id="tyc-link" href="#" target="_blank">Learn more</a>
        </div>
        <div id="btn-accept-cookie" class="close-cookies">Close</div>
    </div>
</script><script type="text/javascript" src="Frontiers%20|%20Correcting%20Principal%20Component%20Maps%20for%20Effects%20of%20Spatial%20Autocorrelation%20in%20Population%20Genetic%20Data%20|%20Genetics_fichiers/fgene.js"></script><div class="container-cookies frontiers" style="display: block;">
        <div class="message-cookies">
            We use cookies on this website to improve your user experience. 
            <a id="tyc-link" href="https://www.frontiersin.org/legal/list-of-cookies" target="_blank">Learn more</a>
        </div>
        <div id="btn-accept-cookie" class="close-cookies">Close</div>
    </div> 
<script type="text/javascript" id="">!function(b,e,f,g,a,c,d){b.fbq||(a=b.fbq=function(){a.callMethod?a.callMethod.apply(a,arguments):a.queue.push(arguments)},b._fbq||(b._fbq=a),a.push=a,a.loaded=!0,a.version="2.0",a.queue=[],c=e.createElement(f),c.async=!0,c.src=g,d=e.getElementsByTagName(f)[0],d.parentNode.insertBefore(c,d))}(window,document,"script","https://connect.facebook.net/en_US/fbevents.js");fbq("init","2098247140231737");fbq("track","PageView");</script>
<noscript><img height="1" width="1" style="display:none" src="https://www.facebook.com/tr?id=2098247140231737&amp;ev=PageView&amp;noscript=1"></noscript>
<div style="position: absolute; width: 0px; height: 0px; overflow: hidden; padding: 0px; border: 0px none; margin: 0px;"><div id="MathJax_Font_Test" style="position: absolute; visibility: hidden; top: 0px; left: 0px; width: auto; padding: 0px; border: 0px none; margin: 0px; white-space: nowrap; text-align: left; text-indent: 0px; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; font-size: 40px; font-weight: normal; font-style: normal; font-size-adjust: none; font-family: MathJax_AMS, sans-serif;"></div></div></body></html>