_altmetric.embed_callback({"title":"Correcting Principal Component Maps for Effects of Spatial Autocorrelation in Population Genetic Data","doi":"10.3389/fgene.2012.00254","pmid":"23181073","pmc":"PMC3501695","isbns":[],"altmetric_jid":"4f6fa81f3cf058f61000bc0d","issns":["1664-8021"],"journal":"Frontiers in Genetics","cohorts":{"pub":1},"abstract":"In many species, spatial genetic variation displays patterns of \"isolation-by-distance.\" Characterized by locally correlated allele frequencies, these patterns are known to create periodic shapes in geographic maps of principal components which confound signatures of specific migration events and influence interpretations of principal component analyses (PCA). In this study, we introduced models combining probabilistic PCA and kriging models to infer population genetic structure from genetic data while correcting for effects generated by spatial autocorrelation. The corresponding algorithms are based on singular value decomposition and low rank approximation of the genotypic data. As their complexity is close to that of PCA, these algorithms scale with the dimensions of the data. To illustrate the utility of these new models, we simulated isolation-by-distance patterns and broad-scale geographic variation using spatial coalescent models. Our methods remove the horseshoe patterns usually observed in PC maps and simplify interpretations of spatial genetic variation. We demonstrate our approach by analyzing single nucleotide polymorphism data from the Human Genome Diversity Panel, and provide comparisons with other recently introduced methods.","abstract_source":"pubmed","context":{"all":{"count":11409579,"mean":7.3105474817611,"rank":9135687,"pct":11,"higher_than":1284946},"journal":{"count":2134,"mean":5.114197843413,"rank":1793,"pct":1,"higher_than":28},"similar_age_3m":{"count":307641,"mean":7.0986493303862,"rank":215797,"pct":15,"higher_than":46540},"similar_age_journal_3m":{"count":269,"mean":5.7657985074627,"rank":225,"pct":1,"higher_than":0}},"authors":["Eric Frichot","Sean Schoville","Guillaume Bouchard","Olivier Fran√ßois"],"type":"article","handles":[],"altmetric_id":1078857,"schema":"1.5.4","is_oa":false,"cited_by_posts_count":1,"cited_by_tweeters_count":1,"cited_by_accounts_count":1,"last_updated":1353418572,"score":0.5,"history":{"1y":0,"6m":0,"3m":0,"1m":0,"1w":0,"6d":0,"5d":0,"4d":0,"3d":0,"2d":0,"1d":0,"at":0.5},"url":"http://dx.doi.org/10.3389/fgene.2012.00254","added_on":1353418577,"published_on":1325376000,"readers":{"citeulike":"0","mendeley":"69","connotea":"0"},"readers_count":69,"images":{"small":"https://badges.altmetric.com/?size=64&score=1&types=tttttttt","medium":"https://badges.altmetric.com/?size=100&score=1&types=tttttttt","large":"https://badges.altmetric.com/?size=180&score=1&types=tttttttt"},"details_url":"http://www.altmetric.com/details.php?citation_id=1078857"})